<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>References – Genomic Foundation Models</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../appendix/app-a-dl.html" rel="next">
<link href="../part_7/p7-ch32-frontiers.html" rel="prev">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="../styles.css">
</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../bib/references.html">References</a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="../">Genomic Foundation Models</a> 
        <div class="sidebar-tools-main">
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Introduction</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../preface.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Preface</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../part_1/p1--foundations.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Part I: Data Foundations</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_1/p1-ch01-ngs.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">From Reads to Variants</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_1/p1-ch02-data.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Data Landscape</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_1/p1-ch03-gwas.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">GWAS and Polygenic Scores</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_1/p1-ch04-vep-classical.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Classical Variant Prediction</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../part_2/p2--architectures.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Part II: Architectures</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_2/p2-ch05-representations.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Tokens and Embeddings</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_2/p2-ch06-cnn.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Convolutional Networks</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_2/p2-ch07-attention.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Transformers and Attention</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../part_3/p3--learning.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Part III: Learning &amp; Evaluation</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_3/p3-ch08-pretraining.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Pretraining Strategies</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_3/p3-ch09-transfer.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Transfer Learning Foundations</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_3/p3-ch10-adaptation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Adaptation Strategies</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_3/p3-ch11-benchmarks.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Benchmark Landscape</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_3/p3-ch12-evaluation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">12</span>&nbsp; <span class="chapter-title">Evaluation Methods</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_3/p3-ch13-confounding.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">13</span>&nbsp; <span class="chapter-title">Confounding and Data Leakage</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../part_4/p4--fm-families.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Part IV: Foundation Model Families</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_4/p4-ch14-fm-principles.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">14</span>&nbsp; <span class="chapter-title">Foundation Model Paradigm</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_4/p4-ch15-dna-lm.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">15</span>&nbsp; <span class="chapter-title">DNA Language Models</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_4/p4-ch16-protein-lm.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">16</span>&nbsp; <span class="chapter-title">Protein Language Models</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_4/p4-ch17-regulatory.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">17</span>&nbsp; <span class="chapter-title">Regulatory Models</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_4/p4-ch18-vep-fm.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">18</span>&nbsp; <span class="chapter-title">Variant Effect Prediction</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../part_5/p5--cellular-context.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Part V: Cellular Context</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-5" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_5/p5-ch19-rna.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">19</span>&nbsp; <span class="chapter-title">RNA Structure and Function</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_5/p5-ch20-single-cell.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">20</span>&nbsp; <span class="chapter-title">Single-Cell Models</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_5/p5-ch21-3d-genome.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">21</span>&nbsp; <span class="chapter-title">3D Genome Organization</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_5/p5-ch22-networks.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">22</span>&nbsp; <span class="chapter-title">Graph and Network Models</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_5/p5-ch23-multi-omics.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">23</span>&nbsp; <span class="chapter-title">Multi-Omics Integration</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../part_6/p6--responsible-deployment.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Part VI: Responsible Deployment</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-6" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_6/p6-ch24-uncertainty.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">24</span>&nbsp; <span class="chapter-title">Uncertainty Quantification</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_6/p6-ch25-interpretability.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">25</span>&nbsp; <span class="chapter-title">Interpretability</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_6/p6-ch26-causal.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">26</span>&nbsp; <span class="chapter-title">Causality</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_6/p6-ch27-regulatory.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">27</span>&nbsp; <span class="chapter-title">Regulatory and Governance</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../part_7/p7--applications.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Part VII: Applications &amp; Frontiers</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-7" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-7" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_7/p7-ch28-clinical-risk.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">28</span>&nbsp; <span class="chapter-title">Clinical Risk Prediction</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_7/p7-ch29-rare-disease.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">29</span>&nbsp; <span class="chapter-title">Rare Disease Diagnosis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_7/p7-ch30-drug-discovery.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">30</span>&nbsp; <span class="chapter-title">Drug Discovery</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_7/p7-ch31-design.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">31</span>&nbsp; <span class="chapter-title">Sequence Design</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../part_7/p7-ch32-frontiers.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">32</span>&nbsp; <span class="chapter-title">Frontiers and Synthesis</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../bib/references.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text">References</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-8" role="navigation" aria-expanded="true">
 <span class="menu-text">Appendices</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-8" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-8" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../appendix/app-a-dl.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">A</span>&nbsp; <span class="chapter-title">Deep Learning Primer</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../appendix/app-b-compute.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">B</span>&nbsp; <span class="chapter-title">Deployment and Compute</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../appendix/app-c-data-curation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">C</span>&nbsp; <span class="chapter-title">Data Curation</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../appendix/app-d-models.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">D</span>&nbsp; <span class="chapter-title">Model Reference</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../appendix/app-e-resources.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">E</span>&nbsp; <span class="chapter-title">Resources</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../appendix/app-f-glossary.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">F</span>&nbsp; <span class="chapter-title">Glossary</span></span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
        
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">References</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list">
<div id="ref-abramoff_pivotal_2018" class="csl-entry" role="listitem">
Abràmoff, Michael D., Philip T. Lavin, Michele Birch, Nilay Shah, and
James C. Folk. 2018. <span>“Pivotal Trial of an Autonomous
<span>AI</span>-Based Diagnostic System for Detection of Diabetic
Retinopathy in Primary Care Offices.”</span> <em>Npj Digital
Medicine</em> 1 (1): 39. <a href="https://doi.org/10.1038/s41746-018-0040-6">https://doi.org/10.1038/s41746-018-0040-6</a>.
</div>
<div id="ref-abramson_alphafold3_2024" class="csl-entry" role="listitem">
Abramson, Josh, Jonas Adler, Jack Dunger, Richard Evans, Tim Green,
Alexander Pritzel, Olaf Ronneberger, et al. 2024.
<span>“[<span>AlphaFold3</span>] <span>Accurate</span> Structure
Prediction of Biomolecular Interactions with <span>AlphaFold</span>
3.”</span> <em>Nature</em> 630 (8016): 493–500. <a href="https://doi.org/10.1038/s41586-024-07487-w">https://doi.org/10.1038/s41586-024-07487-w</a>.
</div>
<div id="ref-adamson_multiplexed_2016" class="csl-entry" role="listitem">
Adamson, Britt, Thomas M. Norman, Marco Jost, Min Y. Cho, James K.
Nuñez, Yuwen Chen, Jacqueline E. Villalta, et al. 2016. <span>“A
<span>Multiplexed</span> <span>Single</span>-<span>Cell</span>
<span>CRISPR</span> <span>Screening</span> <span>Platform</span>
<span>Enables</span> <span>Systematic</span> <span>Dissection</span> of
the <span>Unfolded</span> <span>Protein</span>
<span>Response</span>.”</span> <em>Cell</em> 167 (7): 1867–1882.e21. <a href="https://doi.org/10.1016/j.cell.2016.11.048">https://doi.org/10.1016/j.cell.2016.11.048</a>.
</div>
<div id="ref-adzhubei_method_2010" class="csl-entry" role="listitem">
Adzhubei, Ivan A., Steffen Schmidt, Leonid Peshkin, Vasily E. Ramensky,
Anna Gerasimova, Peer Bork, Alexey S. Kondrashov, and Shamil R. Sunyaev.
2010. <span>“A Method and Server for Predicting Damaging Missense
Mutations.”</span> <em>Nature Methods</em> 7 (4): 248–49. <a href="https://doi.org/10.1038/nmeth0410-248">https://doi.org/10.1038/nmeth0410-248</a>.
</div>
<div id="ref-agarwal_predicting_2020" class="csl-entry" role="listitem">
Agarwal, Vikram, and Jay Shendure. 2020. <span>“Predicting <span class="nocase">mRNA</span> <span>Abundance</span> <span>Directly</span>
from <span>Genomic</span> <span>Sequence</span> <span>Using</span>
<span>Deep</span> <span>Convolutional</span> <span>Neural</span>
<span>Networks</span>.”</span> <em>Cell Reports</em> 31 (7): 107663. <a href="https://doi.org/10.1016/j.celrep.2020.107663">https://doi.org/10.1016/j.celrep.2020.107663</a>.
</div>
<div id="ref-ahlqvist_novel_2018" class="csl-entry" role="listitem">
Ahlqvist, Emma, Petter Storm, Annemari Käräjämäki, Mats Martinell,
Mozhgan Dorkhan, Annelie Carlsson, Petter Vikman, et al. 2018.
<span>“Novel Subgroups of Adult-Onset Diabetes and Their Association
with Outcomes: A Data-Driven Cluster Analysis of Six Variables.”</span>
<em>The Lancet Diabetes &amp; Endocrinology</em> 6 (5): 361–69. <a href="https://doi.org/10.1016/S2213-8587(18)30051-2">https://doi.org/10.1016/S2213-8587(18)30051-2</a>.
</div>
<div id="ref-all_of_us_research_program_investigators_the_all_2019" class="csl-entry" role="listitem">
All of Us Research Program Investigators, The. 2019. <span>“The
<span>‘<span>All</span> of <span>Us</span>’</span> <span>Research</span>
<span>Program</span>.”</span> <em>New England Journal of Medicine</em>
381 (7): 668–76. <a href="https://doi.org/10.1056/NEJMsr1809937">https://doi.org/10.1056/NEJMsr1809937</a>.
</div>
<div id="ref-amberger_omimorg_2015" class="csl-entry" role="listitem">
Amberger, Joanna S., Carol A. Bocchini, François Schiettecatte, Alan F.
Scott, and Ada Hamosh. 2015. <span>“<span>OMIM</span>.org:
<span>Online</span> <span>Mendelian</span> <span>Inheritance</span> in
<span>Man</span> (<span>OMIM</span>®), an Online Catalog of Human Genes
and Genetic Disorders.”</span> <em>Nucleic Acids Research</em> 43 (D1):
D789–98. <a href="https://doi.org/10.1093/nar/gku1205">https://doi.org/10.1093/nar/gku1205</a>.
</div>
<div id="ref-andre_alpelisib_2019" class="csl-entry" role="listitem">
André, Fabrice, Eva Ciruelos, Gabor Rubovszky, Mario Campone, Sibylle
Loibl, Hope S. Rugo, Hiroji Iwata, et al. 2019. <span>“Alpelisib for
<span>PIK3CA</span>-<span>Mutated</span>, <span>Hormone</span>
<span>Receptor</span>–<span>Positive</span> <span>Advanced</span>
<span>Breast</span> <span>Cancer</span>.”</span> <em>New England Journal
of Medicine</em> 380 (20): 1929–40. <a href="https://doi.org/10.1056/NEJMoa1813904">https://doi.org/10.1056/NEJMoa1813904</a>.
</div>
<div id="ref-angelopoulos_conformal_2023" class="csl-entry" role="listitem">
Angelopoulos, Anastasios N., and Stephen Bates. 2023. <span>“Conformal
<span>Prediction</span>: <span>A</span> <span>Gentle</span>
<span>Introduction</span>.”</span> <em>Foundations and Trends® in
Machine Learning</em> 16 (4): 494–591. <a href="https://doi.org/10.1561/2200000101">https://doi.org/10.1561/2200000101</a>.
</div>
<div id="ref-argelaguet_mofa_2018" class="csl-entry" role="listitem">
Argelaguet, Ricard, Damien Arnol, Danila Bredikhin, Yonatan Deloro,
Britta Velten, John C. Marioni, and Oliver Stegle. 2020.
<span>“<span>MOFA</span>+: A Statistical Framework for Comprehensive
Integration of Multi-Modal Single-Cell Data.”</span> <em>Genome
Biology</em> 21: 111. <a href="https://doi.org/10.1186/s13059-020-02015-1">https://doi.org/10.1186/s13059-020-02015-1</a>.
</div>
<div id="ref-arnold_thaler_2021" class="csl-entry" role="listitem">
Arnold, Lord Justice, Lady Justice Laing, and Lord Justice Birss. 2021.
<span>“Thaler v <span>Comptroller</span> <span>General</span> of
<span>Patents</span> <span>Trade</span> <span>Marks</span>
<span>And</span> <span>Designs</span> [2021] <span>EWCA</span>
<span>Civ</span> 1374.”</span>
</div>
<div id="ref-auton_global_2015" class="csl-entry" role="listitem">
Auton, Adam, Gonçalo R. Abecasis, David M. Altshuler, Richard M. Durbin,
Gonçalo R. Abecasis, David R. Bentley, Aravinda Chakravarti, et al.
2015. <span>“A Global Reference for Human Genetic Variation.”</span>
<em>Nature</em> 526 (7571): 68–74. <a href="https://doi.org/10.1038/nature15393">https://doi.org/10.1038/nature15393</a>.
</div>
<div id="ref-avsec_enformer_2021" class="csl-entry" role="listitem">
Avsec, Žiga, Vikram Agarwal, Daniel Visentin, Joseph R. Ledsam,
Agnieszka Grabska-Barwinska, Kyle R. Taylor, Yannis Assael, John Jumper,
Pushmeet Kohli, and David R. Kelley. 2021. <span>“Effective Gene
Expression Prediction from Sequence by Integrating Long-Range
Interactions.”</span> <em>Nature Methods</em> 18 (10): 1196–1203. <a href="https://doi.org/10.1038/s41592-021-01252-x">https://doi.org/10.1038/s41592-021-01252-x</a>.
</div>
<div id="ref-avsec_alphagenome_2025" class="csl-entry" role="listitem">
Avsec, Ziga, Natasha Latysheva, and Jun Cheng. 2025.
<span>“<span>AlphaGenome</span>: <span>AI</span> for Better
Understanding the Genome.”</span>
</div>
<div id="ref-belkin_reconciling_2019" class="csl-entry" role="listitem">
Belkin, Mikhail, Daniel Hsu, Siyuan Ma, and Soumik Mandal. 2019.
<span>“Reconciling Modern Machine-Learning Practice and the Classical
Bias–Variance Trade-Off.”</span> <em>Proceedings of the National Academy
of Sciences</em> 116 (32): 15849–54. <a href="https://doi.org/10.1073/pnas.1903070116">https://doi.org/10.1073/pnas.1903070116</a>.
</div>
<div id="ref-ben-david_theory_2010" class="csl-entry" role="listitem">
Ben-David, Shai, John Blitzer, Koby Crammer, Alex Kulesza, Fernando
Pereira, and Jennifer Wortman Vaughan. 2010. <span>“A Theory of Learning
from Different Domains.”</span> <em>Machine Learning</em> 79 (1):
151–75. <a href="https://doi.org/10.1007/s10994-009-5152-4">https://doi.org/10.1007/s10994-009-5152-4</a>.
</div>
<div id="ref-benegas_gpn-msa_2024" class="csl-entry" role="listitem">
Benegas, Gonzalo, Carlos Albors, Alan J. Aw, Chengzhong Ye, and Yun S.
Song. 2024. <span>“<span>GPN</span>-<span>MSA</span>: An Alignment-Based
<span>DNA</span> Language Model for Genome-Wide Variant Effect
Prediction.”</span> <em>bioRxiv</em>, April, 2023.10.10.561776. <a href="https://doi.org/10.1101/2023.10.10.561776">https://doi.org/10.1101/2023.10.10.561776</a>.
</div>
<div id="ref-benegas_gpn_2023" class="csl-entry" role="listitem">
Benegas, Gonzalo, Sanjit Singh Batra, and Yun S. Song. 2023.
<span>“[<span>GPN</span>] <span>DNA</span> Language Models Are Powerful
Predictors of Genome-Wide Variant Effects.”</span> <em>Proceedings of
the National Academy of Sciences</em> 120 (44): e2311219120. <a href="https://doi.org/10.1073/pnas.2311219120">https://doi.org/10.1073/pnas.2311219120</a>.
</div>
<div id="ref-benegas_traitgym_2025" class="csl-entry" role="listitem">
Benegas, Gonzalo, Gökcen Eraslan, and Yun S. Song. 2025.
<span>“[<span>TraitGym</span>] <span>Benchmarking</span>
<span>DNA</span> <span>Sequence</span> <span>Models</span> for
<span>Causal</span> <span>Regulatory</span> <span>Variant</span>
<span>Prediction</span> in <span>Human</span>
<span>Genetics</span>.”</span> bioRxiv. <a href="https://doi.org/10.1101/2025.02.11.637758">https://doi.org/10.1101/2025.02.11.637758</a>.
</div>
<div id="ref-benjamini_controlling_1995" class="csl-entry" role="listitem">
Benjamini, Yoav, and Yosef Hochberg. 1995. <span>“Controlling the
<span>False</span> <span>Discovery</span> <span>Rate</span>:
<span>A</span> <span>Practical</span> and <span>Powerful</span>
<span>Approach</span> to <span>Multiple</span>
<span>Testing</span>.”</span> <em>Journal of the Royal Statistical
Society: Series B (Methodological)</em> 57 (1): 289–300. <a href="https://doi.org/10.1111/j.2517-6161.1995.tb02031.x">https://doi.org/10.1111/j.2517-6161.1995.tb02031.x</a>.
</div>
<div id="ref-benner_finemap_2016" class="csl-entry" role="listitem">
Benner, Christian, Chris C. A. Spencer, Aki S. Havulinna, Veikko
Salomaa, Samuli Ripatti, and Matti Pirinen. 2016.
<span>“<span>FINEMAP</span>: Efficient Variable Selection Using Summary
Data from Genome-Wide Association Studies.”</span>
<em>Bioinformatics</em> 32 (10): 1493–1501. <a href="https://doi.org/10.1093/bioinformatics/btw018">https://doi.org/10.1093/bioinformatics/btw018</a>.
</div>
<div id="ref-bergquist_calibration_2025" class="csl-entry" role="listitem">
Bergquist, Timothy, Sarah L. Stenton, Emily A. W. Nadeau, Alicia B.
Byrne, Marc S. Greenblatt, Steven M. Harrison, Sean V. Tavtigian, et al.
2025. <span>“Calibration of Additional Computational Tools Expands
<span>ClinGen</span> Recommendation Options for Variant Classification
with <span>PP3</span>/<span>BP4</span> Criteria.”</span> <em>Genetics in
Medicine</em> 27 (6): 101402. <a href="https://doi.org/10.1016/j.gim.2025.101402">https://doi.org/10.1016/j.gim.2025.101402</a>.
</div>
<div id="ref-berman_protein_2000" class="csl-entry" role="listitem">
Berman, Helen M., John Westbrook, Zukang Feng, Gary Gilliland, T. N.
Bhat, Helge Weissig, Ilya N. Shindyalov, and Philip E. Bourne. 2000.
<span>“The <span>Protein</span> <span>Data</span>
<span>Bank</span>.”</span> <em>Nucleic Acids Research</em> 28 (1):
235–42. <a href="https://doi.org/10.1093/nar/28.1.235">https://doi.org/10.1093/nar/28.1.235</a>.
</div>
<div id="ref-birman-deych_accuracy_2005" class="csl-entry" role="listitem">
Birman-Deych, Elena, Amy D. Waterman, Yan Yan, David S. Nilasena, Martha
J. Radford, and Brian F. Gage. 2005. <span>“Accuracy of
<span>ICD</span>-9-<span>CM</span> <span>Codes</span> for
<span>Identifying</span> <span>Cardiovascular</span> and
<span>Stroke</span> <span>Risk</span> <span>Factors</span>.”</span>
<em>Medical Care</em> 43 (5): 480. <a href="https://doi.org/10.1097/01.mlr.0000160417.39497.a9">https://doi.org/10.1097/01.mlr.0000160417.39497.a9</a>.
</div>
<div id="ref-de_boer_deciphering_2019" class="csl-entry" role="listitem">
Boer, Carl G. de, Eeshit Dhaval Vaishnav, Ronen Sadeh, Esteban Luis
Abeyta, Nir Friedman, and Aviv Regev. 2019. <span>“Deciphering
Eukaryotic Gene-Regulatory Logic with 100 Million Random
Promoters.”</span> <em>Nature Biotechnology</em> 38 (1): 56–65. <a href="https://doi.org/10.1038/s41587-019-0315-8">https://doi.org/10.1038/s41587-019-0315-8</a>.
</div>
<div id="ref-bowden_mendelian_2015" class="csl-entry" role="listitem">
Bowden, Jack, George Davey Smith, and Stephen Burgess. 2015.
<span>“Mendelian Randomization with Invalid Instruments: Effect
Estimation and Bias Detection Through <span>Egger</span>
Regression.”</span> <em>International Journal of Epidemiology</em> 44
(2): 512–25. <a href="https://doi.org/10.1093/ije/dyv080">https://doi.org/10.1093/ije/dyv080</a>.
</div>
<div id="ref-brandes_genome-wide_2023" class="csl-entry" role="listitem">
Brandes, Nadav, Grant Goldman, Charlotte H. Wang, Chun Jimmie Ye, and
Vasilis Ntranos. 2023. <span>“Genome-Wide Prediction of Disease Variant
Effects with a Deep Protein Language Model.”</span> <em>Nature
Genetics</em> 55 (9): 1512–22. <a href="https://doi.org/10.1038/s41588-023-01465-0">https://doi.org/10.1038/s41588-023-01465-0</a>.
</div>
<div id="ref-breiman_statistical_2001" class="csl-entry" role="listitem">
Breiman, Leo. 2001. <span>“Statistical <span>Modeling</span>:
<span>The</span> <span>Two</span> <span>Cultures</span>.”</span>
<em>Statistical Science</em>, August.
</div>
<div id="ref-brixi_evo_2025" class="csl-entry" role="listitem">
Brixi, Garyk, Matthew G. Durrant, Jerome Ku, Michael Poli, Greg
Brockman, Daniel Chang, Gabriel A. Gonzalez, et al. 2025.
<span>“[<span>Evo</span> 2] <span>Genome</span> Modeling and Design
Across All Domains of Life with <span>Evo</span> 2.”</span> bioRxiv. <a href="https://doi.org/10.1101/2025.02.18.638918">https://doi.org/10.1101/2025.02.18.638918</a>.
</div>
<div id="ref-brnich_recommendations_2019" class="csl-entry" role="listitem">
Brnich, Sarah E., Ahmad N. Abou Tayoun, Fergus J. Couch, Garry R.
Cutting, Marc S. Greenblatt, Christopher D. Heinen, Dona M. Kanavy, et
al. 2019. <span>“Recommendations for Application of the Functional
Evidence <span>PS3</span>/<span>BS3</span> Criterion Using the
<span>ACMG</span>/<span>AMP</span> Sequence Variant Interpretation
Framework.”</span> <em>Genome Medicine</em> 12 (1): 3. <a href="https://doi.org/10.1186/s13073-019-0690-2">https://doi.org/10.1186/s13073-019-0690-2</a>.
</div>
<div id="ref-brown_language_2020" class="csl-entry" role="listitem">
Brown, Tom, Benjamin Mann, Nick Ryder, Melanie Subbiah, Jared D. Kaplan,
Prafulla Dhariwal, Arvind Neelakantan, et al. 2020. <span>“Language
<span>Models</span> Are <span>Few</span>-<span>Shot</span>
<span>Learners</span>.”</span> <em>Advances in Neural Information
Processing Systems</em> 33 (December): 1877–1901.
</div>
<div id="ref-browning_fast_2021" class="csl-entry" role="listitem">
Browning, Brian L., Xiaowen Tian, Ying Zhou, and Sharon R. Browning.
2021. <span>“Fast Two-Stage Phasing of Large-Scale Sequence
Data.”</span> <em>American Journal of Human Genetics</em> 108 (10):
1880–90. <a href="https://doi.org/10.1016/j.ajhg.2021.08.005">https://doi.org/10.1016/j.ajhg.2021.08.005</a>.
</div>
<div id="ref-buniello_open_2025" class="csl-entry" role="listitem">
Buniello, Annalisa, Daniel Suveges, Carlos Cruz-Castillo, Manuel Bernal
Llinares, Helena Cornu, Irene Lopez, Kirill Tsukanov, et al. 2025.
<span>“Open <span>Targets</span> <span>Platform</span>: Facilitating
Therapeutic Hypotheses Building in Drug Discovery.”</span> <em>Nucleic
Acids Research</em> 53 (D1): D1467–75. <a href="https://doi.org/10.1093/nar/gkae1128">https://doi.org/10.1093/nar/gkae1128</a>.
</div>
<div id="ref-bycroft_uk_2018" class="csl-entry" role="listitem">
Bycroft, Clare, Colin Freeman, Desislava Petkova, Gavin Band, Lloyd T.
Elliott, Kevin Sharp, Allan Motyer, et al. 2018. <span>“The
<span>UK</span> <span>Biobank</span> Resource with Deep Phenotyping and
Genomic Data.”</span> <em>Nature</em> 562 (7726): 203–9. <a href="https://doi.org/10.1038/s41586-018-0579-z">https://doi.org/10.1038/s41586-018-0579-z</a>.
</div>
<div id="ref-camillo_cpgpt_2024" class="csl-entry" role="listitem">
Camillo, Lucas Paulo de Lima, Raghav Sehgal, Jenel Armstrong, Albert T.
Higgins-Chen, Steve Horvath, and Bo Wang. 2024.
<span>“<span>CpGPT</span>: A <span>Foundation</span> <span>Model</span>
for <span>DNA</span> <span>Methylation</span>.”</span> bioRxiv. <a href="https://doi.org/10.1101/2024.10.24.619766">https://doi.org/10.1101/2024.10.24.619766</a>.
</div>
<div id="ref-cao_glue_2022" class="csl-entry" role="listitem">
Cao, Zhi-Jie, and Ge Gao. 2022. <span>“[<span>GLUE</span>]
<span>Multi</span>-Omics Single-Cell Data Integration and Regulatory
Inference with Graph-Linked Embedding.”</span> <em>Nature
Biotechnology</em> 40 (10): 1458–66. <a href="https://doi.org/10.1038/s41587-022-01284-4">https://doi.org/10.1038/s41587-022-01284-4</a>.
</div>
<div id="ref-center_for_disease_control_acce_2022" class="csl-entry" role="listitem">
Center for Disease Control. 2022. <span>“<span>ACCE</span>
<span>Model</span> <span>Process</span> for <span>Evaluating</span>
<span>Genetic</span> <span>Tests</span>.”</span>
</div>
<div id="ref-chapman_improved_2011" class="csl-entry" role="listitem">
Chapman, Paul B., Axel Hauschild, Caroline Robert, John B. Haanen, Paolo
Ascierto, James Larkin, Reinhard Dummer, et al. 2011. <span>“Improved
<span>Survival</span> with <span>Vemurafenib</span> in
<span>Melanoma</span> with <span>BRAF</span> <span>V600E</span>
<span>Mutation</span>.”</span> <em>New England Journal of Medicine</em>
364 (26): 2507–16. <a href="https://doi.org/10.1056/NEJMoa1103782">https://doi.org/10.1056/NEJMoa1103782</a>.
</div>
<div id="ref-chawla_smote_2002" class="csl-entry" role="listitem">
Chawla, Nitesh V., Kevin W. Bowyer, Lawrence O. Hall, and W. Philip
Kegelmeyer. 2002. <span>“<span>SMOTE</span>: Synthetic Minority
over-Sampling Technique.”</span> <em>J. Artif. Int. Res.</em> 16 (1):
321–57.
</div>
<div id="ref-chen_rates_2023" class="csl-entry" role="listitem">
Chen, Elaine, Flavia M. Facio, Kerry W. Aradhya, Susan Rojahn, Kathryn
E. Hatchell, Sienna Aguilar, Karen Ouyang, et al. 2023. <span>“Rates and
<span>Classification</span> of <span>Variants</span> of
<span>Uncertain</span> <span>Significance</span> in
<span>Hereditary</span> <span>Disease</span> <span>Genetic</span>
<span>Testing</span>.”</span> <em>JAMA Network Open</em> 6 (10):
e2339571. <a href="https://doi.org/10.1001/jamanetworkopen.2023.39571">https://doi.org/10.1001/jamanetworkopen.2023.39571</a>.
</div>
<div id="ref-chen_rna-fm_2022" class="csl-entry" role="listitem">
Chen, Jiayang, Zhihang Hu, Siqi Sun, Qingxiong Tan, Yixuan Wang, Qinze
Yu, Licheng Zong, et al. 2022. <span>“[<span>RNA</span>-<span>FM</span>]
<span>Interpretable</span> <span>RNA</span> <span>Foundation</span>
<span>Model</span> from <span>Unannotated</span> <span>Data</span> for
<span>Highly</span> <span>Accurate</span> <span>RNA</span>
<span>Structure</span> and <span>Function</span>
<span>Predictions</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2204.00300">https://doi.org/10.48550/arXiv.2204.00300</a>.
</div>
<div id="ref-chen_deepsea_2022" class="csl-entry" role="listitem">
Chen, Kathleen M., Aaron K. Wong, Olga G. Troyanskaya, and Jian Zhou.
2022. <span>“[<span>DeepSEA</span> <span>Sei</span>] <span>A</span>
Sequence-Based Global Map of Regulatory Activity for Deciphering Human
Genetics.”</span> <em>Nature Genetics</em> 54 (7): 940–49. <a href="https://doi.org/10.1038/s41588-022-01102-2">https://doi.org/10.1038/s41588-022-01102-2</a>.
</div>
<div id="ref-cheng_alphamissense_2023" class="csl-entry" role="listitem">
Cheng, Jun, Guido Novati, Joshua Pan, Clare Bycroft, Akvilė Žemgulytė,
Taylor Applebaum, Alexander Pritzel, et al. 2023.
<span>“[<span>AlphaMissense</span>] <span>Accurate</span> Proteome-Wide
Missense Variant Effect Prediction with
<span>AlphaMissense</span>.”</span> <em>Science</em> 381 (6664):
eadg7492. <a href="https://doi.org/10.1126/science.adg7492">https://doi.org/10.1126/science.adg7492</a>.
</div>
<div id="ref-cheng_dnalongbench_2024" class="csl-entry" role="listitem">
Cheng, Wenduo, Zhenqiao Song, Yang Zhang, Shike Wang, Danqing Wang, Muyu
Yang, Lei Li, and Jian Ma. 2024. <span>“<span>DNALONGBENCH</span>:
<span>A</span> <span>Benchmark</span> <span>Suite</span>
<span>For</span> <span>Long</span>-<span>Range</span> <span>DNA</span>
<span>Prediction</span> <span>Tasks</span>,”</span> October.
</div>
<div id="ref-cho_properties_2014" class="csl-entry" role="listitem">
Cho, Kyunghyun, Bart van Merrienboer, Dzmitry Bahdanau, and Yoshua
Bengio. 2014. <span>“On the <span>Properties</span> of
<span>Neural</span> <span>Machine</span> <span>Translation</span>:
<span>Encoder</span>-<span>Decoder</span>
<span>Approaches</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.1409.1259">https://doi.org/10.48550/arXiv.1409.1259</a>.
</div>
<div id="ref-choi_prs_2020" class="csl-entry" role="listitem">
Choi, Shing Wan, Timothy Shin-Heng Mak, and Paul F. O’Reilly. 2020.
<span>“[<span>PRS</span>] <span>Tutorial</span>: A Guide to Performing
Polygenic Risk Score Analyses.”</span> <em>Nature Protocols</em> 15 (9):
2759–72. <a href="https://doi.org/10.1038/s41596-020-0353-1">https://doi.org/10.1038/s41596-020-0353-1</a>.
</div>
<div id="ref-choromanski_rethinking_2022" class="csl-entry" role="listitem">
Choromanski, Krzysztof, Valerii Likhosherstov, David Dohan, Xingyou
Song, Andreea Gane, Tamas Sarlos, Peter Hawkins, et al. 2022.
<span>“Rethinking <span>Attention</span> with
<span>Performers</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2009.14794">https://doi.org/10.48550/arXiv.2009.14794</a>.
</div>
<div id="ref-chowdhery_palm_2022" class="csl-entry" role="listitem">
Chowdhery, Aakanksha, Sharan Narang, Jacob Devlin, Maarten Bosma, Gaurav
Mishra, Adam Roberts, Paul Barham, et al. 2022.
<span>“<span>PaLM</span>: <span>Scaling</span> <span>Language</span>
<span>Modeling</span> with <span>Pathways</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2204.02311">https://doi.org/10.48550/arXiv.2204.02311</a>.
</div>
<div id="ref-cirulli_genome-wide_2020" class="csl-entry" role="listitem">
Cirulli, Elizabeth T., Simon White, Robert W. Read, Gai Elhanan, William
J. Metcalf, Francisco Tanudjaja, Donna M. Fath, et al. 2020.
<span>“Genome-Wide Rare Variant Analysis for Thousands of Phenotypes in
over 70,000 Exomes from Two Cohorts.”</span> <em>Nature
Communications</em> 11 (1): 542. <a href="https://doi.org/10.1038/s41467-020-14288-y">https://doi.org/10.1038/s41467-020-14288-y</a>.
</div>
<div id="ref-clarke_deeprvat_2024" class="csl-entry" role="listitem">
Clarke, Brian, Eva Holtkamp, Hakime Öztürk, Marcel Mück, Magnus
Wahlberg, Kayla Meyer, Felix Munzlinger, et al. 2024.
<span>“[<span>DeepRVAT</span>] <span>Integration</span> of Variant
Annotations Using Deep Set Networks Boosts Rare Variant Association
Testing.”</span> <em>Nature Genetics</em> 56 (10): 2271–80. <a href="https://doi.org/10.1038/s41588-024-01919-z">https://doi.org/10.1038/s41588-024-01919-z</a>.
</div>
<div id="ref-cui_scgpt_2024" class="csl-entry" role="listitem">
Cui, Haotian, Chloe Wang, Hassaan Maan, Kuan Pang, Fengning Luo, Nan
Duan, and Bo Wang. 2024. <span>“<span class="nocase">scGPT</span>:
Toward Building a Foundation Model for Single-Cell Multi-Omics Using
Generative <span>AI</span>.”</span> <em>Nature Methods</em> 21 (8):
1470–80. <a href="https://doi.org/10.1038/s41592-024-02201-0">https://doi.org/10.1038/s41592-024-02201-0</a>.
</div>
<div id="ref-cui_class-balanced_2019" class="csl-entry" role="listitem">
Cui, Yin, Menglin Jia, Tsung-Yi Lin, Yang Song, and Serge Belongie.
2019. <span>“Class-<span>Balanced</span> <span>Loss</span>
<span>Based</span> on <span>Effective</span> <span>Number</span> of
<span>Samples</span>.”</span> In <em>2019
<span>IEEE</span>/<span>CVF</span> <span>Conference</span> on
<span>Computer</span> <span>Vision</span> and <span>Pattern</span>
<span>Recognition</span> (<span>CVPR</span>)</em>, 9260–69. <a href="https://doi.org/10.1109/CVPR.2019.00949">https://doi.org/10.1109/CVPR.2019.00949</a>.
</div>
<div id="ref-dabernig-heinz_multicenter_2024" class="csl-entry" role="listitem">
Dabernig-Heinz, Johanna, Mara Lohde, Martin Hölzer, Adriana Cabal, Rick
Conzemius, Christian Brandt, Matthias Kohl, et al. 2024. <span>“A
Multicenter Study on Accuracy and Reproducibility of Nanopore
Sequencing-Based Genotyping of Bacterial Pathogens.”</span> <em>Journal
of Clinical Microbiology</em> 62 (9): e00628–24. <a href="https://doi.org/10.1128/jcm.00628-24">https://doi.org/10.1128/jcm.00628-24</a>.
</div>
<div id="ref-dallago_flip_2022" class="csl-entry" role="listitem">
Dallago, Christian, Jody Mou, Kadina E. Johnston, Bruce J. Wittmann,
Nicholas Bhattacharya, Samuel Goldman, Ali Madani, and Kevin K. Yang.
2022. <span>“<span>FLIP</span>: <span>Benchmark</span> Tasks in Fitness
Landscape Inference for Proteins.”</span> bioRxiv. <a href="https://doi.org/10.1101/2021.11.09.467890">https://doi.org/10.1101/2021.11.09.467890</a>.
</div>
<div id="ref-dalla-torre_nucleotide_2023" class="csl-entry" role="listitem">
Dalla-Torre, Hugo, Liam Gonzalez, Javier Mendoza-Revilla, Nicolas Lopez
Carranza, Adam Henryk Grzywaczewski, Francesco Oteri, Christian Dallago,
et al. 2023. <span>“Nucleotide <span>Transformer</span>: Building and
Evaluating Robust Foundation Models for Human Genomics.”</span>
<em>Nature Methods</em> 22 (2): 287–97. <a href="https://doi.org/10.1038/s41592-024-02523-z">https://doi.org/10.1038/s41592-024-02523-z</a>.
</div>
<div id="ref-dao_flashattention_2022" class="csl-entry" role="listitem">
Dao, Tri, Dan Fu, Stefano Ermon, Atri Rudra, and Christopher Ré. 2022.
<span>“<span>FlashAttention</span>: <span>Fast</span> and
<span>Memory</span>-<span>Efficient</span> <span>Exact</span>
<span>Attention</span> with
<span>IO</span>-<span>Awareness</span>.”</span> <em>Advances in Neural
Information Processing Systems</em> 35 (December): 16344–59.
</div>
<div id="ref-dauparas_robust_2022" class="csl-entry" role="listitem">
Dauparas, J., I. Anishchenko, N. Bennett, H. Bai, R. J. Ragotte, L. F.
Milles, B. I. M. Wicky, et al. 2022. <span>“Robust Deep Learning–Based
Protein Sequence Design Using <span>ProteinMPNN</span>.”</span>
<em>Science</em> 378 (6615): 49–56. <a href="https://doi.org/10.1126/science.add2187">https://doi.org/10.1126/science.add2187</a>.
</div>
<div id="ref-davey_smith_mendelian_2003" class="csl-entry" role="listitem">
Davey Smith, George, and Shah Ebrahim. 2003.
<span>“<span>‘<span>Mendelian</span> Randomization’</span>: Can Genetic
Epidemiology Contribute to Understanding Environmental Determinants of
Disease?*.”</span> <em>International Journal of Epidemiology</em> 32
(1): 1–22. <a href="https://doi.org/10.1093/ije/dyg070">https://doi.org/10.1093/ije/dyg070</a>.
</div>
<div id="ref-davydov_identifying_2010" class="csl-entry" role="listitem">
Davydov, Eugene V., David L. Goode, Marina Sirota, Gregory M. Cooper,
Arend Sidow, and Serafim Batzoglou. 2010. <span>“Identifying a
<span>High</span> <span>Fraction</span> of the <span>Human</span>
<span>Genome</span> to Be Under <span>Selective</span>
<span>Constraint</span> <span>Using</span> <span>GERP</span>++.”</span>
<em>PLOS Computational Biology</em> 6 (12): e1001025. <a href="https://doi.org/10.1371/journal.pcbi.1001025">https://doi.org/10.1371/journal.pcbi.1001025</a>.
</div>
<div id="ref-depristo_framework_2011" class="csl-entry" role="listitem">
DePristo, Mark A., Eric Banks, Ryan Poplin, Kiran V. Garimella, Jared R.
Maguire, Christopher Hartl, Anthony A. Philippakis, et al. 2011.
<span>“A Framework for Variation Discovery and Genotyping Using
Next-Generation <span>DNA</span> Sequencing Data.”</span> <em>Nature
Genetics</em> 43 (5): 491–98. <a href="https://doi.org/10.1038/ng.806">https://doi.org/10.1038/ng.806</a>.
</div>
<div id="ref-devlin_bert_2019" class="csl-entry" role="listitem">
Devlin, Jacob, Ming-Wei Chang, Kenton Lee, and Kristina Toutanova. 2019.
<span>“<span>BERT</span>: <span>Pre</span>-Training of <span>Deep</span>
<span>Bidirectional</span> <span>Transformers</span> for
<span>Language</span> <span>Understanding</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.1810.04805">https://doi.org/10.48550/arXiv.1810.04805</a>.
</div>
<div id="ref-dixit_perturb-seq_2016" class="csl-entry" role="listitem">
Dixit, Atray, Oren Parnas, Biyu Li, Jenny Chen, Charles P. Fulco, Livnat
Jerby-Arnon, Nemanja D. Marjanovic, et al. 2016.
<span>“Perturb-<span>Seq</span>: <span>Dissecting</span>
<span>Molecular</span> <span>Circuits</span> with <span>Scalable</span>
<span>Single</span>-<span>Cell</span> <span>RNA</span>
<span>Profiling</span> of <span>Pooled</span> <span>Genetic</span>
<span>Screens</span>.”</span> <em>Cell</em> 167 (7): 1853–1866.e17. <a href="https://doi.org/10.1016/j.cell.2016.11.038">https://doi.org/10.1016/j.cell.2016.11.038</a>.
</div>
<div id="ref-dixon_topological_2012" class="csl-entry" role="listitem">
Dixon, Jesse R., Siddarth Selvaraj, Feng Yue, Audrey Kim, Yan Li, Yin
Shen, Ming Hu, Jun S. Liu, and Bing Ren. 2012. <span>“Topological
Domains in Mammalian Genomes Identified by Analysis of Chromatin
Interactions.”</span> <em>Nature</em> 485 (7398): 376–80. <a href="https://doi.org/10.1038/nature11082">https://doi.org/10.1038/nature11082</a>.
</div>
<div id="ref-dockes_preventing_2021" class="csl-entry" role="listitem">
Dockès, Jérôme, Gaël Varoquaux, and Jean-Baptiste Poline. 2021.
<span>“Preventing Dataset Shift from Breaking Machine-Learning
Biomarkers.”</span> <em>GigaScience</em> 10 (9): giab055. <a href="https://doi.org/10.1093/gigascience/giab055">https://doi.org/10.1093/gigascience/giab055</a>.
</div>
<div id="ref-duncan_analysis_2019" class="csl-entry" role="listitem">
Duncan, L., H. Shen, B. Gelaye, J. Meijsen, K. Ressler, M. Feldman, R.
Peterson, and B. Domingue. 2019. <span>“Analysis of Polygenic Risk Score
Usage and Performance in Diverse Human Populations.”</span> <em>Nature
Communications</em> 10 (1): 3328. <a href="https://doi.org/10.1038/s41467-019-11112-0">https://doi.org/10.1038/s41467-019-11112-0</a>.
</div>
<div id="ref-edgar_gene_2002" class="csl-entry" role="listitem">
Edgar, Ron, Michael Domrachev, and Alex E. Lash. 2002. <span>“Gene
<span>Expression</span> <span>Omnibus</span>: <span>NCBI</span> Gene
Expression and Hybridization Array Data Repository.”</span> <em>Nucleic
Acids Research</em> 30 (1): 207–10. <a href="https://doi.org/10.1093/nar/30.1.207">https://doi.org/10.1093/nar/30.1.207</a>.
</div>
<div id="ref-elks_variability_2012" class="csl-entry" role="listitem">
Elks, Cathy E., Marcel Den Hoed, Jing Hua Zhao, Stephen J. Sharp,
Nicholas J. Wareham, Ruth J. F. Loos, and Ken K. Ong. 2012.
<span>“Variability in the <span>Heritability</span> of <span>Body</span>
<span>Mass</span> <span>Index</span>: <span>A</span>
<span>Systematic</span> <span>Review</span> and
<span>Meta</span>-<span>Regression</span>.”</span> <em>Frontiers in
Endocrinology</em> 3 (February). <a href="https://doi.org/10.3389/fendo.2012.00029">https://doi.org/10.3389/fendo.2012.00029</a>.
</div>
<div id="ref-elnaggar_prottrans_2021" class="csl-entry" role="listitem">
Elnaggar, Ahmed, Michael Heinzinger, Christian Dallago, Ghalia Rihawi,
Yu Wang, Llion Jones, Tom Gibbs, et al. 2021.
<span>“<span>ProtTrans</span>: <span>Towards</span>
<span>Cracking</span> the <span>Language</span> of <span>Life</span>’s
<span>Code</span> <span>Through</span>
<span>Self</span>-<span>Supervised</span> <span>Deep</span>
<span>Learning</span> and <span>High</span> <span>Performance</span>
<span>Computing</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2007.06225">https://doi.org/10.48550/arXiv.2007.06225</a>.
</div>
<div id="ref-erlich_routes_2014" class="csl-entry" role="listitem">
Erlich, Yaniv, and Arvind Narayanan. 2014. <span>“Routes for Breaching
and Protecting Genetic Privacy.”</span> <em>Nature Reviews Genetics</em>
15 (6): 409–21. <a href="https://doi.org/10.1038/nrg3723">https://doi.org/10.1038/nrg3723</a>.
</div>
<div id="ref-european_parliament_regulation_2016" class="csl-entry" role="listitem">
European Parliament. 2016. <span>“Regulation on the Protection of
Natural Persons with Regard to the Processing of Personal Data and on
the Free Movement of Such Data.”</span>
</div>
<div id="ref-european_parliament_regulation_2017" class="csl-entry" role="listitem">
———. 2017. <span>“Regulation on Medical Devices.”</span>
</div>
<div id="ref-european_parliament_regulation_2024" class="csl-entry" role="listitem">
———. 2024. <span>“Regulation Laying down Harmonised Rules on Artificial
Intelligence.”</span>
</div>
<div id="ref-fang_deepprosite_2023" class="csl-entry" role="listitem">
Fang, Yitian, Yi Jiang, Leyi Wei, Qin Ma, Zhixiang Ren, Qianmu Yuan, and
Dong-Qing Wei. 2023. <span>“<span>DeepProSite</span>: Structure-Aware
Protein Binding Site Prediction Using <span>ESMFold</span> and
Pretrained Language Model.”</span> <em>Bioinformatics</em> 39 (12):
btad718. <a href="https://doi.org/10.1093/bioinformatics/btad718">https://doi.org/10.1093/bioinformatics/btad718</a>.
</div>
<div id="ref-fedus_switch_2022" class="csl-entry" role="listitem">
Fedus, William, Barret Zoph, and Noam Shazeer. 2022. <span>“Switch
<span>Transformers</span>: <span>Scaling</span> to <span>Trillion</span>
<span>Parameter</span> <span>Models</span> with <span>Simple</span> and
<span>Efficient</span> <span>Sparsity</span>.”</span> <em>Journal of
Machine Learning Research</em> 23 (120): 1–39.
</div>
<div id="ref-ferruz_protgpt2_2022" class="csl-entry" role="listitem">
Ferruz, Noelia, Steffen Schmidt, and Birte Höcker. 2022.
<span>“<span>ProtGPT2</span> Is a Deep Unsupervised Language Model for
Protein Design.”</span> <em>Nature Communications</em> 13 (1): 4348. <a href="https://doi.org/10.1038/s41467-022-32007-7">https://doi.org/10.1038/s41467-022-32007-7</a>.
</div>
<div id="ref-findlay_accurate_2018" class="csl-entry" role="listitem">
Findlay, Gregory M., Riza M. Daza, Beth Martin, Melissa D. Zhang, Anh P.
Leith, Molly Gasperini, Joseph D. Janizek, Xingfan Huang, Lea M.
Starita, and Jay Shendure. 2018. <span>“Accurate Classification of
<span>BRCA1</span> Variants with Saturation Genome Editing.”</span>
<em>Nature</em> 562 (7726): 217–22. <a href="https://doi.org/10.1038/s41586-018-0461-z">https://doi.org/10.1038/s41586-018-0461-z</a>.
</div>
<div id="ref-finn_model-agnostic_2017" class="csl-entry" role="listitem">
Finn, Chelsea, Pieter Abbeel, and Sergey Levine. 2017.
<span>“Model-<span>Agnostic</span>
<span>Meta</span>-<span>Learning</span> for <span>Fast</span>
<span>Adaptation</span> of <span>Deep</span>
<span>Networks</span>.”</span> In <em>Proceedings of the 34th
<span>International</span> <span>Conference</span> on
<span>Machine</span> <span>Learning</span></em>, 1126–35. PMLR.
</div>
<div id="ref-fokkema_lovd_2011" class="csl-entry" role="listitem">
Fokkema, Ivo F. A. C., Peter E. M. Taschner, Gerard C. P. Schaafsma, J.
Celli, Jeroen F. J. Laros, and Johan T. den Dunnen. 2011.
<span>“<span>LOVD</span> v.2.0: The Next Generation in Gene Variant
Databases.”</span> <em>Human Mutation</em> 32 (5): 557–63. <a href="https://doi.org/10.1002/humu.21438">https://doi.org/10.1002/humu.21438</a>.
</div>
<div id="ref-food_and_drug_administration_laboratory_2024" class="csl-entry" role="listitem">
Food and Drug Administration. 2024. <span>“Laboratory
<span>Developed</span> <span>Tests</span>: <span>Small</span>
<span>Entity</span> <span>Compliance</span> <span>Guide</span>;
<span>Guidance</span> for <span>Laboratory</span>
<span>Manufacturers</span> and <span>Food</span> and <span>Drug</span>
<span>Administration</span> <span>Staff</span>;
<span>Availability</span>.”</span>
</div>
<div id="ref-food_and_drug_administration_artificial_2025" class="csl-entry" role="listitem">
———. 2025. <span>“Artificial
<span>Intelligence</span>-<span>Enabled</span> <span>Medical</span>
<span>Devices</span>.”</span>
</div>
<div id="ref-fowler_deep_2014" class="csl-entry" role="listitem">
Fowler, Douglas M., and Stanley Fields. 2014. <span>“Deep Mutational
Scanning: A New Style of Protein Science.”</span> <em>Nature
Methods</em> 11 (8): 801–7. <a href="https://doi.org/10.1038/nmeth.3027">https://doi.org/10.1038/nmeth.3027</a>.
</div>
<div id="ref-frankish_gencode_2019" class="csl-entry" role="listitem">
Frankish, Adam, Mark Diekhans, Anne-Maud Ferreira, Rory Johnson, Irwin
Jungreis, Jane Loveland, Jonathan M Mudge, et al. 2019.
<span>“<span>GENCODE</span> Reference Annotation for the Human and Mouse
Genomes.”</span> <em>Nucleic Acids Research</em> 47 (D1): D766–73. <a href="https://doi.org/10.1093/nar/gky955">https://doi.org/10.1093/nar/gky955</a>.
</div>
<div id="ref-frazer_eve_2021" class="csl-entry" role="listitem">
Frazer, Jonathan, Pascal Notin, Mafalda Dias, Aidan Gomez, Joseph K.
Min, Kelly Brock, Yarin Gal, and Debora S. Marks. 2021.
<span>“[<span>EVE</span>] <span>Disease</span> Variant Prediction with
Deep Generative Models of Evolutionary Data.”</span> <em>Nature</em> 599
(7883): 91–95. <a href="https://doi.org/10.1038/s41586-021-04043-8">https://doi.org/10.1038/s41586-021-04043-8</a>.
</div>
<div id="ref-fudenberg_akita_2020" class="csl-entry" role="listitem">
Fudenberg, Geoff, David R. Kelley, and Katherine S. Pollard. 2020.
<span>“[<span>Akita</span>] <span>Predicting</span> <span>3D</span>
Genome Folding from <span>DNA</span> Sequence with
<span>Akita</span>.”</span> <em>Nature Methods</em> 17 (11): 1111–17. <a href="https://doi.org/10.1038/s41592-020-0958-x">https://doi.org/10.1038/s41592-020-0958-x</a>.
</div>
<div id="ref-gaedigk_pharmacogene_2017" class="csl-entry" role="listitem">
Gaedigk, Andrea, Magnus Ingelman-Sundberg, Neil A. Miller, J. Steven
Leeder, Michelle Whirl-Carrillo, Teri E. Klein, and the PharmVar
Steering Committee. 2017. <span>“The <span>Pharmacogene</span>
<span>Variation</span> (<span>PharmVar</span>) <span>Consortium</span>:
<span>Incorporation</span> of the <span>Human</span>
<span>Cytochrome</span> <span>P450</span> (<span>CYP</span>)
<span>Allele</span> <span>Nomenclature</span>
<span>Database</span>.”</span> <em>Clinical Pharmacology &amp;
Therapeutics</em> 103 (3): 399–401. <a href="https://doi.org/10.1002/cpt.910">https://doi.org/10.1002/cpt.910</a>.
</div>
<div id="ref-gal_dropout_2016" class="csl-entry" role="listitem">
Gal, Yarin, and Zoubin Ghahramani. 2016. <span>“Dropout as a
<span>Bayesian</span> <span>Approximation</span>:
<span>Representing</span> <span>Model</span> <span>Uncertainty</span> in
<span>Deep</span> <span>Learning</span>.”</span> In <em>Proceedings of
<span>The</span> 33rd <span>International</span> <span>Conference</span>
on <span>Machine</span> <span>Learning</span></em>, 1050–59. PMLR.
</div>
<div id="ref-gamazon_gene-based_2015" class="csl-entry" role="listitem">
Gamazon, Eric R., Heather E. Wheeler, Kaanan P. Shah, Sahar V.
Mozaffari, Keston Aquino-Michaels, Robert J. Carroll, Anne E. Eyler, et
al. 2015. <span>“A Gene-Based Association Method for Mapping Traits
Using Reference Transcriptome Data.”</span> <em>Nature Genetics</em> 47
(9): 1091–98. <a href="https://doi.org/10.1038/ng.3367">https://doi.org/10.1038/ng.3367</a>.
</div>
<div id="ref-ganin_domain-adversarial_2016" class="csl-entry" role="listitem">
Ganin, Yaroslav, Evgeniya Ustinova, Hana Ajakan, Pascal Germain, Hugo
Larochelle, François Laviolette, Mario March, and Victor Lempitsky.
2016. <span>“Domain-<span>Adversarial</span> <span>Training</span> of
<span>Neural</span> <span>Networks</span>.”</span> <em>Journal of
Machine Learning Research</em> 17 (59): 1–35.
</div>
<div id="ref-garrison_variation_2018" class="csl-entry" role="listitem">
Garrison, Erik, Jouni Sirén, Adam M. Novak, Glenn Hickey, Jordan M.
Eizenga, Eric T. Dawson, William Jones, et al. 2018. <span>“Variation
Graph Toolkit Improves Read Mapping by Representing Genetic Variation in
the Reference.”</span> <em>Nature Biotechnology</em> 36 (9): 875–79. <a href="https://doi.org/10.1038/nbt.4227">https://doi.org/10.1038/nbt.4227</a>.
</div>
<div id="ref-gasperini_genome-wide_2019" class="csl-entry" role="listitem">
Gasperini, Molly, Andrew J. Hill, José L. McFaline-Figueroa, Beth
Martin, Seungsoo Kim, Melissa D. Zhang, Dana Jackson, et al. 2019.
<span>“A <span>Genome</span>-Wide <span>Framework</span> for
<span>Mapping</span> <span>Gene</span> <span>Regulation</span> via
<span>Cellular</span> <span>Genetic</span> <span>Screens</span>.”</span>
<em>Cell</em> 176 (1): 377–390.e19. <a href="https://doi.org/10.1016/j.cell.2018.11.029">https://doi.org/10.1016/j.cell.2018.11.029</a>.
</div>
<div id="ref-ge_polygenic_2019" class="csl-entry" role="listitem">
Ge, Tian, Chia-Yen Chen, Yang Ni, Yen-Chen Anne Feng, and Jordan W.
Smoller. 2019. <span>“Polygenic Prediction via <span>Bayesian</span>
Regression and Continuous Shrinkage Priors.”</span> <em>Nature
Communications</em> 10 (1): 1776. <a href="https://doi.org/10.1038/s41467-019-09718-5">https://doi.org/10.1038/s41467-019-09718-5</a>.
</div>
<div id="ref-gebru_datasheets_2021" class="csl-entry" role="listitem">
Gebru, Timnit, Jamie Morgenstern, Briana Vecchione, Jennifer Wortman
Vaughan, Hanna Wallach, Hal Daumé III, and Kate Crawford. 2021.
<span>“Datasheets for Datasets.”</span> <em>Commun. ACM</em> 64 (12):
86–92. <a href="https://doi.org/10.1145/3458723">https://doi.org/10.1145/3458723</a>.
</div>
<div id="ref-georgantas_delphi_2024" class="csl-entry" role="listitem">
Georgantas, Costa, Zoltán Kutalik, and Jonas Richiardi. 2024.
<span>“Delphi: <span>A</span> <span>Deep</span>-Learning
<span>Method</span> for <span>Polygenic</span> <span>Risk</span>
<span>Prediction</span>.”</span> medRxiv. <a href="https://doi.org/10.1101/2024.04.19.24306079">https://doi.org/10.1101/2024.04.19.24306079</a>.
</div>
<div id="ref-giambartolomei_bayesian_2014" class="csl-entry" role="listitem">
Giambartolomei, Claudia, Damjan Vukcevic, Eric E. Schadt, Lude Franke,
Aroon D. Hingorani, Chris Wallace, and Vincent Plagnol. 2014.
<span>“Bayesian <span>Test</span> for <span>Colocalisation</span>
Between <span>Pairs</span> of <span>Genetic</span>
<span>Association</span> <span>Studies</span> <span>Using</span>
<span>Summary</span> <span>Statistics</span>.”</span> <em>PLOS
Genetics</em> 10 (5): e1004383. <a href="https://doi.org/10.1371/journal.pgen.1004383">https://doi.org/10.1371/journal.pgen.1004383</a>.
</div>
<div id="ref-gong_integrating_2025" class="csl-entry" role="listitem">
Gong, Li, Clarissa J Klein, Kelly E Caudle, Ann M Moyer, Stuart A Scott,
Michelle Whirl-Carrillo, Teri E Klein, ClinGen Pharmacogenomics Working
Group (PGxWG), and on behalf of the. 2025. <span>“Integrating
<span>Pharmacogenomics</span> into the <span>Broader</span>
<span>Construct</span> of <span>Genomic</span> <span>Medicine</span>:
<span>Efforts</span> by the <span>ClinGen</span>
<span>Pharmacogenomics</span> <span>Working</span> <span>Group</span>
(<span>PGxWG</span>).”</span> <em>Clinical Chemistry</em> 71 (1): 36–44.
<a href="https://doi.org/10.1093/clinchem/hvae181">https://doi.org/10.1093/clinchem/hvae181</a>.
</div>
<div id="ref-goodwin_coming_2016" class="csl-entry" role="listitem">
Goodwin, Sara, John D. McPherson, and W. Richard McCombie. 2016.
<span>“Coming of Age: Ten Years of Next-Generation Sequencing
Technologies.”</span> <em>Nature Reviews Genetics</em> 17 (6): 333–51.
<a href="https://doi.org/10.1038/nrg.2016.49">https://doi.org/10.1038/nrg.2016.49</a>.
</div>
<div id="ref-granger_investigating_1969" class="csl-entry" role="listitem">
Granger, C. W. J. 1969. <span>“Investigating <span>Causal</span>
<span>Relations</span> by <span>Econometric</span> <span>Models</span>
and <span>Cross</span>-Spectral <span>Methods</span>.”</span>
<em>Econometrica</em> 37 (3).
</div>
<div id="ref-grantham_amino_1974" class="csl-entry" role="listitem">
Grantham, R. 1974. <span>“Amino <span>Acid</span>
<span>Difference</span> <span>Formula</span> to <span>Help</span>
<span>Explain</span> <span>Protein</span>
<span>Evolution</span>.”</span> <em>Science</em> 185 (4154): 862–64. <a href="https://doi.org/10.1126/science.185.4154.862">https://doi.org/10.1126/science.185.4154.862</a>.
</div>
<div id="ref-gresova_genomic_2023" class="csl-entry" role="listitem">
Grešová, Katarína, Vlastimil Martinek, David Čechák, Petr Šimeček, and
Panagiotis Alexiou. 2023. <span>“Genomic Benchmarks: A Collection of
Datasets for Genomic Sequence Classification.”</span> <em>BMC Genomic
Data</em> 24 (1): 25. <a href="https://doi.org/10.1186/s12863-023-01123-8">https://doi.org/10.1186/s12863-023-01123-8</a>.
</div>
<div id="ref-gtex_consortium_the_gtex_2020" class="csl-entry" role="listitem">
GTEx Consortium, The. 2020. <span>“The <span>GTEx</span>
<span>Consortium</span> Atlas of Genetic Regulatory Effects Across Human
Tissues.”</span> <em>Science</em> 369 (6509): 1318–30. <a href="https://doi.org/10.1126/science.aaz1776">https://doi.org/10.1126/science.aaz1776</a>.
</div>
<div id="ref-gu_mamba_2024" class="csl-entry" role="listitem">
Gu, Albert, and Tri Dao. 2024. <span>“Mamba:
<span>Linear</span>-<span>Time</span> <span>Sequence</span>
<span>Modeling</span> with <span>Selective</span> <span>State</span>
<span>Spaces</span>.”</span> In.
</div>
<div id="ref-gu_efficiently_2022" class="csl-entry" role="listitem">
Gu, Albert, Karan Goel, and Christopher Ré. 2022. <span>“Efficiently
<span>Modeling</span> <span>Long</span> <span>Sequences</span> with
<span>Structured</span> <span>State</span> <span>Spaces</span>.”</span>
arXiv. <a href="https://doi.org/10.48550/arXiv.2111.00396">https://doi.org/10.48550/arXiv.2111.00396</a>.
</div>
<div id="ref-gudbjartsson_sequence_2015" class="csl-entry" role="listitem">
Gudbjartsson, Daniel F., Patrick Sulem, Hannes Helgason, Arnaldur
Gylfason, Sigurjon A. Gudjonsson, Florian Zink, Asmundur Oddson, et al.
2015. <span>“Sequence Variants from Whole Genome Sequencing a Large
Group of <span>Icelanders</span>.”</span> <em>Scientific Data</em> 2
(1): 150011. <a href="https://doi.org/10.1038/sdata.2015.11">https://doi.org/10.1038/sdata.2015.11</a>.
</div>
<div id="ref-guo_calibration_2017" class="csl-entry" role="listitem">
Guo, Chuan, Geoff Pleiss, Yu Sun, and Kilian Q. Weinberger. 2017.
<span>“On <span>Calibration</span> of <span>Modern</span>
<span>Neural</span> <span>Networks</span>.”</span> In <em>Proceedings of
the 34th <span>International</span> <span>Conference</span> on
<span>Machine</span> <span>Learning</span></em>, 1321–30. PMLR.
</div>
<div id="ref-gusev_integrative_2016" class="csl-entry" role="listitem">
Gusev, Alexander, Arthur Ko, Huwenbo Shi, Gaurav Bhatia, Wonil Chung,
Brenda W. J. H. Penninx, Rick Jansen, et al. 2016. <span>“Integrative
Approaches for Large-Scale Transcriptome-Wide Association
Studies.”</span> <em>Nature Genetics</em> 48 (3): 245–52. <a href="https://doi.org/10.1038/ng.3506">https://doi.org/10.1038/ng.3506</a>.
</div>
<div id="ref-gymrek_identifying_2013" class="csl-entry" role="listitem">
Gymrek, Melissa, Amy L. McGuire, David Golan, Eran Halperin, and Yaniv
Erlich. 2013. <span>“Identifying <span>Personal</span>
<span>Genomes</span> by <span>Surname</span>
<span>Inference</span>.”</span> <em>Science</em> 339 (6117): 321–24. <a href="https://doi.org/10.1126/science.1229566">https://doi.org/10.1126/science.1229566</a>.
</div>
<div id="ref-hao_large-scale_2024" class="csl-entry" role="listitem">
Hao, Minsheng, Jing Gong, Xin Zeng, Chiming Liu, Yucheng Guo, Xingyi
Cheng, Taifeng Wang, Jianzhu Ma, Xuegong Zhang, and Le Song. 2024.
<span>“Large-Scale Foundation Model on Single-Cell
Transcriptomics.”</span> <em>Nature Methods</em> 21 (8): 1481–91. <a href="https://doi.org/10.1038/s41592-024-02305-7">https://doi.org/10.1038/s41592-024-02305-7</a>.
</div>
<div id="ref-henikoff_amino_1992" class="csl-entry" role="listitem">
Henikoff, S, and J G Henikoff. 1992. <span>“Amino Acid Substitution
Matrices from Protein Blocks.”</span> <em>Proceedings of the National
Academy of Sciences</em> 89 (22): 10915–19. <a href="https://doi.org/10.1073/pnas.89.22.10915">https://doi.org/10.1073/pnas.89.22.10915</a>.
</div>
<div id="ref-hilker_heritability_2018" class="csl-entry" role="listitem">
Hilker, Rikke, Dorte Helenius, Birgitte Fagerlund, Axel Skytthe, Kaare
Christensen, Thomas M. Werge, Merete Nordentoft, and Birte Glenthøj.
2018. <span>“Heritability of <span>Schizophrenia</span> and
<span>Schizophrenia</span> <span>Spectrum</span> <span>Based</span> on
the <span>Nationwide</span> <span>Danish</span> <span>Twin</span>
<span>Register</span>.”</span> <em>Biological Psychiatry</em>, Novel
<span>Mechanisms</span> in <span>Schizophrenia</span>
<span>Pathophysiology</span>, 83 (6): 492–98. <a href="https://doi.org/10.1016/j.biopsych.2017.08.017">https://doi.org/10.1016/j.biopsych.2017.08.017</a>.
</div>
<div id="ref-hoang_locality-aware_2025" class="csl-entry" role="listitem">
Hoang, Minh, and Mona Singh. 2025. <span>“Locality-Aware Pooling
Enhances Protein Language Model Performance Across Varied
Applications.”</span> <em>Bioinformatics</em> 41 (Supplement_1):
i217–26. <a href="https://doi.org/10.1093/bioinformatics/btaf178">https://doi.org/10.1093/bioinformatics/btaf178</a>.
</div>
<div id="ref-hochreiter_long_1997" class="csl-entry" role="listitem">
Hochreiter, Sepp, and Jürgen Schmidhuber. 1997. <span>“Long
<span>Short</span>-<span>Term</span> <span>Memory</span>.”</span>
<em>Neural Computation</em> 9 (8): 1735–80. <a href="https://doi.org/10.1162/neco.1997.9.8.1735">https://doi.org/10.1162/neco.1997.9.8.1735</a>.
</div>
<div id="ref-hoffmann_training_2022" class="csl-entry" role="listitem">
Hoffmann, Jordan, Sebastian Borgeaud, Arthur Mensch, Elena Buchatskaya,
Trevor Cai, Eliza Rutherford, Diego de Las Casas, et al. 2022.
<span>“Training <span>Compute</span>-<span>Optimal</span>
<span>Large</span> <span>Language</span> <span>Models</span>.”</span>
arXiv. <a href="https://doi.org/10.48550/arXiv.2203.15556">https://doi.org/10.48550/arXiv.2203.15556</a>.
</div>
<div id="ref-homer_resolving_2008" class="csl-entry" role="listitem">
Homer, Nils, Szabolcs Szelinger, Margot Redman, David Duggan, Waibhav
Tembe, Jill Muehling, John V. Pearson, Dietrich A. Stephan, Stanley F.
Nelson, and David W. Craig. 2008. <span>“Resolving
<span>Individuals</span> <span>Contributing</span> <span>Trace</span>
<span>Amounts</span> of <span>DNA</span> to <span>Highly</span>
<span>Complex</span> <span>Mixtures</span> <span>Using</span>
<span>High</span>-<span>Density</span> <span>SNP</span>
<span>Genotyping</span> <span>Microarrays</span>.”</span> <em>PLOS
Genetics</em> 4 (8): e1000167. <a href="https://doi.org/10.1371/journal.pgen.1000167">https://doi.org/10.1371/journal.pgen.1000167</a>.
</div>
<div id="ref-hormozdiari_identifying_2014" class="csl-entry" role="listitem">
Hormozdiari, Farhad, Emrah Kostem, Eun Yong kang, Bogdan Pasaniuc, and
Eleazar Eskin. 2014. <span>“Identifying Causal Variants at Loci with
Multiple Signals of Association.”</span> In <em>Proceedings of the 5th
<span>ACM</span> <span>Conference</span> on <span>Bioinformatics</span>,
<span>Computational</span> <span>Biology</span>, and <span>Health</span>
<span>Informatics</span></em>, 610–11. <span>BCB</span> ’14. New York,
NY, USA: Association for Computing Machinery. <a href="https://doi.org/10.1145/2649387.2660800">https://doi.org/10.1145/2649387.2660800</a>.
</div>
<div id="ref-horvath_dna_2013" class="csl-entry" role="listitem">
Horvath, Steve. 2013. <span>“<span>DNA</span> Methylation Age of Human
Tissues and Cell Types.”</span> <em>Genome Biology</em> 14 (10): R115.
<a href="https://doi.org/10.1186/gb-2013-14-10-r115">https://doi.org/10.1186/gb-2013-14-10-r115</a>.
</div>
<div id="ref-howard_universal_2018" class="csl-entry" role="listitem">
Howard, Jeremy, and Sebastian Ruder. 2018. <span>“Universal
<span>Language</span> <span>Model</span> <span>Fine</span>-Tuning for
<span>Text</span> <span>Classification</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.1801.06146">https://doi.org/10.48550/arXiv.1801.06146</a>.
</div>
<div id="ref-hsieh_resolving_2020" class="csl-entry" role="listitem">
Hsieh, Tsung-Han S., Claudia Cattoglio, Elena Slobodyanyuk, Anders S.
Hansen, Oliver J. Rando, Robert Tjian, and Xavier Darzacq. 2020.
<span>“Resolving the <span>3D</span> <span>Landscape</span> of
<span>Transcription</span>-<span>Linked</span> <span>Mammalian</span>
<span>Chromatin</span> <span>Folding</span>.”</span> <em>Molecular
Cell</em> 78 (3): 539–553.e8. <a href="https://doi.org/10.1016/j.molcel.2020.03.002">https://doi.org/10.1016/j.molcel.2020.03.002</a>.
</div>
<div id="ref-hsu_learning_2022" class="csl-entry" role="listitem">
Hsu, Chloe, Robert Verkuil, Jason Liu, Zeming Lin, Brian Hie, Tom Sercu,
Adam Lerer, and Alexander Rives. 2022. <span>“Learning Inverse Folding
from Millions of Predicted Structures.”</span> In <em>Proceedings of the
39th <span>International</span> <span>Conference</span> on
<span>Machine</span> <span>Learning</span></em>, 8946–70. PMLR.
</div>
<div id="ref-hu_lora_2021" class="csl-entry" role="listitem">
Hu, Edward J., Yelong Shen, Phillip Wallis, Zeyuan Allen-Zhu, Yuanzhi
Li, Shean Wang, and Weizhu Chen. 2021. <span>“<span>LoRA</span>:
<span>Low</span>-<span>Rank</span> <span>Adaptation</span> of
<span>Large</span> <span>Language</span> <span>Models</span>.”</span>
arXiv. <a href="https://doi.org/10.48550/arXiv.2106.09685">https://doi.org/10.48550/arXiv.2106.09685</a>.
</div>
<div id="ref-huang_coming_2016" class="csl-entry" role="listitem">
Huang, Po-Ssu, Scott E. Boyken, and David Baker. 2016. <span>“The Coming
of Age of de Novo Protein Design.”</span> <em>Nature</em> 537 (7620):
320–27. <a href="https://doi.org/10.1038/nature19946">https://doi.org/10.1038/nature19946</a>.
</div>
<div id="ref-hubisz_exploring_2014" class="csl-entry" role="listitem">
Hubisz, Melissa J, and Katherine S Pollard. 2014. <span>“Exploring the
Genesis and Functions of <span>Human</span> <span>Accelerated</span>
<span>Regions</span> Sheds Light on Their Role in Human
Evolution.”</span> <em>Current Opinion in Genetics &amp;
Development</em>, Genetics of human evolution, 29 (December): 15–21. <a href="https://doi.org/10.1016/j.gde.2014.07.005">https://doi.org/10.1016/j.gde.2014.07.005</a>.
</div>
<div id="ref-huynh-thu_inferring_2010" class="csl-entry" role="listitem">
Huynh-Thu, Vân Anh, Alexandre Irrthum, Louis Wehenkel, and Pierre
Geurts. 2010. <span>“Inferring <span>Regulatory</span>
<span>Networks</span> from <span>Expression</span> <span>Data</span>
<span>Using</span> <span>Tree</span>-<span>Based</span>
<span>Methods</span>.”</span> <em>PLOS ONE</em> 5 (9): e12776. <a href="https://doi.org/10.1371/journal.pone.0012776">https://doi.org/10.1371/journal.pone.0012776</a>.
</div>
<div id="ref-ingelman-sundberg_genetic_2004" class="csl-entry" role="listitem">
Ingelman-Sundberg, M. 2004. <span>“Genetic Polymorphisms of Cytochrome
<span>P450</span> <span>2D6</span> (<span>CYP2D6</span>): Clinical
Consequences, Evolutionary Aspects and Functional Diversity.”</span>
<em>The Pharmacogenomics Journal</em> 5 (1): 6–13. <a href="https://doi.org/10.1038/sj.tpj.6500285">https://doi.org/10.1038/sj.tpj.6500285</a>.
</div>
<div id="ref-international_medical_device_regulators_forum_software_2014" class="csl-entry" role="listitem">
International Medical Device Regulators Forum. 2014. <span>“Software as
a <span>Medical</span> <span>Device</span>: <span>Possible</span>
<span>Framework</span> for <span>Risk</span> <span>Categorization</span>
and <span>Corresponding</span> <span>Considerations</span>.”</span>
</div>
<div id="ref-international_medical_device_regulators_forum_software_2017" class="csl-entry" role="listitem">
———. 2017. <span>“Software as a <span>Medical</span> <span>Device</span>
(<span>SaMD</span>): <span>Clinical</span>
<span>Evaluation</span>.”</span>
</div>
<div id="ref-ioannidis_revel_2016" class="csl-entry" role="listitem">
Ioannidis, Nilah M., Joseph H. Rothstein, Vikas Pejaver, Sumit Middha,
Shannon K. McDonnell, Saurabh Baheti, Anthony Musolf, et al. 2016.
<span>“<span>REVEL</span>: <span>An</span> <span>Ensemble</span>
<span>Method</span> for <span>Predicting</span> the
<span>Pathogenicity</span> of <span>Rare</span> <span>Missense</span>
<span>Variants</span>.”</span> <em>The American Journal of Human
Genetics</em> 99 (4): 877–85. <a href="https://doi.org/10.1016/j.ajhg.2016.08.016">https://doi.org/10.1016/j.ajhg.2016.08.016</a>.
</div>
<div id="ref-ionita-laza_spectral_2016" class="csl-entry" role="listitem">
Ionita-Laza, Iuliana, Kenneth McCallum, Bin Xu, and Joseph D. Buxbaum.
2016. <span>“A Spectral Approach Integrating Functional Genomic
Annotations for Coding and Noncoding Variants.”</span> <em>Nature
Genetics</em> 48 (2): 214–20. <a href="https://doi.org/10.1038/ng.3477">https://doi.org/10.1038/ng.3477</a>.
</div>
<div id="ref-jagadeesh_m-cap_2016" class="csl-entry" role="listitem">
Jagadeesh, Karthik A., Aaron M. Wenger, Mark J. Berger, Harendra Guturu,
Peter D. Stenson, David N. Cooper, Jonathan A. Bernstein, and Gill
Bejerano. 2016. <span>“M-<span>CAP</span> Eliminates a Majority of
Variants of Uncertain Significance in Clinical Exomes at High
Sensitivity.”</span> <em>Nature Genetics</em> 48 (12): 1581–86. <a href="https://doi.org/10.1038/ng.3703">https://doi.org/10.1038/ng.3703</a>.
</div>
<div id="ref-jaganathan_spliceai_2019" class="csl-entry" role="listitem">
Jaganathan, Kishore, Sofia Kyriazopoulou Panagiotopoulou, Jeremy F.
McRae, Siavash Fazel Darbandi, David Knowles, Yang I. Li, Jack A.
Kosmicki, et al. 2019. <span>“[<span>SpliceAI</span>]
<span>Predicting</span> <span>Splicing</span> from <span>Primary</span>
<span>Sequence</span> with <span>Deep</span>
<span>Learning</span>.”</span> <em>Cell</em> 176 (3): 535–548.e24. <a href="https://doi.org/10.1016/j.cell.2018.12.015">https://doi.org/10.1016/j.cell.2018.12.015</a>.
</div>
<div id="ref-jaganathan_predicting_2019" class="csl-entry" role="listitem">
Jaganathan, Kishore, Sofia Kyriazopoulou Panagiotopoulou, Jeremy F.
McRae, Siavash Fazel Darbber, Helen V. Firth, Neil Banerjee, Richard
Pointon, et al. 2019. <span>“Predicting <span>Splicing</span> from
<span>Primary</span> <span>Sequence</span> with <span>Deep</span>
<span>Learning</span>.”</span> <em>Cell</em> 176 (3): 535–548.e24. <a href="https://doi.org/10.1016/j.cell.2018.12.015">https://doi.org/10.1016/j.cell.2018.12.015</a>.
</div>
<div id="ref-jain_attention_2019" class="csl-entry" role="listitem">
Jain, Sarthak, and Byron C. Wallace. 2019. <span>“Attention Is Not
<span>Explanation</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.1902.10186">https://doi.org/10.48550/arXiv.1902.10186</a>.
</div>
<div id="ref-jawahar_what_2019" class="csl-entry" role="listitem">
Jawahar, Ganesh, Benoît Sagot, and Djamé Seddah. 2019. <span>“What Does
<span>BERT</span> Learn about the Structure of Language?”</span> In
<em><span>ACL</span> 2019 - 57th <span>Annual</span>
<span>Meeting</span> of the <span>Association</span> for
<span>Computational</span> <span>Linguistics</span></em>. Florence,
Italy.
</div>
<div id="ref-ji_dnabert_2021" class="csl-entry" role="listitem">
Ji, Yanrong, Zhihan Zhou, Han Liu, and Ramana V Davuluri. 2021.
<span>“<span>DNABERT</span>: Pre-Trained <span>Bidirectional</span>
<span>Encoder</span> <span>Representations</span> from
<span>Transformers</span> Model for <span>DNA</span>-Language in
Genome.”</span> <em>Bioinformatics</em> 37 (15): 2112–20. <a href="https://doi.org/10.1093/bioinformatics/btab083">https://doi.org/10.1093/bioinformatics/btab083</a>.
</div>
<div id="ref-jiang_long-read-based_2020" class="csl-entry" role="listitem">
Jiang, Tao, Yongzhuang Liu, Yue Jiang, Junyi Li, Yan Gao, Zhe Cui,
Yadong Liu, Bo Liu, and Yadong Wang. 2020. <span>“Long-Read-Based Human
Genomic Structural Variation Detection with <span class="nocase">cuteSV</span>.”</span> <em>Genome Biology</em> 21 (1):
189. <a href="https://doi.org/10.1186/s13059-020-02107-y">https://doi.org/10.1186/s13059-020-02107-y</a>.
</div>
<div id="ref-jumper_alphafold2_2021" class="csl-entry" role="listitem">
Jumper, John, Richard Evans, Alexander Pritzel, Tim Green, Michael
Figurnov, Olaf Ronneberger, Kathryn Tunyasuvunakool, et al. 2021.
<span>“[<span>AlphaFold2</span>] <span>Highly</span> Accurate Protein
Structure Prediction with <span>AlphaFold</span>.”</span>
<em>Nature</em> 596 (7873): 583–89. <a href="https://doi.org/10.1038/s41586-021-03819-2">https://doi.org/10.1038/s41586-021-03819-2</a>.
</div>
<div id="ref-jurenaite_setquence_2024" class="csl-entry" role="listitem">
Jurenaite, Neringa, Daniel León-Periñán, Veronika Donath, Sunna Torge,
and René Jäkel. 2024. <span>“<span>SetQuence</span> &amp;
<span>SetOmic</span>: <span>Deep</span> Set Transformers for Whole
Genome and Exome Tumour Analysis.”</span> <em>BioSystems</em> 235
(January): 105095. <a href="https://doi.org/10.1016/j.biosystems.2023.105095">https://doi.org/10.1016/j.biosystems.2023.105095</a>.
</div>
<div id="ref-kagda_data_2025" class="csl-entry" role="listitem">
Kagda, Meenakshi S., Bonita Lam, Casey Litton, Corinn Small, Cricket A.
Sloan, Emma Spragins, Forrest Tanaka, et al. 2025. <span>“Data
Navigation on the <span>ENCODE</span> Portal.”</span> <em>Nature
Communications</em> 16 (1): 9592. <a href="https://doi.org/10.1038/s41467-025-64343-9">https://doi.org/10.1038/s41467-025-64343-9</a>.
</div>
<div id="ref-kalvari_rfam_2021" class="csl-entry" role="listitem">
Kalvari, Ioanna, Eric P. Nawrocki, Nancy Ontiveros-Palacios, Joanna
Argasinska, Kevin Lamkiewicz, Manja Marz, Sam Griffiths-Jones, et al.
2021. <span>“Rfam 14: Expanded Coverage of Metagenomic, Viral and <span class="nocase">microRNA</span> Families.”</span> <em>Nucleic Acids
Research</em> 49 (D1): D192–200. <a href="https://doi.org/10.1093/nar/gkaa1047">https://doi.org/10.1093/nar/gkaa1047</a>.
</div>
<div id="ref-kaplan_scaling_2020" class="csl-entry" role="listitem">
Kaplan, Jared, Sam McCandlish, Tom Henighan, Tom B. Brown, Benjamin
Chess, Rewon Child, Scott Gray, Alec Radford, Jeffrey Wu, and Dario
Amodei. 2020. <span>“Scaling <span>Laws</span> for <span>Neural</span>
<span>Language</span> <span>Models</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2001.08361">https://doi.org/10.48550/arXiv.2001.08361</a>.
</div>
<div id="ref-karczewski_mutational_2020" class="csl-entry" role="listitem">
Karczewski, Konrad J., Laurent C. Francioli, Grace Tiao, Beryl B.
Cummings, Jessica Alföldi, Qingbo Wang, Ryan L. Collins, et al. 2020.
<span>“The Mutational Constraint Spectrum Quantified from Variation in
141,456 Humans.”</span> <em>Nature</em> 581 (7809): 434–43. <a href="https://doi.org/10.1038/s41586-020-2308-7">https://doi.org/10.1038/s41586-020-2308-7</a>.
</div>
<div id="ref-katzman_deepsurv_2018" class="csl-entry" role="listitem">
Katzman, Jared L., Uri Shaham, Alexander Cloninger, Jonathan Bates,
Tingting Jiang, and Yuval Kluger. 2018. <span>“<span>DeepSurv</span>:
Personalized Treatment Recommender System Using a <span>Cox</span>
Proportional Hazards Deep Neural Network.”</span> <em>BMC Medical
Research Methodology</em> 18 (1): 24. <a href="https://doi.org/10.1186/s12874-018-0482-1">https://doi.org/10.1186/s12874-018-0482-1</a>.
</div>
<div id="ref-kaye_dynamic_2014" class="csl-entry" role="listitem">
Kaye, Jane, Edgar A. Whitley, David Lund, Michael Morrison, Harriet
Teare, and Karen Melham. 2014. <span>“Dynamic Consent: A Patient
Interface for Twenty-First Century Research Networks.”</span>
<em>European Journal of Human Genetics</em> 23 (2): 141–46. <a href="https://doi.org/10.1038/ejhg.2014.71">https://doi.org/10.1038/ejhg.2014.71</a>.
</div>
<div id="ref-kelley_basenji2_2020" class="csl-entry" role="listitem">
Kelley, David R. 2020. <span>“[<span>Basenji2</span>]
<span>Cross</span>-Species Regulatory Sequence Activity
Prediction.”</span> <em>PLOS Computational Biology</em> 16 (7):
e1008050. <a href="https://doi.org/10.1371/journal.pcbi.1008050">https://doi.org/10.1371/journal.pcbi.1008050</a>.
</div>
<div id="ref-kelley_basenji_2018" class="csl-entry" role="listitem">
Kelley, David R., Yakir A. Reshef, Maxwell Bileschi, David Belanger,
Cory Y. McLean, and Jasper Snoek. 2018. <span>“[<span>Basenji</span>]
<span>Sequential</span> Regulatory Activity Prediction Across
Chromosomes with Convolutional Neural Networks.”</span> <em>Genome
Research</em> 28 (5): 739–50. <a href="https://doi.org/10.1101/gr.227819.117">https://doi.org/10.1101/gr.227819.117</a>.
</div>
<div id="ref-kelley_basset_2016" class="csl-entry" role="listitem">
Kelley, David R., Jasper Snoek, and John L. Rinn. 2016. <span>“Basset:
Learning the Regulatory Code of the Accessible Genome with Deep
Convolutional Neural Networks.”</span> <em>Genome Research</em> 26 (7):
990–99. <a href="https://doi.org/10.1101/gr.200535.115">https://doi.org/10.1101/gr.200535.115</a>.
</div>
<div id="ref-khera_genetics_2017" class="csl-entry" role="listitem">
Khera, Amit V., and Sekar Kathiresan. 2017. <span>“Genetics of Coronary
Artery Disease: Discovery, Biology and Clinical Translation.”</span>
<em>Nature Reviews Genetics</em> 18 (6): 331–44. <a href="https://doi.org/10.1038/nrg.2016.160">https://doi.org/10.1038/nrg.2016.160</a>.
</div>
<div id="ref-kichaev_improved_2017" class="csl-entry" role="listitem">
Kichaev, Gleb, Megan Roytman, Ruth Johnson, Eleazar Eskin, Sara
Lindström, Peter Kraft, and Bogdan Pasaniuc. 2017. <span>“Improved
Methods for Multi-Trait Fine Mapping of Pleiotropic Risk Loci.”</span>
<em>Bioinformatics</em> 33 (2): 248–55. <a href="https://doi.org/10.1093/bioinformatics/btw615">https://doi.org/10.1093/bioinformatics/btw615</a>.
</div>
<div id="ref-kircher_general_2014" class="csl-entry" role="listitem">
Kircher, Martin, Daniela M. Witten, Preti Jain, Brian J. O’Roak, Gregory
M. Cooper, and Jay Shendure. 2014. <span>“A General Framework for
Estimating the Relative Pathogenicity of Human Genetic Variants.”</span>
<em>Nature Genetics</em> 46 (3): 310–15. <a href="https://doi.org/10.1038/ng.2892">https://doi.org/10.1038/ng.2892</a>.
</div>
<div id="ref-kiciman_causal_2024" class="csl-entry" role="listitem">
Kıcıman, Emre, Robert Ness, Amit Sharma, and Chenhao Tan. 2024.
<span>“Causal <span>Reasoning</span> and <span>Large</span>
<span>Language</span> <span>Models</span>: <span>Opening</span> a
<span>New</span> <span>Frontier</span> for
<span>Causality</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2305.00050">https://doi.org/10.48550/arXiv.2305.00050</a>.
</div>
<div id="ref-kong_rate_2012" class="csl-entry" role="listitem">
Kong, Augustine, Michael L. Frigge, Gisli Masson, Soren Besenbacher,
Patrick Sulem, Gisli Magnusson, Sigurjon A. Gudjonsson, et al. 2012.
<span>“Rate of de Novo Mutations and the Importance of Father’s Age to
Disease Risk.”</span> <em>Nature</em> 488 (7412): 471–75. <a href="https://doi.org/10.1038/nature11396">https://doi.org/10.1038/nature11396</a>.
</div>
<div id="ref-krusche_best_2019" class="csl-entry" role="listitem">
Krusche, Peter, Len Trigg, Paul C. Boutros, Christopher E. Mason,
Francisco M. De La Vega, Benjamin L. Moore, Mar Gonzalez-Porta, et al.
2019. <span>“Best <span>Practices</span> for <span>Benchmarking</span>
<span>Germline</span> <span>Small</span> <span>Variant</span>
<span>Calls</span> in <span>Human</span> <span>Genomes</span>.”</span>
<em>Nature Biotechnology</em> 37 (5): 555–60. <a href="https://doi.org/10.1038/s41587-019-0054-x">https://doi.org/10.1038/s41587-019-0054-x</a>.
</div>
<div id="ref-kulmanov_protein_2024" class="csl-entry" role="listitem">
Kulmanov, Maxat, Francisco J. Guzmán-Vega, Paula Duek Roggli, Lydie
Lane, Stefan T. Arold, and Robert Hoehndorf. 2024. <span>“Protein
Function Prediction as Approximate Semantic Entailment.”</span>
<em>Nature Machine Intelligence</em> 6 (2): 220–28. <a href="https://doi.org/10.1038/s42256-024-00795-w">https://doi.org/10.1038/s42256-024-00795-w</a>.
</div>
<div id="ref-kundaje_integrative_2015" class="csl-entry" role="listitem">
Kundaje, Anshul, Wouter Meuleman, Jason Ernst, Misha Bilenky, Angela
Yen, Alireza Heravi-Moussavi, Pouya Kheradpour, et al. 2015.
<span>“Integrative Analysis of 111 Reference Human Epigenomes.”</span>
<em>Nature</em> 518 (7539): 317–30. <a href="https://doi.org/10.1038/nature14248">https://doi.org/10.1038/nature14248</a>.
</div>
<div id="ref-kurki_finngen_2023" class="csl-entry" role="listitem">
Kurki, Mitja I., Juha Karjalainen, Priit Palta, Timo P. Sipilä, Kati
Kristiansson, Kati M. Donner, Mary P. Reeve, et al. 2023.
<span>“<span>FinnGen</span> Provides Genetic Insights from a
Well-Phenotyped Isolated Population.”</span> <em>Nature</em> 613 (7944):
508–18. <a href="https://doi.org/10.1038/s41586-022-05473-8">https://doi.org/10.1038/s41586-022-05473-8</a>.
</div>
<div id="ref-la_manno_rna_2018" class="csl-entry" role="listitem">
La Manno, Gioele, Ruslan Soldatov, Amit Zeisel, Emelie Braun, Hannah
Hochgerner, Viktor Petukhov, Katja Lidschreiber, et al. 2018.
<span>“<span>RNA</span> Velocity of Single Cells.”</span>
<em>Nature</em> 560 (7719): 494–98. <a href="https://doi.org/10.1038/s41586-018-0414-6">https://doi.org/10.1038/s41586-018-0414-6</a>.
</div>
<div id="ref-lambert_towards_2019" class="csl-entry" role="listitem">
Lambert, Samuel A, Gad Abraham, and Michael Inouye. 2019. <span>“Towards
Clinical Utility of Polygenic Risk Scores.”</span> <em>Human Molecular
Genetics</em> 28 (R2): R133–42. <a href="https://doi.org/10.1093/hmg/ddz187">https://doi.org/10.1093/hmg/ddz187</a>.
</div>
<div id="ref-lambert_polygenic_2021" class="csl-entry" role="listitem">
Lambert, Samuel A., Laurent Gil, Simon Jupp, Scott C. Ritchie, Yu Xu,
Annalisa Buniello, Aoife McMahon, et al. 2021. <span>“The
<span>Polygenic</span> <span>Score</span> <span>Catalog</span> as an
Open Database for Reproducibility and Systematic Evaluation.”</span>
<em>Nature Genetics</em> 53 (4): 420–25. <a href="https://doi.org/10.1038/s41588-021-00783-5">https://doi.org/10.1038/s41588-021-00783-5</a>.
</div>
<div id="ref-landrum_clinvar_2018" class="csl-entry" role="listitem">
Landrum, Melissa J, Jennifer M Lee, Mark Benson, Garth R Brown, Chen
Chao, Shanmuga Chitipiralla, Baoshan Gu, et al. 2018.
<span>“<span>ClinVar</span>: Improving Access to Variant Interpretations
and Supporting Evidence.”</span> <em>Nucleic Acids Research</em> 46
(D1): D1062–67. <a href="https://doi.org/10.1093/nar/gkx1153">https://doi.org/10.1093/nar/gkx1153</a>.
</div>
<div id="ref-larson_liquid_2017" class="csl-entry" role="listitem">
Larson, Adam G., Daniel Elnatan, Madeline M. Keenen, Michael J. Trnka,
Jonathan B. Johnston, Alma L. Burlingame, David A. Agard, Sy Redding,
and Geeta J. Narlikar. 2017. <span>“Liquid Droplet Formation by
<span>HP1α</span> Suggests a Role for Phase Separation in
Heterochromatin.”</span> <em>Nature</em> 547 (7662): 236–40. <a href="https://doi.org/10.1038/nature22822">https://doi.org/10.1038/nature22822</a>.
</div>
<div id="ref-lawlor_mendelian_2008" class="csl-entry" role="listitem">
Lawlor, Debbie A., Roger M. Harbord, Jonathan A. C. Sterne, Nic Timpson,
and George Davey Smith. 2008. <span>“Mendelian Randomization:
<span>Using</span> Genes as Instruments for Making Causal Inferences in
Epidemiology.”</span> <em>Statistics in Medicine</em> 27 (8): 1133–63.
<a href="https://doi.org/10.1002/sim.3034">https://doi.org/10.1002/sim.3034</a>.
</div>
<div id="ref-lee_g2pt_2025" class="csl-entry" role="listitem">
Lee, Ingoo, Zachary S. Wallace, Yuqi Wang, Sungjoon Park, Hojung Nam,
Amit R. Majithia, and Trey Ideker. 2025. <span>“[<span>G2PT</span>]
<span>A</span> Genotype-Phenotype Transformer to Assess and Explain
Polygenic Risk.”</span> bioRxiv. <a href="https://doi.org/10.1101/2024.10.23.619940">https://doi.org/10.1101/2024.10.23.619940</a>.
</div>
<div id="ref-li_aligning_2013" class="csl-entry" role="listitem">
Li, Heng. 2013. <span>“Aligning Sequence Reads, Clone Sequences and
Assembly Contigs with <span>BWA</span>-<span>MEM</span>.”</span> arXiv.
<a href="https://doi.org/10.48550/arXiv.1303.3997">https://doi.org/10.48550/arXiv.1303.3997</a>.
</div>
<div id="ref-li_towards_2014" class="csl-entry" role="listitem">
———. 2014. <span>“Towards <span>Better</span> <span>Understanding</span>
of <span>Artifacts</span> in <span>Variant</span> <span>Calling</span>
from <span>High</span>-<span>Coverage</span>
<span>Samples</span>.”</span> <em>Bioinformatics</em> 30 (20): 2843–51.
<a href="https://doi.org/10.1093/bioinformatics/btu356">https://doi.org/10.1093/bioinformatics/btu356</a>.
</div>
<div id="ref-li_minimap2_2018" class="csl-entry" role="listitem">
———. 2018. <span>“Minimap2: Pairwise Alignment for Nucleotide
Sequences.”</span> <em>Bioinformatics</em> 34 (18): 3094–3100. <a href="https://doi.org/10.1093/bioinformatics/bty191">https://doi.org/10.1093/bioinformatics/bty191</a>.
</div>
<div id="ref-li_codonbert_2023" class="csl-entry" role="listitem">
Li, Sizhen, Saeed Moayedpour, Ruijiang Li, Michael Bailey, Saleh Riahi,
Milad Miladi, Jacob Miner, et al. 2023. <span>“<span>CodonBERT</span>:
<span>Large</span> <span>Language</span> <span>Models</span> for <span class="nocase">mRNA</span> Design and Optimization.”</span> bioRxiv. <a href="https://doi.org/10.1101/2023.09.09.556981">https://doi.org/10.1101/2023.09.09.556981</a>.
</div>
<div id="ref-li_mogcn_2022" class="csl-entry" role="listitem">
Li, Xiao, Jie Ma, Ling Leng, Mingfei Han, Mansheng Li, Fuchu He, and
Yunping Zhu. 2022. <span>“<span>MoGCN</span>: <span>A</span>
<span>Multi</span>-<span>Omics</span> <span>Integration</span>
<span>Method</span> <span>Based</span> on <span>Graph</span>
<span>Convolutional</span> <span>Network</span> for <span>Cancer</span>
<span>Subtype</span> <span>Analysis</span>.”</span> <em>Frontiers in
Genetics</em> 13 (February). <a href="https://doi.org/10.3389/fgene.2022.806842">https://doi.org/10.3389/fgene.2022.806842</a>.
</div>
<div id="ref-li_omni-dna_2025" class="csl-entry" role="listitem">
Li, Zehui, Vallijah Subasri, Yifei Shen, Dongsheng Li, Yiren Zhao,
Guy-Bart Stan, and Caihua Shan. 2025. <span>“Omni-<span>DNA</span>:
<span>A</span> <span>Unified</span> <span>Genomic</span>
<span>Foundation</span> <span>Model</span> for
<span>Cross</span>-<span>Modal</span> and
<span>Multi</span>-<span>Task</span> <span>Learning</span>.”</span>
arXiv. <a href="https://doi.org/10.48550/arXiv.2502.03499">https://doi.org/10.48550/arXiv.2502.03499</a>.
</div>
<div id="ref-liao_draft_2023" class="csl-entry" role="listitem">
Liao, Wen-Wei, Mobin Asri, Jana Ebler, Daniel Doerr, Marina Haukness,
Glenn Hickey, Shuangjia Lu, et al. 2023. <span>“A Draft Human Pangenome
Reference.”</span> <em>Nature</em> 617 (7960): 312–24. <a href="https://doi.org/10.1038/s41586-023-05896-x">https://doi.org/10.1038/s41586-023-05896-x</a>.
</div>
<div id="ref-lieberman-aiden_comprehensive_2009" class="csl-entry" role="listitem">
Lieberman-Aiden, Erez, Nynke L. van Berkum, Louise Williams, Noam
Kaplan, Peter J. Sabo, Michael O. Dorschner, Job Dekker, et al. 2009.
<span>“Comprehensive Mapping of Long-Range Interactions Reveals Folding
Principles of the Human Genome.”</span> <em>Science</em> 326 (5950):
289–93. <a href="https://doi.org/10.1126/science.1181369">https://doi.org/10.1126/science.1181369</a>.
</div>
<div id="ref-lin_focal_2020" class="csl-entry" role="listitem">
Lin, Tsung-Yi, Priya Goyal, Ross Girshick, Kaiming He, and Piotr Dollár.
2020. <span>“Focal <span>Loss</span> for <span>Dense</span>
<span>Object</span> <span>Detection</span>.”</span> <em>IEEE
Transactions on Pattern Analysis and Machine Intelligence</em> 42 (02):
318–27. <a href="https://doi.org/10.1109/TPAMI.2018.2858826">https://doi.org/10.1109/TPAMI.2018.2858826</a>.
</div>
<div id="ref-lin_esm-2_2022" class="csl-entry" role="listitem">
Lin, Zeming, Halil Akin, Roshan Rao, Brian Hie, Zhongkai Zhu, Wenting
Lu, Allan dos Santos Costa, et al. 2022. <span>“[<span>ESM</span>-2]
<span>Language</span> Models of Protein Sequences at the Scale of
Evolution Enable Accurate Structure Prediction.”</span> bioRxiv. <a href="https://doi.org/10.1101/2022.07.20.500902">https://doi.org/10.1101/2022.07.20.500902</a>.
</div>
<div id="ref-linder_borzoi_2025" class="csl-entry" role="listitem">
Linder, Johannes, Divyanshi Srivastava, Han Yuan, Vikram Agarwal, and
David R. Kelley. 2025. <span>“[<span>Borzoi</span>]
<span>Predicting</span> <span>RNA</span>-Seq Coverage from
<span>DNA</span> Sequence as a Unifying Model of Gene
Regulation.”</span> <em>Nature Genetics</em> 57 (4): 949–61. <a href="https://doi.org/10.1038/s41588-024-02053-6">https://doi.org/10.1038/s41588-024-02053-6</a>.
</div>
<div id="ref-lipsitch_negative_2010" class="csl-entry" role="listitem">
Lipsitch, Marc, Eric Tchetgen Tchetgen, and Ted Cohen. 2010.
<span>“Negative <span>Controls</span>: <span>A</span> <span>Tool</span>
for <span>Detecting</span> <span>Confounding</span> and
<span>Bias</span> in <span>Observational</span>
<span>Studies</span>.”</span> <em>Epidemiology</em> 21 (3): 383. <a href="https://doi.org/10.1097/EDE.0b013e3181d61eeb">https://doi.org/10.1097/EDE.0b013e3181d61eeb</a>.
</div>
<div id="ref-liu_life-code_2025" class="csl-entry" role="listitem">
Liu, Zicheng, Siyuan Li, Zhiyuan Chen, Fang Wu, Chang Yu, Qirong Yang,
Yucheng Guo, Yujie Yang, Xiaoming Zhang, and Stan Z. Li. 2025.
<span>“Life-<span>Code</span>: <span>Central</span> <span>Dogma</span>
<span>Modeling</span> with <span>Multi</span>-<span>Omics</span>
<span>Sequence</span> <span>Unification</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2502.07299">https://doi.org/10.48550/arXiv.2502.07299</a>.
</div>
<div id="ref-logsdon_long-read_2020" class="csl-entry" role="listitem">
Logsdon, Glennis A., Mitchell R. Vollger, and Evan E. Eichler. 2020.
<span>“Long-Read Human Genome Sequencing and Its Applications.”</span>
<em>Nature Reviews Genetics</em> 21 (10): 597–614. <a href="https://doi.org/10.1038/s41576-020-0236-x">https://doi.org/10.1038/s41576-020-0236-x</a>.
</div>
<div id="ref-loh_reference-based_2016" class="csl-entry" role="listitem">
Loh, Po-Ru, Petr Danecek, Pier Francesco Palamara, Christian
Fuchsberger, Yakir A Reshef, Hilary K Finucane, Sebastian Schoenherr, et
al. 2016. <span>“Reference-Based Phasing Using the
<span>Haplotype</span> <span>Reference</span> <span>Consortium</span>
Panel.”</span> <em>Nature Genetics</em> 48 (11): 1443–48. <a href="https://doi.org/10.1038/ng.3679">https://doi.org/10.1038/ng.3679</a>.
</div>
<div id="ref-loshchilov_decoupled_2019" class="csl-entry" role="listitem">
Loshchilov, Ilya, and Frank Hutter. 2019. <span>“Decoupled
<span>Weight</span> <span>Decay</span>
<span>Regularization</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.1711.05101">https://doi.org/10.48550/arXiv.1711.05101</a>.
</div>
<div id="ref-lupianez_disruptions_2015" class="csl-entry" role="listitem">
Lupiáñez, Darío G., Katerina Kraft, Verena Heinrich, Peter Krawitz,
Francesco Brancati, Eva Klopocki, Denise Horn, et al. 2015.
<span>“Disruptions of <span>Topological</span> <span>Chromatin</span>
<span>Domains</span> <span>Cause</span> <span>Pathogenic</span>
<span>Rewiring</span> of <span>Gene</span>-<span>Enhancer</span>
<span>Interactions</span>.”</span> <em>Cell</em> 161 (5): 1012–25. <a href="https://doi.org/10.1016/j.cell.2015.04.004">https://doi.org/10.1016/j.cell.2015.04.004</a>.
</div>
<div id="ref-lynch_activating_2004" class="csl-entry" role="listitem">
Lynch, Thomas J., Daphne W. Bell, Raffaella Sordella, Sarada
Gurubhagavatula, Ross A. Okimoto, Brian W. Brannigan, Patricia L.
Harris, et al. 2004. <span>“Activating <span>Mutations</span> in the
<span>Epidermal</span> <span>Growth</span> <span>Factor</span>
<span>Receptor</span> <span>Underlying</span>
<span>Responsiveness</span> of
<span>Non</span>–<span>Small</span>-<span>Cell</span> <span>Lung</span>
<span>Cancer</span> to <span>Gefitinib</span>.”</span> <em>New England
Journal of Medicine</em> 350 (21): 2129–39. <a href="https://doi.org/10.1056/NEJMoa040938">https://doi.org/10.1056/NEJMoa040938</a>.
</div>
<div id="ref-madani_large_2023" class="csl-entry" role="listitem">
Madani, Ali, Ben Krause, Eric R. Greene, Subu Subramanian, Benjamin P.
Mohr, James M. Holton, Jose Luis Olmos, et al. 2023. <span>“Large
Language Models Generate Functional Protein Sequences Across Diverse
Families.”</span> <em>Nature Biotechnology</em> 41 (8): 1099–1106. <a href="https://doi.org/10.1038/s41587-022-01618-2">https://doi.org/10.1038/s41587-022-01618-2</a>.
</div>
<div id="ref-mallal_hla-b5701_2008" class="csl-entry" role="listitem">
Mallal, Simon, Elizabeth Phillips, Giampiero Carosi, Jean-Michel Molina,
Cassy Workman, Janez Tomažič, Eva Jägel-Guedes, et al. 2008.
<span>“<span>HLA</span>-<span>B</span>*5701 <span>Screening</span> for
<span>Hypersensitivity</span> to <span>Abacavir</span>.”</span> <em>New
England Journal of Medicine</em> 358 (6): 568–79. <a href="https://doi.org/10.1056/NEJMoa0706135">https://doi.org/10.1056/NEJMoa0706135</a>.
</div>
<div id="ref-maller_bayesian_2012" class="csl-entry" role="listitem">
Maller, Julian B., Gilean McVean, Jake Byrnes, Damjan Vukcevic, Kimmo
Palin, Zhan Su, Joanna M. M. Howson, et al. 2012. <span>“Bayesian
Refinement of Association Signals for 14 Loci in 3 Common
Diseases.”</span> <em>Nature Genetics</em> 44 (12): 1294–1301. <a href="https://doi.org/10.1038/ng.2435">https://doi.org/10.1038/ng.2435</a>.
</div>
<div id="ref-manolio_finding_2009" class="csl-entry" role="listitem">
Manolio, Teri A., Francis S. Collins, Nancy J. Cox, David B. Goldstein,
Lucia A. Hindorff, David J. Hunter, Mark I. McCarthy, et al. 2009.
<span>“Finding the Missing Heritability of Complex Diseases.”</span>
<em>Nature</em> 461 (7265): 747–53. <a href="https://doi.org/10.1038/nature08494">https://doi.org/10.1038/nature08494</a>.
</div>
<div id="ref-manzo_comparative_2025" class="csl-entry" role="listitem">
Manzo, Gaetano, Kathryn Borkowski, and Ivan Ovcharenko. 2025.
<span>“Comparative <span>Analysis</span> of <span>Deep</span>
<span>Learning</span> <span>Models</span> for <span>Predicting</span>
<span>Causative</span> <span>Regulatory</span>
<span>Variants</span>.”</span> <em>bioRxiv: The Preprint Server for
Biology</em>, June, 2025.05.19.654920. <a href="https://doi.org/10.1101/2025.05.19.654920">https://doi.org/10.1101/2025.05.19.654920</a>.
</div>
<div id="ref-marees_gwas_2018" class="csl-entry" role="listitem">
Marees, Andries T., Hilde de Kluiver, Sven Stringer, Florence Vorspan,
Emmanuel Curis, Cynthia Marie-Claire, and Eske M. Derks. 2018.
<span>“[<span>GWAS</span>] <span>A</span> Tutorial on Conducting
Genome-Wide Association Studies: <span>Quality</span> Control and
Statistical Analysis.”</span> <em>International Journal of Methods in
Psychiatric Research</em> 27 (2): e1608. <a href="https://doi.org/10.1002/mpr.1608">https://doi.org/10.1002/mpr.1608</a>.
</div>
<div id="ref-marin_bend_2024" class="csl-entry" role="listitem">
Marin, Frederikke Isa, Felix Teufel, Marc Horlacher, Dennis Madsen,
Dennis Pultz, Ole Winther, and Wouter Boomsma. 2024.
<span>“<span>BEND</span>: <span>Benchmarking</span> <span>DNA</span>
<span>Language</span> <span>Models</span> on Biologically Meaningful
Tasks.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2311.12570">https://doi.org/10.48550/arXiv.2311.12570</a>.
</div>
<div id="ref-marquez-luna_multiethnic_2017" class="csl-entry" role="listitem">
Márquez-Luna, Carla, Po-Ru Loh, South Asian Type 2 Diabetes (SAT2D)
Consortium, The SIGMA Type 2 Diabetes Consortium, and Alkes L. Price.
2017. <span>“Multiethnic Polygenic Risk Scores Improve Risk Prediction
in Diverse Populations.”</span> <em>Genetic Epidemiology</em> 41 (8):
811–23. <a href="https://doi.org/10.1002/gepi.22083">https://doi.org/10.1002/gepi.22083</a>.
</div>
<div id="ref-martin_clinical_2019" class="csl-entry" role="listitem">
Martin, Alicia R., Masahiro Kanai, Yoichiro Kamatani, Yukinori Okada,
Benjamin M. Neale, and Mark J. Daly. 2019. <span>“Clinical Use of
Current Polygenic Risk Scores May Exacerbate Health Disparities.”</span>
<em>Nature Genetics</em> 51 (4): 584–91. <a href="https://doi.org/10.1038/s41588-019-0379-x">https://doi.org/10.1038/s41588-019-0379-x</a>.
</div>
<div id="ref-mavaddat_polygenic_2019" class="csl-entry" role="listitem">
Mavaddat, Nasim, Kyriaki Michailidou, Joe Dennis, Michael Lush, Laura
Fachal, Andrew Lee, Jonathan P. Tyrer, et al. 2019. <span>“Polygenic
<span>Risk</span> <span>Scores</span> for <span>Prediction</span> of
<span>Breast</span> <span>Cancer</span> and <span>Breast</span>
<span>Cancer</span> <span>Subtypes</span>.”</span> <em>The American
Journal of Human Genetics</em> 104 (1): 21–34. <a href="https://doi.org/10.1016/j.ajhg.2018.11.002">https://doi.org/10.1016/j.ajhg.2018.11.002</a>.
</div>
<div id="ref-mccloskey_catastrophic_1989" class="csl-entry" role="listitem">
McCloskey, Michael, and Neal Cohen. 1989. <span>“Catastrophic
<span>Interference</span> in <span>Connectionist</span>
<span>Networks</span>: <span>The</span> <span>Sequential</span>
<span>Learning</span> <span>Problem</span>.”</span> <em>Psychology of
Learning and Motivation</em> 24 (January): 109–65. <a href="https://doi.org/10.1016/S0079-7421(08)60536-8">https://doi.org/10.1016/S0079-7421(08)60536-8</a>.
</div>
<div id="ref-medvedev_biotoken_2025" class="csl-entry" role="listitem">
Medvedev, Aleksandr, Karthik Viswanathan, Praveenkumar Kanithi, Kirill
Vishniakov, Prateek Munjal, Clément Christophe, Marco AF Pimentel,
Ronnie Rajan, and Shadab Khan. 2025. <span>“<span>BioToken</span> and
<span>BioFM</span> – <span>Biologically</span>-<span>Informed</span>
<span>Tokenization</span> <span>Enables</span> <span>Accurate</span> and
<span>Efficient</span> <span>Genomic</span> <span>Foundation</span>
<span>Models</span>.”</span> bioRxiv. <a href="https://doi.org/10.1101/2025.03.27.645711">https://doi.org/10.1101/2025.03.27.645711</a>.
</div>
<div id="ref-meier_esm-1v_2021" class="csl-entry" role="listitem">
Meier, Joshua, Roshan Rao, Robert Verkuil, Jason Liu, Tom Sercu, and
Alexander Rives. 2021. <span>“[<span>ESM</span>-1v]
<span>Language</span> Models Enable Zero-Shot Prediction of the Effects
of Mutations on Protein Function.”</span> bioRxiv. <a href="https://doi.org/10.1101/2021.07.09.450648">https://doi.org/10.1101/2021.07.09.450648</a>.
</div>
<div id="ref-mitchell_model_2019" class="csl-entry" role="listitem">
Mitchell, Margaret, Simone Wu, Andrew Zaldivar, Parker Barnes, Lucy
Vasserman, Ben Hutchinson, Elena Spitzer, Inioluwa Deborah Raji, and
Timnit Gebru. 2019. <span>“Model <span>Cards</span> for
<span>Model</span> <span>Reporting</span>.”</span> In <em>Proceedings of
the <span>Conference</span> on <span>Fairness</span>,
<span>Accountability</span>, and <span>Transparency</span></em>, 220–29.
<span>FAT</span>* ’19. New York, NY, USA: Association for Computing
Machinery. <a href="https://doi.org/10.1145/3287560.3287596">https://doi.org/10.1145/3287560.3287596</a>.
</div>
<div id="ref-morales_joint_2022" class="csl-entry" role="listitem">
Morales, Joannella, Shashikant Pujar, Jane E. Loveland, Alex Astashyn,
Ruth Bennett, Andrew Berry, Eric Cox, et al. 2022. <span>“A Joint
<span>NCBI</span> and <span>EMBL</span>-<span>EBI</span> Transcript Set
for Clinical Genomics and Research.”</span> <em>Nature</em> 604 (7905):
310–15. <a href="https://doi.org/10.1038/s41586-022-04558-8">https://doi.org/10.1038/s41586-022-04558-8</a>.
</div>
<div id="ref-morcos_direct-coupling_2011" class="csl-entry" role="listitem">
Morcos, Faruck, Andrea Pagnani, Bryan Lunt, Arianna Bertolino, Debora S.
Marks, Chris Sander, Riccardo Zecchina, José N. Onuchic, Terence Hwa,
and Martin Weigt. 2011. <span>“Direct-Coupling Analysis of Residue
Coevolution Captures Native Contacts Across Many Protein
Families.”</span> <em>Proceedings of the National Academy of
Sciences</em> 108 (49): E1293–1301. <a href="https://doi.org/10.1073/pnas.1111471108">https://doi.org/10.1073/pnas.1111471108</a>.
</div>
<div id="ref-mountjoy_open_2021" class="csl-entry" role="listitem">
Mountjoy, Edward, Ellen M. Schmidt, Miguel Carmona, Jeremy
Schwartzentruber, Gareth Peat, Alfredo Miranda, Luca Fumis, et al. 2021.
<span>“An Open Approach to Systematically Prioritize Causal Variants and
Genes at All Published Human <span>GWAS</span> Trait-Associated
Loci.”</span> <em>Nature Genetics</em> 53 (11): 1527–33. <a href="https://doi.org/10.1038/s41588-021-00945-5">https://doi.org/10.1038/s41588-021-00945-5</a>.
</div>
<div id="ref-mukherjee_embedgem_2024" class="csl-entry" role="listitem">
Mukherjee, Sumit, Zachary R. McCaw, Jingwen Pei, Anna Merkoulovitch, Tom
Soare, Raghav Tandon, David Amar, et al. 2024.
<span>“<span>EmbedGEM</span>: A Framework to Evaluate the Utility of
Embeddings for Genetic Discovery.”</span> <em>Bioinformatics
Advances</em> 4 (1). <a href="https://doi.org/10.1093/bioadv/vbae135">https://doi.org/10.1093/bioadv/vbae135</a>.
</div>
<div id="ref-naderializadeh_aggregating_2025" class="csl-entry" role="listitem">
NaderiAlizadeh, Navid, and Rohit Singh. 2025. <span>“Aggregating
Residue-Level Protein Language Model Embeddings with Optimal
Transport.”</span> <em>Bioinformatics Advances</em> 5 (1): vbaf060. <a href="https://doi.org/10.1093/bioadv/vbaf060">https://doi.org/10.1093/bioadv/vbaf060</a>.
</div>
<div id="ref-naghipourfar_cdsfm_2024" class="csl-entry" role="listitem">
Naghipourfar, Mohsen, Siyu Chen, Mathew K. Howard, Christian B.
Macdonald, Ali Saberi, Timo Hagen, Mohammad R. K. Mofrad, Willow
Coyote-Maestas, and Hani Goodarzi. 2024. <span>“[<span class="nocase">cdsFM</span> - <span>EnCodon</span>/<span>DeCodon</span>]
<span>A</span> <span>Suite</span> of <span>Foundation</span>
<span>Models</span> <span>Captures</span> the <span>Contextual</span>
<span>Interplay</span> <span>Between</span> <span>Codons</span>.”</span>
bioRxiv. <a href="https://doi.org/10.1101/2024.10.10.617568">https://doi.org/10.1101/2024.10.10.617568</a>.
</div>
<div id="ref-nagpal_deep_2021" class="csl-entry" role="listitem">
Nagpal, Chirag, Xinyu Li, and Artur Dubrawski. 2021. <span>“Deep
<span>Survival</span> <span>Machines</span>: <span>Fully</span>
<span>Parametric</span> <span>Survival</span> <span>Regression</span>
and <span>Representation</span> <span>Learning</span> for
<span>Censored</span> <span>Data</span> <span>With</span>
<span>Competing</span> <span>Risks</span>.”</span> <em>IEEE Journal of
Biomedical and Health Informatics</em> 25 (8): 3163–75. <a href="https://doi.org/10.1109/JBHI.2021.3052441">https://doi.org/10.1109/JBHI.2021.3052441</a>.
</div>
<div id="ref-ng_sift_2003" class="csl-entry" role="listitem">
Ng, Pauline C., and Steven Henikoff. 2003. <span>“<span>SIFT</span>:
<span>Predicting</span> Amino Acid Changes That Affect Protein
Function.”</span> <em>Nucleic Acids Research</em> 31 (13): 3812–14. <a href="https://doi.org/10.1093/nar/gkg509">https://doi.org/10.1093/nar/gkg509</a>.
</div>
<div id="ref-nguengang_wakap_estimating_2019" class="csl-entry" role="listitem">
Nguengang Wakap, Stéphanie, Deborah M. Lambert, Annie Olry, Charlotte
Rodwell, Charlotte Gueydan, Valérie Lanneau, Daniel Murphy, Yann Le Cam,
and Ana Rath. 2019. <span>“Estimating Cumulative Point Prevalence of
Rare Diseases: Analysis of the <span>Orphanet</span> Database.”</span>
<em>European Journal of Human Genetics</em> 28 (2): 165–73. <a href="https://doi.org/10.1038/s41431-019-0508-0">https://doi.org/10.1038/s41431-019-0508-0</a>.
</div>
<div id="ref-nguyen_sequence_2024" class="csl-entry" role="listitem">
Nguyen, Eric, Michael Poli, Matthew G. Durrant, Brian Kang, Dhruva
Katrekar, David B. Li, Liam J. Bartie, et al. 2024. <span>“Sequence
Modeling and Design from Molecular to Genome Scale with
<span>Evo</span>.”</span> <em>Science</em> 386 (6723): eado9336. <a href="https://doi.org/10.1126/science.ado9336">https://doi.org/10.1126/science.ado9336</a>.
</div>
<div id="ref-nguyen_hyenadna_2023" class="csl-entry" role="listitem">
Nguyen, Eric, Michael Poli, Marjan Faizi, Armin Thomas, Callum
Birch-Sykes, Michael Wornow, Aman Patel, et al. 2023.
<span>“<span>HyenaDNA</span>: <span>Long</span>-<span>Range</span>
<span>Genomic</span> <span>Sequence</span> <span>Modeling</span> at
<span>Single</span> <span>Nucleotide</span>
<span>Resolution</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2306.15794">https://doi.org/10.48550/arXiv.2306.15794</a>.
</div>
<div id="ref-nielsen_genotype_2011" class="csl-entry" role="listitem">
Nielsen, Rasmus, Joshua S. Paul, Anders Albrechtsen, and Yun S. Song.
2011. <span>“Genotype and <span>SNP</span> Calling from Next-Generation
Sequencing Data.”</span> <em>Nature Reviews. Genetics</em> 12 (6):
443–51. <a href="https://doi.org/10.1038/nrg2986">https://doi.org/10.1038/nrg2986</a>.
</div>
<div id="ref-nofziger_pharmvar_2019" class="csl-entry" role="listitem">
Nofziger, Charity, Amy J. Turner, Katrin Sangkuhl, Michelle
Whirl-Carrillo, José A. G. Agúndez, John L. Black, Henry M.
Dunnenberger, et al. 2019. <span>“<span>PharmVar</span>
<span>GeneFocus</span>: <span>CYP2D6</span>.”</span> <em>Clinical
Pharmacology &amp; Therapeutics</em> 107 (1): 154–70. <a href="https://doi.org/10.1002/cpt.1643">https://doi.org/10.1002/cpt.1643</a>.
</div>
<div id="ref-notin_proteingym_2023" class="csl-entry" role="listitem">
Notin, Pascal, Aaron Kollasch, Daniel Ritter, Lood van Niekerk,
Steffanie Paul, Han Spinner, Nathan Rollins, et al. 2023.
<span>“<span>ProteinGym</span>: <span>Large</span>-<span>Scale</span>
<span>Benchmarks</span> for <span>Protein</span> <span>Fitness</span>
<span>Prediction</span> and <span>Design</span>.”</span> <em>Advances in
Neural Information Processing Systems</em> 36 (December): 64331–79.
</div>
<div id="ref-nurk_complete_2022" class="csl-entry" role="listitem">
Nurk, Sergey, Sergey Koren, Arang Rhie, Mikko Rautiainen, Andrey V.
Bzikadze, Alla Mikheenko, Mitchell R. Vollger, et al. 2022. <span>“The
Complete Sequence of a Human Genome.”</span> <em>Science</em> 376
(6588): 44–53. <a href="https://doi.org/10.1126/science.abj6987">https://doi.org/10.1126/science.abj6987</a>.
</div>
<div id="ref-oconnell_general_2014" class="csl-entry" role="listitem">
O’Connell, Jared, Deepti Gurdasani, Olivier Delaneau, Nicola Pirastu,
Sheila Ulivi, Massimiliano Cocca, Michela Traglia, et al. 2014. <span>“A
<span>General</span> <span>Approach</span> for <span>Haplotype</span>
<span>Phasing</span> Across the <span>Full</span> <span>Spectrum</span>
of <span>Relatedness</span>.”</span> <em>PLOS Genetics</em> 10 (4):
e1004234. <a href="https://doi.org/10.1371/journal.pgen.1004234">https://doi.org/10.1371/journal.pgen.1004234</a>.
</div>
<div id="ref-oleary_reference_2016" class="csl-entry" role="listitem">
O’Leary, Nuala A., Mathew W. Wright, J. Rodney Brister, Stacy Ciufo,
Diana Haddad, Rich McVeigh, Bhanu Rajput, et al. 2016. <span>“Reference
Sequence (<span>RefSeq</span>) Database at <span>NCBI</span>: Current
Status, Taxonomic Expansion, and Functional Annotation.”</span>
<em>Nucleic Acids Research</em> 44 (D1): D733–45. <a href="https://doi.org/10.1093/nar/gkv1189">https://doi.org/10.1093/nar/gkv1189</a>.
</div>
<div id="ref-oord_representation_2019" class="csl-entry" role="listitem">
Oord, Aaron van den, Yazhe Li, and Oriol Vinyals. 2019.
<span>“Representation <span>Learning</span> with
<span>Contrastive</span> <span>Predictive</span>
<span>Coding</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.1807.03748">https://doi.org/10.48550/arXiv.1807.03748</a>.
</div>
<div id="ref-orenbuch_popeve_2025" class="csl-entry" role="listitem">
Orenbuch, Rose, Courtney A. Shearer, Aaron W. Kollasch, Aviv D. Spinner,
Thomas Hopf, Lood van Niekerk, Dinko Franceschi, Mafalda Dias, Jonathan
Frazer, and Debora S. Marks. 2025. <span>“[<span class="nocase">popEVE</span>] <span>Proteome</span>-Wide Model for Human
Disease Genetics.”</span> <em>Nature Genetics</em>, November, 1–10. <a href="https://doi.org/10.1038/s41588-025-02400-1">https://doi.org/10.1038/s41588-025-02400-1</a>.
</div>
<div id="ref-noauthor_pacificbiosciencespbsv_2025" class="csl-entry" role="listitem">
<span>“<span>PacificBiosciences</span>/Pbsv.”</span> 2025. PacBio.
</div>
<div id="ref-parasuraman_complacency_2010" class="csl-entry" role="listitem">
Parasuraman, Raja, and Dietrich H. Manzey. 2010. <span>“Complacency and
<span>Bias</span> in <span>Human</span> <span>Use</span> of
<span>Automation</span>: <span>An</span> <span>Attentional</span>
<span>Integration</span>.”</span> <em>Human Factors</em> 52 (3):
381–410. <a href="https://doi.org/10.1177/0018720810376055">https://doi.org/10.1177/0018720810376055</a>.
</div>
<div id="ref-patterson_population_2006" class="csl-entry" role="listitem">
Patterson, Nick, Alkes L. Price, and David Reich. 2006.
<span>“Population <span>Structure</span> and
<span>Eigenanalysis</span>.”</span> <em>PLOS Genetics</em> 2 (12): e190.
<a href="https://doi.org/10.1371/journal.pgen.0020190">https://doi.org/10.1371/journal.pgen.0020190</a>.
</div>
<div id="ref-peer_estimation_2008" class="csl-entry" role="listitem">
Pe’er, Itsik, Roman Yelensky, David Altshuler, and Mark J. Daly. 2008.
<span>“Estimation of the Multiple Testing Burden for Genomewide
Association Studies of Nearly All Common Variants.”</span> <em>Genetic
Epidemiology</em> 32 (4): 381–85. <a href="https://doi.org/10.1002/gepi.20303">https://doi.org/10.1002/gepi.20303</a>.
</div>
<div id="ref-pearce_transcriptformer_2025" class="csl-entry" role="listitem">
Pearce, James D., Sara E. Simmonds, Gita Mahmoudabadi, Lakshmi Krishnan,
Giovanni Palla, Ana-Maria Istrate, Alexander Tarashansky, et al. 2025.
<span>“[<span>TranscriptFormer</span>]
<span>Cross</span>-<span>Species</span> <span>Generative</span>
<span>Cell</span> <span>Atlas</span> <span>Across</span> 1.5
<span>Billion</span> <span>Years</span> of <span>Evolution</span>:
<span>The</span> <span>TranscriptFormer</span> <span>Single</span>-Cell
<span>Model</span>.”</span> bioRxiv. <a href="https://doi.org/10.1101/2025.04.25.650731">https://doi.org/10.1101/2025.04.25.650731</a>.
</div>
<div id="ref-pearl_causality_2009" class="csl-entry" role="listitem">
Pearl, Judea. 2009. <em>Causality</em>. Cambridge University Press.
</div>
<div id="ref-pearl_book_2018" class="csl-entry" role="listitem">
Pearl, Judea, and Dana Mackenzie. 2018. <em>The <span>Book</span> of
<span>Why</span></em>. Hachette Book Group.
</div>
<div id="ref-pejaver_calibration_2022" class="csl-entry" role="listitem">
Pejaver, Vikas, Alicia B. Byrne, Bing-Jian Feng, Kymberleigh A. Pagel,
Sean D. Mooney, Rachel Karchin, Anne O’Donnell-Luria, et al. 2022.
<span>“Calibration of Computational Tools for Missense Variant
Pathogenicity Classification and <span>ClinGen</span> Recommendations
for <span>PP3</span>/<span>BP4</span> Criteria.”</span> <em>American
Journal of Human Genetics</em> 109 (12): 2163–77. <a href="https://doi.org/10.1016/j.ajhg.2022.10.013">https://doi.org/10.1016/j.ajhg.2022.10.013</a>.
</div>
<div id="ref-peters_deep_2018" class="csl-entry" role="listitem">
Peters, Matthew E., Mark Neumann, Mohit Iyyer, Matt Gardner, Christopher
Clark, Kenton Lee, and Luke Zettlemoyer. 2018. <span>“Deep
<span>Contextualized</span> <span>Word</span>
<span>Representations</span>.”</span> In <em>Proceedings of the 2018
<span>Conference</span> of the <span>North</span> <span>American</span>
<span>Chapter</span> of the <span>Association</span> for
<span>Computational</span> <span>Linguistics</span>: <span>Human</span>
<span>Language</span> <span>Technologies</span>, <span>Volume</span> 1
(<span>Long</span> <span>Papers</span>)</em>, edited by Marilyn Walker,
Heng Ji, and Amanda Stent, 2227–37. New Orleans, Louisiana: Association
for Computational Linguistics. <a href="https://doi.org/10.18653/v1/N18-1202">https://doi.org/10.18653/v1/N18-1202</a>.
</div>
<div id="ref-poli_hyena_2023" class="csl-entry" role="listitem">
Poli, Michael, Stefano Massaroli, Eric Nguyen, Daniel Y. Fu, Tri Dao,
Stephen Baccus, Yoshua Bengio, Stefano Ermon, and Christopher Re. 2023.
<span>“Hyena <span>Hierarchy</span>: <span>Towards</span>
<span>Larger</span> <span>Convolutional</span> <span>Language</span>
<span>Models</span>.”</span> In <em>Proceedings of the 40th
<span>International</span> <span>Conference</span> on
<span>Machine</span> <span>Learning</span></em>, 28043–78. PMLR.
</div>
<div id="ref-poplin_deepvariant_2018" class="csl-entry" role="listitem">
Poplin, Ryan, Pi-Chuan Chang, David Alexander, Scott Schwartz, Thomas
Colthurst, Alexander Ku, Dan Newburger, et al. 2018.
<span>“[<span>DeepVariant</span>] <span>A</span> Universal
<span>SNP</span> and Small-Indel Variant Caller Using Deep Neural
Networks.”</span> <em>Nature Biotechnology</em> 36 (10): 983–87. <a href="https://doi.org/10.1038/nbt.4235">https://doi.org/10.1038/nbt.4235</a>.
</div>
<div id="ref-press_train_2022" class="csl-entry" role="listitem">
Press, Ofir, Noah A. Smith, and Mike Lewis. 2022. <span>“Train
<span>Short</span>, <span>Test</span> <span>Long</span>:
<span>Attention</span> with <span>Linear</span> <span>Biases</span>
<span>Enables</span> <span>Input</span> <span>Length</span>
<span>Extrapolation</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2108.12409">https://doi.org/10.48550/arXiv.2108.12409</a>.
</div>
<div id="ref-price_principal_2006" class="csl-entry" role="listitem">
Price, Alkes L., Nick J. Patterson, Robert M. Plenge, Michael E.
Weinblatt, Nancy A. Shadick, and David Reich. 2006. <span>“Principal
Components Analysis Corrects for Stratification in Genome-Wide
Association Studies.”</span> <em>Nature Genetics</em> 38 (8): 904–9. <a href="https://doi.org/10.1038/ng1847">https://doi.org/10.1038/ng1847</a>.
</div>
<div id="ref-quan_coding_2005" class="csl-entry" role="listitem">
Quan, Hude, Vijaya Sundararajan, Patricia Halfon, Andrew Fong, Bernard
Burnand, Jean-Christophe Luthi, L. Duncan Saunders, Cynthia A. Beck,
Thomas E. Feasby, and William A. Ghali. 2005. <span>“Coding
<span>Algorithms</span> for <span>Defining</span>
<span>Comorbidities</span> in <span>ICD</span>-9-<span>CM</span> and
<span>ICD</span>-10 <span>Administrative</span>
<span>Data</span>.”</span> <em>Medical Care</em> 43 (11): 1130. <a href="https://doi.org/10.1097/01.mlr.0000182534.19832.83">https://doi.org/10.1097/01.mlr.0000182534.19832.83</a>.
</div>
<div id="ref-quang_dann_2015" class="csl-entry" role="listitem">
Quang, Daniel, Yifei Chen, and Xiaohui Xie. 2015.
<span>“<span>DANN</span>: A Deep Learning Approach for Annotating the
Pathogenicity of Genetic Variants.”</span> <em>Bioinformatics</em> 31
(5): 761–63. <a href="https://doi.org/10.1093/bioinformatics/btu703">https://doi.org/10.1093/bioinformatics/btu703</a>.
</div>
<div id="ref-quang_danq_2016" class="csl-entry" role="listitem">
Quang, Daniel, and Xiaohui Xie. 2016. <span>“<span>DanQ</span>: A Hybrid
Convolutional and Recurrent Deep Neural Network for Quantifying the
Function of <span>DNA</span> Sequences.”</span> <em>Nucleic Acids
Research</em> 44 (11): e107. <a href="https://doi.org/10.1093/nar/gkw226">https://doi.org/10.1093/nar/gkw226</a>.
</div>
<div id="ref-raffel_exploring_2023" class="csl-entry" role="listitem">
Raffel, Colin, Noam Shazeer, Adam Roberts, Katherine Lee, Sharan Narang,
Michael Matena, Yanqi Zhou, Wei Li, and Peter J. Liu. 2023.
<span>“Exploring the <span>Limits</span> of <span>Transfer</span>
<span>Learning</span> with a <span>Unified</span>
<span>Text</span>-to-<span>Text</span> <span>Transformer</span>.”</span>
arXiv. <a href="https://doi.org/10.48550/arXiv.1910.10683">https://doi.org/10.48550/arXiv.1910.10683</a>.
</div>
<div id="ref-rakowski_mifm_2025" class="csl-entry" role="listitem">
Rakowski, Alexander, and Christoph Lippert. 2025.
<span>“[<span>MIFM</span>] <span>Multiple</span> Instance Fine-Mapping:
Predicting Causal Regulatory Variants with a Deep Sequence
Model.”</span> medRxiv. <a href="https://doi.org/10.1101/2025.06.13.25329551">https://doi.org/10.1101/2025.06.13.25329551</a>.
</div>
<div id="ref-rao_evaluating_2019" class="csl-entry" role="listitem">
Rao, Roshan, Nicholas Bhattacharya, Neil Thomas, Yan Duan, Xi Chen, John
Canny, Pieter Abbeel, and Yun S. Song. 2019. <span>“Evaluating
<span>Protein</span> <span>Transfer</span> <span>Learning</span> with
<span>TAPE</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.1906.08230">https://doi.org/10.48550/arXiv.1906.08230</a>.
</div>
<div id="ref-rao_transformer_2020" class="csl-entry" role="listitem">
Rao, Roshan, Joshua Meier, Tom Sercu, Sergey Ovchinnikov, and Alexander
Rives. 2020. <span>“Transformer Protein Language Models Are Unsupervised
Structure Learners.”</span> bioRxiv. <a href="https://doi.org/10.1101/2020.12.15.422761">https://doi.org/10.1101/2020.12.15.422761</a>.
</div>
<div id="ref-rao_cohesin_2017" class="csl-entry" role="listitem">
Rao, Suhas S. P., Su-Chen Huang, Brian Glenn St Hilaire, Jesse M.
Engreitz, Elizabeth M. Perez, Kyong-Rim Kieffer-Kwon, Adrian L. Sanborn,
et al. 2017. <span>“Cohesin <span>Loss</span> <span>Eliminates</span>
<span>All</span> <span>Loop</span> <span>Domains</span>.”</span>
<em>Cell</em> 171 (2): 305–320.e24. <a href="https://doi.org/10.1016/j.cell.2017.09.026">https://doi.org/10.1016/j.cell.2017.09.026</a>.
</div>
<div id="ref-rao_3d_2014" class="csl-entry" role="listitem">
Rao, Suhas S. P., Miriam H. Huntley, Neva C. Durand, Elena K. Stamenova,
Ivan D. Bochkov, James T. Robinson, Adrian L. Sanborn, et al. 2014.
<span>“A <span>3D</span> <span>Map</span> of the <span>Human</span>
<span>Genome</span> at <span>Kilobase</span> <span>Resolution</span>
<span>Reveals</span> <span>Principles</span> of <span>Chromatin</span>
<span>Looping</span>.”</span> <em>Cell</em> 159 (7): 1665–80. <a href="https://doi.org/10.1016/j.cell.2014.11.021">https://doi.org/10.1016/j.cell.2014.11.021</a>.
</div>
<div id="ref-noauthor_realtimegenomicsrtg-core_2025" class="csl-entry" role="listitem">
<span>“<span>RealTimeGenomics</span>/Rtg-Core.”</span> 2025. Real Time
Genomics.
</div>
<div id="ref-regev_human_2017" class="csl-entry" role="listitem">
Regev, Aviv, Sarah A Teichmann, Eric S Lander, Ido Amit, Christophe
Benoist, Ewan Birney, Bernd Bodenmiller, et al. 2017. <span>“The
<span>Human</span> <span>Cell</span> <span>Atlas</span>.”</span> Edited
by Thomas R Gingeras. <em>eLife</em> 6 (December): e27041. <a href="https://doi.org/10.7554/eLife.27041">https://doi.org/10.7554/eLife.27041</a>.
</div>
<div id="ref-rehm_clingen_2015" class="csl-entry" role="listitem">
Rehm, Heidi L., Jonathan S. Berg, Lisa D. Brooks, Carlos D. Bustamante,
James P. Evans, Melissa J. Landrum, David H. Ledbetter, et al. 2015.
<span>“<span>ClinGen</span> — <span>The</span> <span>Clinical</span>
<span>Genome</span> <span>Resource</span>.”</span> <em>New England
Journal of Medicine</em> 372 (23): 2235–42. <a href="https://doi.org/10.1056/NEJMsr1406261">https://doi.org/10.1056/NEJMsr1406261</a>.
</div>
<div id="ref-relling_clinical_2019" class="csl-entry" role="listitem">
Relling, Mary V., Teri E. Klein, Roseann S. Gammal, Michelle
Whirl-Carrillo, James M. Hoffman, and Kelly E. Caudle. 2019. <span>“The
<span>Clinical</span> <span>Pharmacogenetics</span>
<span>Implementation</span> <span>Consortium</span>: 10
<span>Years</span> <span>Later</span>.”</span> <em>Clinical Pharmacology
&amp; Therapeutics</em> 107 (1): 171–75. <a href="https://doi.org/10.1002/cpt.1651">https://doi.org/10.1002/cpt.1651</a>.
</div>
<div id="ref-rentzsch_cadd-spliceimproving_2021" class="csl-entry" role="listitem">
Rentzsch, Philipp, Max Schubach, Jay Shendure, and Martin Kircher. 2021.
<span>“<span>CADD</span>-<span>Splice</span>—Improving Genome-Wide
Variant Effect Prediction Using Deep Learning-Derived Splice
Scores.”</span> <em>Genome Medicine</em> 13 (1): 31. <a href="https://doi.org/10.1186/s13073-021-00835-9">https://doi.org/10.1186/s13073-021-00835-9</a>.
</div>
<div id="ref-rentzsch_cadd_2019" class="csl-entry" role="listitem">
Rentzsch, Philipp, Daniela Witten, Gregory M Cooper, Jay Shendure, and
Martin Kircher. 2019. <span>“<span>CADD</span>: Predicting the
Deleteriousness of Variants Throughout the Human Genome.”</span>
<em>Nucleic Acids Research</em> 47 (D1): D886–94. <a href="https://doi.org/10.1093/nar/gky1016">https://doi.org/10.1093/nar/gky1016</a>.
</div>
<div id="ref-richards_standards_2015" class="csl-entry" role="listitem">
Richards, Sue, Nazneen Aziz, Sherri Bale, David Bick, Soma Das, Julie
Gastier-Foster, Wayne W. Grody, et al. 2015. <span>“Standards and
Guidelines for the Interpretation of Sequence Variants: A Joint
Consensus Recommendation of the <span>American</span>
<span>College</span> of <span>Medical</span> <span>Genetics</span> and
<span>Genomics</span> and the <span>Association</span> for
<span>Molecular</span> <span>Pathology</span>.”</span> <em>Genetics in
Medicine</em> 17 (5): 405–24. <a href="https://doi.org/10.1038/gim.2015.30">https://doi.org/10.1038/gim.2015.30</a>.
</div>
<div id="ref-rieke_future_2020" class="csl-entry" role="listitem">
Rieke, Nicola, Jonny Hancox, Wenqi Li, Fausto Milletarì, Holger R. Roth,
Shadi Albarqouni, Spyridon Bakas, et al. 2020. <span>“The Future of
Digital Health with Federated Learning.”</span> <em>Npj Digital
Medicine</em> 3 (1): 119. <a href="https://doi.org/10.1038/s41746-020-00323-1">https://doi.org/10.1038/s41746-020-00323-1</a>.
</div>
<div id="ref-risch_future_1996" class="csl-entry" role="listitem">
Risch, Neil, and Kathleen Merikangas. 1996. <span>“The
<span>Future</span> of <span>Genetic</span> <span>Studies</span> of
<span>Complex</span> <span>Human</span> <span>Diseases</span>.”</span>
<em>Science</em> 273 (5281): 1516–17. <a href="https://doi.org/10.1126/science.273.5281.1516">https://doi.org/10.1126/science.273.5281.1516</a>.
</div>
<div id="ref-rives_esm-1b_2021" class="csl-entry" role="listitem">
Rives, Alexander, Joshua Meier, Tom Sercu, Siddharth Goyal, Zeming Lin,
Jason Liu, Demi Guo, et al. 2021. <span>“[<span>ESM</span>-1b]
<span>Biological</span> Structure and Function Emerge from Scaling
Unsupervised Learning to 250 Million Protein Sequences.”</span>
<em>Proceedings of the National Academy of Sciences of the United States
of America</em> 118 (15): e2016239118. <a href="https://doi.org/10.1073/pnas.2016239118">https://doi.org/10.1073/pnas.2016239118</a>.
</div>
<div id="ref-robinson_ipd-imgthla_2020" class="csl-entry" role="listitem">
Robinson, James, Dominic J Barker, Xenia Georgiou, Michael A Cooper,
Paul Flicek, and Steven G E Marsh. 2020.
<span>“<span>IPD</span>-<span>IMGT</span>/<span>HLA</span>
<span>Database</span>.”</span> <em>Nucleic Acids Research</em> 48 (D1):
D948–55. <a href="https://doi.org/10.1093/nar/gkz950">https://doi.org/10.1093/nar/gkz950</a>.
</div>
<div id="ref-ruan_improving_2022" class="csl-entry" role="listitem">
Ruan, Yunfeng, Yen-Feng Lin, Yen-Chen Anne Feng, Chia-Yen Chen, Max Lam,
Zhenglin Guo, Lin He, et al. 2022. <span>“Improving Polygenic Prediction
in Ancestrally Diverse Populations.”</span> <em>Nature Genetics</em> 54
(5): 573–80. <a href="https://doi.org/10.1038/s41588-022-01054-7">https://doi.org/10.1038/s41588-022-01054-7</a>.
</div>
<div id="ref-rubin_statistical_2017" class="csl-entry" role="listitem">
Rubin, Alan F., Hannah Gelman, Nathan Lucas, Sandra M. Bajjalieh,
Anthony T. Papenfuss, Terence P. Speed, and Douglas M. Fowler. 2017.
<span>“A Statistical Framework for Analyzing Deep Mutational Scanning
Data.”</span> <em>Genome Biology</em> 18 (1): 150. <a href="https://doi.org/10.1186/s13059-017-1272-5">https://doi.org/10.1186/s13059-017-1272-5</a>.
</div>
<div id="ref-rubinacci_efficient_2021" class="csl-entry" role="listitem">
Rubinacci, Simone, Diogo M. Ribeiro, Robin J. Hofmeister, and Olivier
Delaneau. 2021. <span>“Efficient Phasing and Imputation of Low-Coverage
Sequencing Data Using Large Reference Panels.”</span> <em>Nature
Genetics</em> 53 (1): 120–26. <a href="https://doi.org/10.1038/s41588-020-00756-0">https://doi.org/10.1038/s41588-020-00756-0</a>.
</div>
<div id="ref-sainz_nlp_2023" class="csl-entry" role="listitem">
Sainz, Oscar, Jon Campos, Iker García-Ferrero, Julen Etxaniz, Oier Lopez
de Lacalle, and Eneko Agirre. 2023. <span>“<span>NLP</span>
<span>Evaluation</span> in Trouble: <span>On</span> the
<span>Need</span> to <span>Measure</span> <span>LLM</span>
<span>Data</span> <span>Contamination</span> for Each
<span>Benchmark</span>.”</span> In <em>Findings of the
<span>Association</span> for <span>Computational</span>
<span>Linguistics</span>: <span>EMNLP</span> 2023</em>, edited by Houda
Bouamor, Juan Pino, and Kalika Bali, 10776–87. Singapore: Association
for Computational Linguistics. <a href="https://doi.org/10.18653/v1/2023.findings-emnlp.722">https://doi.org/10.18653/v1/2023.findings-emnlp.722</a>.
</div>
<div id="ref-sakaue_tutorial_2023" class="csl-entry" role="listitem">
Sakaue, Saori, Saisriram Gurajala, Michelle Curtis, Yang Luo, Wanson
Choi, Kazuyoshi Ishigaki, Joyce B. Kang, et al. 2023. <span>“Tutorial: A
Statistical Genetics Guide to Identifying <span>HLA</span> Alleles
Driving Complex Disease.”</span> <em>Nature Protocols</em> 18 (9):
2625–41. <a href="https://doi.org/10.1038/s41596-023-00853-4">https://doi.org/10.1038/s41596-023-00853-4</a>.
</div>
<div id="ref-sample_human_2019" class="csl-entry" role="listitem">
Sample, Paul J., Ban Wang, David W. Reid, Vlad Presnyak, Iain J.
McFadyen, David R. Morris, and Georg Seelig. 2019. <span>“Human 5′
<span>UTR</span> Design and Variant Effect Prediction from a Massively
Parallel Translation Assay.”</span> <em>Nature Biotechnology</em> 37
(7): 803–9. <a href="https://doi.org/10.1038/s41587-019-0164-5">https://doi.org/10.1038/s41587-019-0164-5</a>.
</div>
<div id="ref-sanabria_grover_2024" class="csl-entry" role="listitem">
Sanabria, Melissa, Jonas Hirsch, Pierre M. Joubert, and Anna R. Poetsch.
2024. <span>“[<span>GROVER</span>] <span>DNA</span> Language Model
<span>GROVER</span> Learns Sequence Context in the Human Genome.”</span>
<em>Nature Machine Intelligence</em> 6 (8): 911–23. <a href="https://doi.org/10.1038/s42256-024-00872-0">https://doi.org/10.1038/s42256-024-00872-0</a>.
</div>
<div id="ref-sanborn_enhancer_2015" class="csl-entry" role="listitem">
Sanborn, Adrian L., Suhas S. P. Rao, Su-Chen Huang, Neva C. Durand,
Miriam H. Huntley, Andrew I. Jewett, Ivan D. Bochkov, et al. 2015.
<span>“Chromatin Extrusion Explains Key Features of Loop and Domain
Formation in Wild-Type and Engineered Genomes.”</span> <em>Proceedings
of the National Academy of Sciences</em> 112 (47): E6456–65. <a href="https://doi.org/10.1073/pnas.1518552112">https://doi.org/10.1073/pnas.1518552112</a>.
</div>
<div id="ref-sangkuhl_pharmacogenomics_2019" class="csl-entry" role="listitem">
Sangkuhl, Katrin, Michelle Whirl-Carrillo, Ryan M. Whaley, Mark Woon,
Adam Lavertu, Russ B. Altman, Lester Carter, Anurag Verma, Marylyn D.
Ritchie, and Teri E. Klein. 2019. <span>“Pharmacogenomics
<span>Clinical</span> <span>Annotation</span> <span>Tool</span>
(<span>PharmCAT</span>).”</span> <em>Clinical Pharmacology &amp;
Therapeutics</em> 107 (1): 203–10. <a href="https://doi.org/10.1002/cpt.1568">https://doi.org/10.1002/cpt.1568</a>.
</div>
<div id="ref-schiff_caduceus_2024" class="csl-entry" role="listitem">
Schiff, Yair, Chia-Hsiang Kao, Aaron Gokaslan, Tri Dao, Albert Gu, and
Volodymyr Kuleshov. 2024. <span>“Caduceus:
<span>Bi</span>-<span>Directional</span> <span>Equivariant</span>
<span>Long</span>-<span>Range</span> <span>DNA</span>
<span>Sequence</span> <span>Modeling</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2403.03234">https://doi.org/10.48550/arXiv.2403.03234</a>.
</div>
<div id="ref-schubach_cadd_2024" class="csl-entry" role="listitem">
Schubach, Max, Thorben Maass, Lusiné Nazaretyan, Sebastian Röner, and
Martin Kircher. 2024. <span>“<span>CADD</span> V1.7: Using Protein
Language Models, Regulatory <span>CNNs</span> and Other Nucleotide-Level
Scores to Improve Genome-Wide Variant Predictions.”</span> <em>Nucleic
Acids Research</em> 52 (D1): D1143–54. <a href="https://doi.org/10.1093/nar/gkad989">https://doi.org/10.1093/nar/gkad989</a>.
</div>
<div id="ref-shafin_haplotype-aware_2021" class="csl-entry" role="listitem">
Shafin, Kishwar, Trevor Pesout, Pi-Chuan Chang, Maria Nattestad, Alexey
Kolesnikov, Sidharth Goel, Gunjan Baid, et al. 2021.
<span>“Haplotype-Aware Variant Calling with
<span>PEPPER</span>-<span>Margin</span>-<span>DeepVariant</span> Enables
High Accuracy in Nanopore Long-Reads.”</span> <em>Nature Methods</em> 18
(11): 1322–32. <a href="https://doi.org/10.1038/s41592-021-01299-w">https://doi.org/10.1038/s41592-021-01299-w</a>.
</div>
<div id="ref-shalem_genome-scale_2014" class="csl-entry" role="listitem">
Shalem, Ophir, Neville E. Sanjana, Ella Hartenian, Xi Shi, David A.
Scott, Tarjei S. Mikkelsen, Dirk Heckl, et al. 2014.
<span>“Genome-<span>Scale</span> <span>CRISPR</span>-<span>Cas9</span>
<span>Knockout</span> <span>Screening</span> in <span>Human</span>
<span>Cells</span>.”</span> <em>Science</em> 343 (6166): 84–87. <a href="https://doi.org/10.1126/science.1247005">https://doi.org/10.1126/science.1247005</a>.
</div>
<div id="ref-sherry_dbsnp_2001" class="csl-entry" role="listitem">
Sherry, S. T., M.-H. Ward, M. Kholodov, J. Baker, L. Phan, E. M.
Smigielski, and K. Sirotkin. 2001. <span>“<span class="nocase">dbSNP</span>: The <span>NCBI</span> Database of Genetic
Variation.”</span> <em>Nucleic Acids Research</em> 29 (1): 308–11. <a href="https://doi.org/10.1093/nar/29.1.308">https://doi.org/10.1093/nar/29.1.308</a>.
</div>
<div id="ref-shevlane_structured_2022" class="csl-entry" role="listitem">
Shevlane, Toby. 2022. <span>“Structured Access: An Emerging Paradigm for
Safe <span>AI</span> Deployment.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2201.05159">https://doi.org/10.48550/arXiv.2201.05159</a>.
</div>
<div id="ref-shrikumar_learning_2017" class="csl-entry" role="listitem">
Shrikumar, Avanti, Peyton Greenside, and Anshul Kundaje. 2017.
<span>“Learning <span>Important</span> <span>Features</span>
<span>Through</span> <span>Propagating</span> <span>Activation</span>
<span>Differences</span>.”</span> In <em>Proceedings of the 34th
<span>International</span> <span>Conference</span> on
<span>Machine</span> <span>Learning</span></em>, 3145–53. PMLR.
</div>
<div id="ref-shrikumar_technical_2018" class="csl-entry" role="listitem">
Shrikumar, Avanti, Katherine Tian, Žiga Avsec, Anna Shcherbina,
Abhimanyu Banerjee, Mahfuza Sharmin, Surag Nair, and Anshul Kundaje.
2018. <span>“Technical <span>Note</span> on <span>Transcription</span>
<span>Factor</span> <span>Motif</span> <span>Discovery</span> from
<span>Importance</span> <span>Scores</span>
(<span>TF</span>-<span>MoDISco</span>) Version 0.5.6.5.”</span> arXiv.
<a href="https://doi.org/10.48550/arXiv.1811.00416">https://doi.org/10.48550/arXiv.1811.00416</a>.
</div>
<div id="ref-siepel_phastcons_2005" class="csl-entry" role="listitem">
Siepel, Adam, Gill Bejerano, Jakob S. Pedersen, Angie S. Hinrichs,
Minmei Hou, Kate Rosenbloom, Hiram Clawson, et al. 2005.
<span>“[<span>PhastCons</span>] <span>Evolutionarily</span> Conserved
Elements in Vertebrate, Insect, Worm, and Yeast Genomes.”</span>
<em>Genome Research</em> 15 (8): 1034–50. <a href="https://doi.org/10.1101/gr.3715005">https://doi.org/10.1101/gr.3715005</a>.
</div>
<div id="ref-singh_rna_2019" class="csl-entry" role="listitem">
Singh, Jaswinder, Jack Hanson, Kuldip Paliwal, and Yaoqi Zhou. 2019.
<span>“<span>RNA</span> Secondary Structure Prediction Using an Ensemble
of Two-Dimensional Deep Neural Networks and Transfer Learning.”</span>
<em>Nature Communications</em> 10 (1): 5407. <a href="https://doi.org/10.1038/s41467-019-13395-9">https://doi.org/10.1038/s41467-019-13395-9</a>.
</div>
<div id="ref-sirugo_missing_2019" class="csl-entry" role="listitem">
Sirugo, Giorgio, Scott M. Williams, and Sarah A. Tishkoff. 2019.
<span>“The <span>Missing</span> <span>Diversity</span> in
<span>Human</span> <span>Genetic</span> <span>Studies</span>.”</span>
<em>Cell</em> 177 (1): 26–31. <a href="https://doi.org/10.1016/j.cell.2019.02.048">https://doi.org/10.1016/j.cell.2019.02.048</a>.
</div>
<div id="ref-smolka_detection_2024" class="csl-entry" role="listitem">
Smolka, Moritz, Luis F. Paulin, Christopher M. Grochowski, Dominic W.
Horner, Medhat Mahmoud, Sairam Behera, Ester Kalef-Ezra, et al. 2024.
<span>“Detection of Mosaic and Population-Level Structural Variants with
<span>Sniffles2</span>.”</span> <em>Nature Biotechnology</em> 42 (10):
1571–80. <a href="https://doi.org/10.1038/s41587-023-02024-y">https://doi.org/10.1038/s41587-023-02024-y</a>.
</div>
<div id="ref-snell_prototypical_2017" class="csl-entry" role="listitem">
Snell, Jake, Kevin Swersky, and Richard Zemel. 2017. <span>“Prototypical
<span>Networks</span> for <span>Few</span>-Shot
<span>Learning</span>.”</span> In <em>Advances in <span>Neural</span>
<span>Information</span> <span>Processing</span>
<span>Systems</span></em>. Vol. 30. Curran Associates, Inc.
</div>
<div id="ref-soice_can_2023" class="csl-entry" role="listitem">
Soice, Emily H., Rafael Rocha, Kimberlee Cordova, Michael Specter, and
Kevin M. Esvelt. 2023. <span>“Can Large Language Models Democratize
Access to Dual-Use Biotechnology?”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2306.03809">https://doi.org/10.48550/arXiv.2306.03809</a>.
</div>
<div id="ref-sollis_nhgri-ebi_2023" class="csl-entry" role="listitem">
Sollis, Elliot, Abayomi Mosaku, Ala Abid, Annalisa Buniello, Maria
Cerezo, Laurent Gil, Tudor Groza, et al. 2023. <span>“The
<span>NHGRI</span>-<span>EBI</span> <span>GWAS</span>
<span>Catalog</span>: Knowledgebase and Deposition Resource.”</span>
<em>Nucleic Acids Research</em> 51 (D1): D977–85. <a href="https://doi.org/10.1093/nar/gkac1010">https://doi.org/10.1093/nar/gkac1010</a>.
</div>
<div id="ref-song_t1k_2022" class="csl-entry" role="listitem">
Song, Li, Gali Bai, X. Shirley Liu, Bo Li, and Heng Li. 2022.
<span>“<span>T1K</span>: Efficient and Accurate <span>KIR</span> and
<span>HLA</span> Genotyping with Next-Generation Sequencing
Data.”</span> bioRxiv. <a href="https://doi.org/10.1101/2022.10.26.513955">https://doi.org/10.1101/2022.10.26.513955</a>.
</div>
<div id="ref-spitale_structural_2015" class="csl-entry" role="listitem">
Spitale, Robert C., Ryan A. Flynn, Qiangfeng Cliff Zhang, Pete Crisalli,
Byron Lee, Jee-Wook Jung, Hannes Y. Kuchelmeister, et al. 2015.
<span>“Structural Imprints in Vivo Decode <span>RNA</span> Regulatory
Mechanisms.”</span> <em>Nature</em> 519 (7544): 486–90. <a href="https://doi.org/10.1038/nature14263">https://doi.org/10.1038/nature14263</a>.
</div>
<div id="ref-steinegger_protein-level_2019" class="csl-entry" role="listitem">
Steinegger, Martin, Milot Mirdita, and Johannes Söding. 2019.
<span>“Protein-Level Assembly Increases Protein Sequence Recovery from
Metagenomic Samples Manyfold.”</span> <em>Nature Methods</em> 16 (7):
603–6. <a href="https://doi.org/10.1038/s41592-019-0437-4">https://doi.org/10.1038/s41592-019-0437-4</a>.
</div>
<div id="ref-stenson_human_2017" class="csl-entry" role="listitem">
Stenson, Peter D., Matthew Mort, Edward V. Ball, Katy Evans, Matthew
Hayden, Sally Heywood, Michelle Hussain, Andrew D. Phillips, and David
N. Cooper. 2017. <span>“The <span>Human</span> <span>Gene</span>
<span>Mutation</span> <span>Database</span>: Towards a Comprehensive
Repository of Inherited Mutation Data for Medical Research, Genetic
Diagnosis and Next-Generation Sequencing Studies.”</span> <em>Human
Genetics</em> 136 (6): 665–77. <a href="https://doi.org/10.1007/s00439-017-1779-6">https://doi.org/10.1007/s00439-017-1779-6</a>.
</div>
<div id="ref-su_roformer_2024" class="csl-entry" role="listitem">
Su, Jianlin, Murtadha Ahmed, Yu Lu, Shengfeng Pan, Wen Bo, and Yunfeng
Liu. 2024. <span>“<span>RoFormer</span>: <span>Enhanced</span>
Transformer with <span>Rotary</span> <span>Position</span>
<span>Embedding</span>.”</span> <em>Neurocomputing</em> 568 (February):
127063. <a href="https://doi.org/10.1016/j.neucom.2023.127063">https://doi.org/10.1016/j.neucom.2023.127063</a>.
</div>
<div id="ref-sudlow_uk_2015" class="csl-entry" role="listitem">
Sudlow, Cathie, John Gallacher, Naomi Allen, Valerie Beral, Paul Burton,
John Danesh, Paul Downey, et al. 2015. <span>“<span>UK</span>
<span>Biobank</span>: <span>An</span> <span>Open</span>
<span>Access</span> <span>Resource</span> for <span>Identifying</span>
the <span>Causes</span> of a <span>Wide</span> <span>Range</span> of
<span>Complex</span> <span>Diseases</span> of <span>Middle</span> and
<span>Old</span> <span>Age</span>.”</span> <em>PLOS Medicine</em> 12
(3): e1001779. <a href="https://doi.org/10.1371/journal.pmed.1001779">https://doi.org/10.1371/journal.pmed.1001779</a>.
</div>
<div id="ref-sullivan_leveraging_2023" class="csl-entry" role="listitem">
Sullivan, Patrick F., Jennifer R. S. Meadows, Steven Gazal, BaDoi N.
Phan, Xue Li, Diane P. Genereux, Michael X. Dong, et al. 2023.
<span>“Leveraging Base-Pair Mammalian Constraint to Understand Genetic
Variation and Human Disease.”</span> <em>Science</em> 380 (6643):
eabn2937. <a href="https://doi.org/10.1126/science.abn2937">https://doi.org/10.1126/science.abn2937</a>.
</div>
<div id="ref-sundararajan_axiomatic_2017" class="csl-entry" role="listitem">
Sundararajan, Mukund, Ankur Taly, and Qiqi Yan. 2017. <span>“Axiomatic
<span>Attribution</span> for <span>Deep</span>
<span>Networks</span>.”</span> In <em>Proceedings of the 34th
<span>International</span> <span>Conference</span> on
<span>Machine</span> <span>Learning</span></em>, 3319–28. PMLR.
</div>
<div id="ref-supreme_court_of_the_united_states_assoc_2013" class="csl-entry" role="listitem">
Supreme Court of the United States. 2013. <span>“Assoc. For
<span>Molecular</span> <span>Pathology</span> v. <span>Myriad</span>
<span>Genetics</span>, <span>Inc</span>., 569
<span>U</span>.<span>S</span>. 576 (2013).”</span>
</div>
<div id="ref-suzek_uniref_2007" class="csl-entry" role="listitem">
Suzek, Baris E., Hongzhan Huang, Peter McGarvey, Raja Mazumder, and
Cathy H. Wu. 2007. <span>“<span>UniRef</span>: Comprehensive and
Non-Redundant <span>UniProt</span> Reference Clusters.”</span>
<em>Bioinformatics</em> 23 (10): 1282–88. <a href="https://doi.org/10.1093/bioinformatics/btm098">https://doi.org/10.1093/bioinformatics/btm098</a>.
</div>
<div id="ref-tabula_sapiens_consortium_the_tabula_2022" class="csl-entry" role="listitem">
Tabula Sapiens Consortium, The. 2022. <span>“The <span>Tabula</span>
<span>Sapiens</span>: <span>A</span> Multiple-Organ, Single-Cell
Transcriptomic Atlas of Humans.”</span> <em>Science</em> 376 (6594):
eabl4896. <a href="https://doi.org/10.1126/science.abl4896">https://doi.org/10.1126/science.abl4896</a>.
</div>
<div id="ref-tan_cell-type-specific_2023" class="csl-entry" role="listitem">
Tan, Jimin, Nina Shenker-Tauris, Javier Rodriguez-Hernaez, Eric Wang,
Theodore Sakellaropoulos, Francesco Boccalatte, Palaniraja Thandapani,
et al. 2023. <span>“Cell-Type-Specific Prediction of <span>3D</span>
Chromatin Organization Enables High-Throughput in Silico Genetic
Screening.”</span> <em>Nature Biotechnology</em> 41 (8): 1140–50. <a href="https://doi.org/10.1038/s41587-022-01612-8">https://doi.org/10.1038/s41587-022-01612-8</a>.
</div>
<div id="ref-tate_cosmic_2019" class="csl-entry" role="listitem">
Tate, John G, Sally Bamford, Harry C Jubb, Zbyslaw Sondka, David M
Beare, Nidhi Bindal, Harry Boutselakis, et al. 2019.
<span>“<span>COSMIC</span>: The <span>Catalogue</span> <span>Of</span>
<span>Somatic</span> <span>Mutations</span> <span>In</span>
<span>Cancer</span>.”</span> <em>Nucleic Acids Research</em> 47 (D1):
D941–47. <a href="https://doi.org/10.1093/nar/gky1015">https://doi.org/10.1093/nar/gky1015</a>.
</div>
<div id="ref-tavtigian_modeling_2018" class="csl-entry" role="listitem">
Tavtigian, Sean V., Marc S. Greenblatt, Steven M. Harrison, Robert L.
Nussbaum, Snehit A. Prabhu, Kenneth M. Boucher, and Leslie G. Biesecker.
2018. <span>“Modeling the <span>ACMG</span>/<span>AMP</span> Variant
Classification Guidelines as a <span>Bayesian</span> Classification
Framework.”</span> <em>Genetics in Medicine</em> 20 (9): 1054–60. <a href="https://doi.org/10.1038/gim.2017.210">https://doi.org/10.1038/gim.2017.210</a>.
</div>
<div id="ref-theodoris_geneformer_2023" class="csl-entry" role="listitem">
Theodoris, Christina V., Ling Xiao, Anant Chopra, Mark D. Chaffin, Zeina
R. Al Sayed, Matthew C. Hill, Helene Mantineo, et al. 2023.
<span>“[<span>Geneformer</span>] <span>Transfer</span> Learning Enables
Predictions in Network Biology.”</span> <em>Nature</em> 618 (7965):
616–24. <a href="https://doi.org/10.1038/s41586-023-06139-9">https://doi.org/10.1038/s41586-023-06139-9</a>.
</div>
<div id="ref-tipirneni_self-supervised_2022" class="csl-entry" role="listitem">
Tipirneni, Sindhu, and Chandan K. Reddy. 2022.
<span>“Self-<span>Supervised</span> <span>Transformer</span> for
<span>Sparse</span> and <span>Irregularly</span> <span>Sampled</span>
<span>Multivariate</span> <span>Clinical</span>
<span>Time</span>-<span>Series</span>.”</span> <em>ACM Trans. Knowl.
Discov. Data</em> 16 (6): 105:1–17. <a href="https://doi.org/10.1145/3516367">https://doi.org/10.1145/3516367</a>.
</div>
<div id="ref-torkamani_personal_2018" class="csl-entry" role="listitem">
Torkamani, Ali, Nathan E. Wineinger, and Eric J. Topol. 2018. <span>“The
Personal and Clinical Utility of Polygenic Risk Scores.”</span>
<em>Nature Reviews Genetics</em> 19 (9): 581–90. <a href="https://doi.org/10.1038/s41576-018-0018-x">https://doi.org/10.1038/s41576-018-0018-x</a>.
</div>
<div id="ref-us_food_and_drug_administration_artificial_2021" class="csl-entry" role="listitem">
U.S. Food and Drug Administration. 2021. <span>“Artificial
<span>Intelligence</span>/<span>Machine</span> <span>Learning</span>
(<span>‘<span>AI</span>/<span>ML</span>’</span>)-<span>Based</span>
<span>Software</span> as a <span>Medical</span> <span>Device</span>
(<span>‘<span>SaMD</span>’</span>) <span>Action</span>
<span>Plan</span>.”</span>
</div>
<div id="ref-us_congress_genetic_2008" class="csl-entry" role="listitem">
US Congress. 2008. <span>“Genetic <span>Information</span>
<span>Nondiscrimination</span> <span>Act</span> of 2008.”</span>
</div>
<div id="ref-van_der_auwera_fastq_2018" class="csl-entry" role="listitem">
Van der Auwera, Geraldine A., Mauricio O. Carneiro, Christopher Hartl,
Ryan Poplin, Guillermo del Angel, Ami Levy-Moonshine, Tadeusz Jordan, et
al. 2018. <span>“From <span>FastQ</span> <span>Data</span> to
<span>High</span>-<span>Confidence</span> <span>Variant</span>
<span>Calls</span>: <span>The</span> <span>Genome</span>
<span>Analysis</span> <span>Toolkit</span> <span>Best</span>
<span>Practices</span> <span>Pipeline</span>.”</span> <em>Current
Protocols in Bioinformatics</em> 43 (1): 11.10.1–33. <a href="https://doi.org/10.1002/0471250953.bi1110s43">https://doi.org/10.1002/0471250953.bi1110s43</a>.
</div>
<div id="ref-vapnik_statistical_1998" class="csl-entry" role="listitem">
Vapnik, Vladimir. 1998. <em>Statistical <span>Learning</span>
<span>Theory</span></em>. Wiley.
</div>
<div id="ref-varadi_alphafold_2022" class="csl-entry" role="listitem">
Varadi, Mihaly, Stephen Anyango, Mandar Deshpande, Sreenath Nair, Cindy
Natassia, Galabina Yordanova, David Yuan, et al. 2022.
<span>“<span>AlphaFold</span> <span>Protein</span>
<span>Structure</span> <span>Database</span>: Massively Expanding the
Structural Coverage of Protein-Sequence Space with High-Accuracy
Models.”</span> <em>Nucleic Acids Research</em> 50 (D1): D439–44. <a href="https://doi.org/10.1093/nar/gkab1061">https://doi.org/10.1093/nar/gkab1061</a>.
</div>
<div id="ref-vaswani_attention_2023" class="csl-entry" role="listitem">
Vaswani, Ashish, Noam Shazeer, Niki Parmar, Jakob Uszkoreit, Llion
Jones, Aidan N. Gomez, Lukasz Kaiser, and Illia Polosukhin. 2023.
<span>“Attention <span>Is</span> <span>All</span> <span>You</span>
<span>Need</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.1706.03762">https://doi.org/10.48550/arXiv.1706.03762</a>.
</div>
<div id="ref-vilhjalmsson_modeling_2015" class="csl-entry" role="listitem">
Vilhjálmsson, Bjarni J., Jian Yang, Hilary K. Finucane, Alexander Gusev,
Sara Lindström, Stephan Ripke, Giulio Genovese, et al. 2015.
<span>“Modeling <span>Linkage</span> <span>Disequilibrium</span>
<span>Increases</span> <span>Accuracy</span> of <span>Polygenic</span>
<span>Risk</span> <span>Scores</span>.”</span> <em>American Journal of
Human Genetics</em> 97 (4): 576–92. <a href="https://doi.org/10.1016/j.ajhg.2015.09.001">https://doi.org/10.1016/j.ajhg.2015.09.001</a>.
</div>
<div id="ref-vishniakov_gene42_2025" class="csl-entry" role="listitem">
Vishniakov, Kirill, Boulbaba Ben Amor, Engin Tekin, Nancy A. ElNaker,
Karthik Viswanathan, Aleksandr Medvedev, Aahan Singh, et al. 2025.
<span>“Gene42: <span>Long</span>-<span>Range</span> <span>Genomic</span>
<span>Foundation</span> <span>Model</span> <span>With</span>
<span>Dense</span> <span>Attention</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2503.16565">https://doi.org/10.48550/arXiv.2503.16565</a>.
</div>
<div id="ref-visscher_heritability_2008" class="csl-entry" role="listitem">
Visscher, Peter M., William G. Hill, and Naomi R. Wray. 2008.
<span>“Heritability in the Genomics Era — Concepts and
Misconceptions.”</span> <em>Nature Reviews Genetics</em> 9 (4): 255–66.
<a href="https://doi.org/10.1038/nrg2322">https://doi.org/10.1038/nrg2322</a>.
</div>
<div id="ref-vosa_large-scale_2021" class="csl-entry" role="listitem">
Võsa, Urmo, Annique Claringbould, Harm-Jan Westra, Marc Jan Bonder,
Patrick Deelen, Biao Zeng, Holger Kirsten, et al. 2021.
<span>“Large-Scale Cis- and Trans-<span class="nocase">eQTL</span>
Analyses Identify Thousands of Genetic Loci and Polygenic Scores That
Regulate Blood Gene Expression.”</span> <em>Nature Genetics</em> 53 (9):
1300–1310. <a href="https://doi.org/10.1038/s41588-021-00913-z">https://doi.org/10.1038/s41588-021-00913-z</a>.
</div>
<div id="ref-wang_tent_2021" class="csl-entry" role="listitem">
Wang, Dequan, Evan Shelhamer, Shaoteng Liu, Bruno Olshausen, and Trevor
Darrell. 2021. <span>“Tent: <span>Fully</span> <span>Test</span>-Time
<span>Adaptation</span> by <span>Entropy</span>
<span>Minimization</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2006.10726">https://doi.org/10.48550/arXiv.2006.10726</a>.
</div>
<div id="ref-wang_simple_2020" class="csl-entry" role="listitem">
Wang, Gao, Abhishek Sarkar, Peter Carbonetto, and Matthew Stephens.
2020. <span>“A <span>Simple</span> <span>New</span>
<span>Approach</span> to <span>Variable</span> <span>Selection</span> in
<span>Regression</span>, with <span>Application</span> to
<span>Genetic</span> <span>Fine</span> <span>Mapping</span>.”</span>
<em>Journal of the Royal Statistical Society Series B: Statistical
Methodology</em> 82 (5): 1273–1300. <a href="https://doi.org/10.1111/rssb.12388">https://doi.org/10.1111/rssb.12388</a>.
</div>
<div id="ref-wang_linformer_2020" class="csl-entry" role="listitem">
Wang, Sinong, Belinda Z. Li, Madian Khabsa, Han Fang, and Hao Ma. 2020.
<span>“Linformer: <span>Self</span>-<span>Attention</span> with
<span>Linear</span> <span>Complexity</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2006.04768">https://doi.org/10.48550/arXiv.2006.04768</a>.
</div>
<div id="ref-wang_characterizing_2018" class="csl-entry" role="listitem">
Wang, Zirui, Zihang Dai, Barnabas Poczos, and Jaime Carbonell. 2018.
<span>“Characterizing and <span>Avoiding</span> <span>Negative</span>
<span>Transfer</span>.”</span> In, 11293–302.
</div>
<div id="ref-watson_novo_2023" class="csl-entry" role="listitem">
Watson, Joseph L., David Juergens, Nathaniel R. Bennett, Brian L.
Trippe, Jason Yim, Helen E. Eisenach, Woody Ahern, et al. 2023.
<span>“De Novo Design of Protein Structure and Function with
<span>RFdiffusion</span>.”</span> <em>Nature</em> 620 (7976): 1089–1100.
<a href="https://doi.org/10.1038/s41586-023-06415-8">https://doi.org/10.1038/s41586-023-06415-8</a>.
</div>
<div id="ref-wei_emergent_2022" class="csl-entry" role="listitem">
Wei, Jason, Yi Tay, Rishi Bommasani, Colin Raffel, Barret Zoph,
Sebastian Borgeaud, Dani Yogatama, et al. 2022. <span>“Emergent
<span>Abilities</span> of <span>Large</span> <span>Language</span>
<span>Models</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2206.07682">https://doi.org/10.48550/arXiv.2206.07682</a>.
</div>
<div id="ref-wenger_accurate_2019" class="csl-entry" role="listitem">
Wenger, Aaron M., Paul Peluso, William J. Rowell, Pi-Chuan Chang,
Richard J. Hall, Gregory T. Concepcion, Jana Ebler, et al. 2019.
<span>“Accurate Circular Consensus Long-Read Sequencing Improves Variant
Detection and Assembly of a Human Genome.”</span> <em>Nature
Biotechnology</em> 37 (10): 1155–62. <a href="https://doi.org/10.1038/s41587-019-0217-9">https://doi.org/10.1038/s41587-019-0217-9</a>.
</div>
<div id="ref-whirl-carrillo_pharmacogenomics_2012" class="csl-entry" role="listitem">
Whirl-Carrillo, M, E M McDonagh, J M Hebert, L Gong, K Sangkuhl, C F
Thorn, R B Altman, and T E Klein. 2012. <span>“Pharmacogenomics
<span>Knowledge</span> for <span>Personalized</span>
<span>Medicine</span>.”</span> <em>Clinical Pharmacology &amp;
Therapeutics</em> 92 (4): 414–17. <a href="https://doi.org/10.1038/clpt.2012.96">https://doi.org/10.1038/clpt.2012.96</a>.
</div>
<div id="ref-wu_genome-wide_2024" class="csl-entry" role="listitem">
Wu, Yang, Zhili Zheng, Loic Thibaut2, Michael E. Goddard, Naomi R. Wray,
Peter M. Visscher, and Jian Zeng. 2024. <span>“Genome-Wide Fine-Mapping
Improves Identification of Causal Variants.”</span> <em>Research
Square</em>, August, rs.3.rs–4759390. <a href="https://doi.org/10.21203/rs.3.rs-4759390/v1">https://doi.org/10.21203/rs.3.rs-4759390/v1</a>.
</div>
<div id="ref-xiong_layer_2020" class="csl-entry" role="listitem">
Xiong, Ruibin, Yunchang Yang, Di He, Kai Zheng, Shuxin Zheng, Chen Xing,
Huishuai Zhang, Yanyan Lan, Liwei Wang, and Tieyan Liu. 2020. <span>“On
<span>Layer</span> <span>Normalization</span> in the
<span>Transformer</span> <span>Architecture</span>.”</span> In
<em>Proceedings of the 37th <span>International</span>
<span>Conference</span> on <span>Machine</span>
<span>Learning</span></em>, 10524–33. PMLR.
</div>
<div id="ref-yang_common_2010" class="csl-entry" role="listitem">
Yang, Jian, Beben Benyamin, Brian P. McEvoy, Scott Gordon, Anjali K.
Henders, Dale R. Nyholt, Pamela A. Madden, et al. 2010. <span>“Common
<span>SNPs</span> Explain a Large Proportion of the Heritability for
Human Height.”</span> <em>Nature Genetics</em> 42 (7): 565–69. <a href="https://doi.org/10.1038/ng.608">https://doi.org/10.1038/ng.608</a>.
</div>
<div id="ref-yang_xlnet_2020" class="csl-entry" role="listitem">
Yang, Zhilin, Zihang Dai, Yiming Yang, Jaime Carbonell, Ruslan
Salakhutdinov, and Quoc V. Le. 2020. <span>“<span>XLNet</span>:
<span>Generalized</span> <span>Autoregressive</span>
<span>Pretraining</span> for <span>Language</span>
<span>Understanding</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.1906.08237">https://doi.org/10.48550/arXiv.1906.08237</a>.
</div>
<div id="ref-yengo_saturated_2022" class="csl-entry" role="listitem">
Yengo, Loïc, Sailaja Vedantam, Eirini Marouli, Julia Sidorenko, Eric
Bartell, Saori Sakaue, Marielisa Graff, et al. 2022. <span>“A Saturated
Map of Common Genetic Variants Associated with Human Height.”</span>
<em>Nature</em> 610 (7933): 704–12. <a href="https://doi.org/10.1038/s41586-022-05275-y">https://doi.org/10.1038/s41586-022-05275-y</a>.
</div>
<div id="ref-yeo_maximum_2004" class="csl-entry" role="listitem">
Yeo, Gene, and Christopher B. Burge. 2004. <span>“Maximum
<span>Entropy</span> <span>Modeling</span> of <span>Short</span>
<span>Sequence</span> <span>Motifs</span> with <span>Applications</span>
to <span>RNA</span> <span>Splicing</span> <span>Signals</span>.”</span>
<em>Journal of Computational Biology</em> 11 (2-3): 377–94. <a href="https://doi.org/10.1089/1066527041410418">https://doi.org/10.1089/1066527041410418</a>.
</div>
<div id="ref-yu_assessing_2024" class="csl-entry" role="listitem">
Yu, Ying, Yuanbang Mai, Yuanting Zheng, and Leming Shi. 2024.
<span>“Assessing and Mitigating Batch Effects in Large-Scale Omics
Studies.”</span> <em>Genome Biology</em> 25 (1): 254. <a href="https://doi.org/10.1186/s13059-024-03401-9">https://doi.org/10.1186/s13059-024-03401-9</a>.
</div>
<div id="ref-yun_accurate_2021" class="csl-entry" role="listitem">
Yun, Taedong, Helen Li, Pi-Chuan Chang, Michael F Lin, Andrew Carroll,
and Cory Y McLean. 2021. <span>“Accurate, Scalable Cohort Variant Calls
Using <span>DeepVariant</span> and <span>GLnexus</span>.”</span>
<em>Bioinformatics</em> 36 (24): 5582–89. <a href="https://doi.org/10.1093/bioinformatics/btaa1081">https://doi.org/10.1093/bioinformatics/btaa1081</a>.
</div>
<div id="ref-zanger_cytochrome_2013" class="csl-entry" role="listitem">
Zanger, Ulrich M., and Matthias Schwab. 2013. <span>“Cytochrome
<span>P450</span> Enzymes in Drug Metabolism: <span>Regulation</span> of
Gene Expression, Enzyme Activities, and Impact of Genetic
Variation.”</span> <em>Pharmacology &amp; Therapeutics</em> 138 (1):
103–41. <a href="https://doi.org/10.1016/j.pharmthera.2012.12.007">https://doi.org/10.1016/j.pharmthera.2012.12.007</a>.
</div>
<div id="ref-zhang_understanding_2021" class="csl-entry" role="listitem">
Zhang, Chiyuan, Samy Bengio, Moritz Hardt, Benjamin Recht, and Oriol
Vinyals. 2021. <span>“Understanding Deep Learning (Still) Requires
Rethinking Generalization.”</span> <em>Commun. ACM</em> 64 (3): 107–15.
<a href="https://doi.org/10.1145/3446776">https://doi.org/10.1145/3446776</a>.
</div>
<div id="ref-zhang_spatial_2012" class="csl-entry" role="listitem">
Zhang, Yu, Rachel Patton McCord, Yu-Jui Ho, Brian R. Laber, Diana S.
Aber, Jungha Kim, Xiaowen Zhang, and Tom Misteli. 2012. <span>“Spatial
Organization of the Mouse Genome and Its Role in Recurrent Chromosomal
Translocations.”</span> <em>Cell</em> 148 (5): 908–21. <a href="https://doi.org/10.1016/j.cell.2012.02.002">https://doi.org/10.1016/j.cell.2012.02.002</a>.
</div>
<div id="ref-zheng_cistrome_2019" class="csl-entry" role="listitem">
Zheng, Rongbin, Changxin Wan, Shenglin Mei, Qian Qin, Qiu Wu, Hanfei
Sun, Chen-Hao Chen, et al. 2019. <span>“Cistrome <span>Data</span>
<span>Browser</span>: Expanded Datasets and New Tools for Gene
Regulatory Analysis.”</span> <em>Nucleic Acids Research</em> 47 (D1):
D729–35. <a href="https://doi.org/10.1093/nar/gky1094">https://doi.org/10.1093/nar/gky1094</a>.
</div>
<div id="ref-zheng_symphonizing_2022" class="csl-entry" role="listitem">
Zheng, Zhenxian, Shumin Li, Junhao Su, Amy Wing-Sze Leung, Tak-Wah Lam,
and Ruibang Luo. 2022. <span>“Symphonizing Pileup and Full-Alignment for
Deep Learning-Based Long-Read Variant Calling.”</span> <em>Nature
Computational Science</em> 2 (12): 797–803. <a href="https://doi.org/10.1038/s43588-022-00387-x">https://doi.org/10.1038/s43588-022-00387-x</a>.
</div>
<div id="ref-zhou_sequence-based_2022" class="csl-entry" role="listitem">
Zhou, Jian. 2022. <span>“Sequence-Based Modeling of Three-Dimensional
Genome Architecture from Kilobase to Chromosome Scale.”</span>
<em>Nature Genetics</em> 54 (5): 725–34. <a href="https://doi.org/10.1038/s41588-022-01065-4">https://doi.org/10.1038/s41588-022-01065-4</a>.
</div>
<div id="ref-zhou_expecto_2018" class="csl-entry" role="listitem">
Zhou, Jian, Chandra L. Theesfeld, Kevin Yao, Kathleen M. Chen, Aaron K.
Wong, and Olga G. Troyanskaya. 2018. <span>“[<span>Expecto</span>]
<span>Deep</span> Learning Sequence-Based Ab Initio Prediction of
Variant Effects on Expression and Disease Risk.”</span> <em>Nature
Genetics</em> 50 (8): 1171–79. <a href="https://doi.org/10.1038/s41588-018-0160-6">https://doi.org/10.1038/s41588-018-0160-6</a>.
</div>
<div id="ref-zhou_deepsea_2015" class="csl-entry" role="listitem">
Zhou, Jian, and Olga G. Troyanskaya. 2015. <span>“[<span>DeepSEA</span>]
<span>Predicting</span> Effects of Noncoding Variants with Deep
Learning–Based Sequence Model.”</span> <em>Nature Methods</em> 12 (10):
931–34. <a href="https://doi.org/10.1038/nmeth.3547">https://doi.org/10.1038/nmeth.3547</a>.
</div>
<div id="ref-zhou_dnabert-2_2024" class="csl-entry" role="listitem">
Zhou, Zhihan, Yanrong Ji, Weijian Li, Pratik Dutta, Ramana Davuluri, and
Han Liu. 2024. <span>“<span>DNABERT</span>-2: <span>Efficient</span>
<span>Foundation</span> <span>Model</span> and <span>Benchmark</span>
<span>For</span> <span>Multi</span>-<span>Species</span>
<span>Genome</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2306.15006">https://doi.org/10.48550/arXiv.2306.15006</a>.
</div>
<div id="ref-zhou_dnabert-s_2025" class="csl-entry" role="listitem">
Zhou, Zhihan, Weimin Wu, Harrison Ho, Jiayi Wang, Lizhen Shi, Ramana V
Davuluri, Zhong Wang, and Han Liu. 2025.
<span>“<span>DNABERT</span>-<span>S</span>: Pioneering Species
Differentiation with Species-Aware <span>DNA</span> Embeddings.”</span>
<em>Bioinformatics</em> 41 (Supplement_1): i255–64. <a href="https://doi.org/10.1093/bioinformatics/btaf188">https://doi.org/10.1093/bioinformatics/btaf188</a>.
</div>
<div id="ref-zhu_deep_2019" class="csl-entry" role="listitem">
Zhu, Ligeng, Zhijian Liu, and Song Han. 2019. <span>“Deep
<span>Leakage</span> from <span>Gradients</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.1906.08935">https://doi.org/10.48550/arXiv.1906.08935</a>.
</div>
<div id="ref-zitnik_modeling_2018" class="csl-entry" role="listitem">
Zitnik, Marinka, Monica Agrawal, and Jure Leskovec. 2018.
<span>“Modeling Polypharmacy Side Effects with Graph Convolutional
Networks.”</span> <em>Bioinformatics</em> 34 (13): i457–66. <a href="https://doi.org/10.1093/bioinformatics/bty294">https://doi.org/10.1093/bioinformatics/bty294</a>.
</div>
<div id="ref-zook_open_2019" class="csl-entry" role="listitem">
Zook, Justin M., Jennifer McDaniel, Nathan D. Olson, Justin Wagner,
Hemang Parikh, Haynes Heaton, Sean A. Irvine, et al. 2019. <span>“An
Open Resource for Accurately Benchmarking Small Variant and Reference
Calls.”</span> <em>Nature Biotechnology</em> 37 (5): 561–66. <a href="https://doi.org/10.1038/s41587-019-0074-6">https://doi.org/10.1038/s41587-019-0074-6</a>.
</div>
<div id="ref-zvyagin_genslms_2022" class="csl-entry" role="listitem">
Zvyagin, Maxim, Alexander Brace, Kyle Hippe, Yuntian Deng, Bin Zhang,
Cindy Orozco Bohorquez, Austin Clyde, et al. 2022.
<span>“<span>GenSLMs</span>: <span>Genome</span>-Scale Language Models
Reveal <span>SARS</span>-<span>CoV</span>-2 Evolutionary
Dynamics.”</span> bioRxiv. <a href="https://doi.org/10.1101/2022.10.10.511571">https://doi.org/10.1101/2022.10.10.511571</a>.
</div>
</div>



</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
    const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
    const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
    let newTheme = '';
    if(darkModeDefault) {
      newTheme = isAlternate ? baseTheme : alternateTheme;
    } else {
      newTheme = isAlternate ? alternateTheme : baseTheme;
    }
    const changeGiscusTheme = () => {
      // From: https://github.com/giscus/giscus/issues/336
      const sendMessage = (message) => {
        const iframe = document.querySelector('iframe.giscus-frame');
        if (!iframe) return;
        iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
      }
      sendMessage({
        setConfig: {
          theme: newTheme
        }
      });
    }
    const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
    if (isGiscussLoaded) {
      changeGiscusTheme();
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  const darkModeDefault = false;
  let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
    toggleGiscusIfUsed(toAlternate, darkModeDefault);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } else {
    toggleColorMode(false);
  }
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../part_7/p7-ch32-frontiers.html" class="pagination-link" aria-label="Frontiers and Synthesis">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">32</span>&nbsp; <span class="chapter-title">Frontiers and Synthesis</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../appendix/app-a-dl.html" class="pagination-link" aria-label="Deep Learning Primer">
        <span class="nav-page-text"><span class="chapter-number">A</span>&nbsp; <span class="chapter-title">Deep Learning Primer</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      &nbsp;
    </div>   
    <div class="nav-footer-center">
<p>Copyright 2025-2026, Josh Meehl</p>
</div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>




</body></html>