<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.8.26">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Appendix E — Glossary – Genomic Foundation Models</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./app-d-resources.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js" type="module"></script>
<script src="site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="site_libs/quarto-html/axe/axe-check.js" type="module"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-587c61ba64f3a5504c4d52d930310e48.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-dark-b758ccaa5987ceb1b75504551e579abf.css" rel="stylesheet" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-587c61ba64f3a5504c4d52d930310e48.css" rel="stylesheet" class="quarto-color-scheme-extra" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-a1553387a0f784068632030e9fbb8a3c.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="site_libs/bootstrap/bootstrap-dark-d1e9b63c6b6094879b9f94a7628e3370.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<link href="site_libs/bootstrap/bootstrap-a1553387a0f784068632030e9fbb8a3c.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme-extra" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-sidebar floating quarto-light"><script id="quarto-html-before-body" type="application/javascript">
    const toggleBodyColorMode = (bsSheetEl) => {
      const mode = bsSheetEl.getAttribute("data-mode");
      const bodyEl = window.document.querySelector("body");
      if (mode === "dark") {
        bodyEl.classList.add("quarto-dark");
        bodyEl.classList.remove("quarto-light");
      } else {
        bodyEl.classList.add("quarto-light");
        bodyEl.classList.remove("quarto-dark");
      }
    }
    const toggleBodyColorPrimary = () => {
      const bsSheetEl = window.document.querySelector("link#quarto-bootstrap:not([rel=disabled-stylesheet])");
      if (bsSheetEl) {
        toggleBodyColorMode(bsSheetEl);
      }
    }
    const setColorSchemeToggle = (alternate) => {
      const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
      for (let i=0; i < toggles.length; i++) {
        const toggle = toggles[i];
        if (toggle) {
          if (alternate) {
            toggle.classList.add("alternate");
          } else {
            toggle.classList.remove("alternate");
          }
        }
      }
    };
    const toggleColorMode = (alternate) => {
      // Switch the stylesheets
      const primaryStylesheets = window.document.querySelectorAll('link.quarto-color-scheme:not(.quarto-color-alternate)');
      const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
      manageTransitions('#quarto-margin-sidebar .nav-link', false);
      if (alternate) {
        // note: dark is layered on light, we don't disable primary!
        enableStylesheet(alternateStylesheets);
        for (const sheetNode of alternateStylesheets) {
          if (sheetNode.id === "quarto-bootstrap") {
            toggleBodyColorMode(sheetNode);
          }
        }
      } else {
        disableStylesheet(alternateStylesheets);
        enableStylesheet(primaryStylesheets)
        toggleBodyColorPrimary();
      }
      manageTransitions('#quarto-margin-sidebar .nav-link', true);
      // Switch the toggles
      setColorSchemeToggle(alternate)
      // Hack to workaround the fact that safari doesn't
      // properly recolor the scrollbar when toggling (#1455)
      if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
        manageTransitions("body", false);
        window.scrollTo(0, 1);
        setTimeout(() => {
          window.scrollTo(0, 0);
          manageTransitions("body", true);
        }, 40);
      }
    }
    const disableStylesheet = (stylesheets) => {
      for (let i=0; i < stylesheets.length; i++) {
        const stylesheet = stylesheets[i];
        stylesheet.rel = 'disabled-stylesheet';
      }
    }
    const enableStylesheet = (stylesheets) => {
      for (let i=0; i < stylesheets.length; i++) {
        const stylesheet = stylesheets[i];
        if(stylesheet.rel !== 'stylesheet') { // for Chrome, which will still FOUC without this check
          stylesheet.rel = 'stylesheet';
        }
      }
    }
    const manageTransitions = (selector, allowTransitions) => {
      const els = window.document.querySelectorAll(selector);
      for (let i=0; i < els.length; i++) {
        const el = els[i];
        if (allowTransitions) {
          el.classList.remove('notransition');
        } else {
          el.classList.add('notransition');
        }
      }
    }
    const isFileUrl = () => {
      return window.location.protocol === 'file:';
    }
    const hasAlternateSentinel = () => {
      let styleSentinel = getColorSchemeSentinel();
      if (styleSentinel !== null) {
        return styleSentinel === "alternate";
      } else {
        return false;
      }
    }
    const setStyleSentinel = (alternate) => {
      const value = alternate ? "alternate" : "default";
      if (!isFileUrl()) {
        window.localStorage.setItem("quarto-color-scheme", value);
      } else {
        localAlternateSentinel = value;
      }
    }
    const getColorSchemeSentinel = () => {
      if (!isFileUrl()) {
        const storageValue = window.localStorage.getItem("quarto-color-scheme");
        return storageValue != null ? storageValue : localAlternateSentinel;
      } else {
        return localAlternateSentinel;
      }
    }
    const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
      const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
      const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
      let newTheme = '';
      if(authorPrefersDark) {
        newTheme = isAlternate ? baseTheme : alternateTheme;
      } else {
        newTheme = isAlternate ? alternateTheme : baseTheme;
      }
      const changeGiscusTheme = () => {
        // From: https://github.com/giscus/giscus/issues/336
        const sendMessage = (message) => {
          const iframe = document.querySelector('iframe.giscus-frame');
          if (!iframe) return;
          iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
        }
        sendMessage({
          setConfig: {
            theme: newTheme
          }
        });
      }
      const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
      if (isGiscussLoaded) {
        changeGiscusTheme();
      }
    };
    const authorPrefersDark = false;
    const darkModeDefault = authorPrefersDark;
      document.querySelector('link#quarto-text-highlighting-styles.quarto-color-scheme-extra').rel = 'disabled-stylesheet';
      document.querySelector('link#quarto-bootstrap.quarto-color-scheme-extra').rel = 'disabled-stylesheet';
    let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
    // Dark / light mode switch
    window.quartoToggleColorScheme = () => {
      // Read the current dark / light value
      let toAlternate = !hasAlternateSentinel();
      toggleColorMode(toAlternate);
      setStyleSentinel(toAlternate);
      toggleGiscusIfUsed(toAlternate, darkModeDefault);
      window.dispatchEvent(new Event('resize'));
    };
    // Switch to dark mode if need be
    if (hasAlternateSentinel()) {
      toggleColorMode(true);
    } else {
      toggleColorMode(false);
    }
  </script>

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./app-a-dl.html">Appendices</a></li><li class="breadcrumb-item"><a href="./app-e-glossary.html"><span class="chapter-number">E</span>&nbsp; <span class="chapter-title">Glossary</span></a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="./">Genomic Foundation Models</a> 
        <div class="sidebar-tools-main">
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Introduction</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./preface.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Preface</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="./p1--foundations.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Part I: Foundations</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p1-ch01-ngs.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Sequencing: From Reads to Variants</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p1-ch02-data.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">The Genomic Data Landscape</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p1-ch03-pgs.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">GWAS &amp; Polygenic Scores</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p1-ch04-cadd.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Deleteriousness Scores</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="./p2--architectures.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Part II: Deep Learning Architectures</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p2-ch05-cnn.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">CNN Sequence-to-Function Models</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p2-ch06-plm.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Protein Language Models</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p2-ch07-dna.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Genomic Foundation Models</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p2-ch08-rna.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">RNA &amp; Transcript-Level Models</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p2-ch09-hybrid.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Long-range Hybrid Models</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="./p3--principles.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Part III: Core Principles</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p3-ch10-tokens.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Sequence Representation &amp; Tokens</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p3-ch11-principles.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Genomic FMs: Principles &amp; Practice</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p3-ch12-vep.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">12</span>&nbsp; <span class="chapter-title">Cross-Protein Transfer from Deep Mutational Scanning</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="./p4--multi-scale.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Part IV: Multi-Scale Modeling</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p4-ch13-sc-epi.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">13</span>&nbsp; <span class="chapter-title">Single-Cell &amp; Epigenomic Foundation Models</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p4-ch14-networks.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">14</span>&nbsp; <span class="chapter-title">Chapter 14: Graphs, Networks, and Deep Learning</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p4-ch15-systems.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">15</span>&nbsp; <span class="chapter-title">Multi-Omics Integration &amp; Systems Biology</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="./p5--eval-interp.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Part V: Evaluation and Reliability</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-5" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p5-ch16-benchmarks.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">16</span>&nbsp; <span class="chapter-title">Benchmarks in Genomic Deep Learning</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p5-ch17-eval.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">17</span>&nbsp; <span class="chapter-title">Model Evaluation &amp; Benchmarks</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p5-ch18-confound.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">18</span>&nbsp; <span class="chapter-title">Confounders in Model Training</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p5-ch19-interp.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">19</span>&nbsp; <span class="chapter-title">Interpretability &amp; Mechanisms</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="./p6--translation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Part VI — Translation and Application</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-6" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p6-ch20-clinical.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">20</span>&nbsp; <span class="chapter-title">Clinical Risk Prediction</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p6-ch21-variants.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">21</span>&nbsp; <span class="chapter-title">Pathogenic Variant Discovery</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p6-ch22-drugs.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">22</span>&nbsp; <span class="chapter-title">Drug Discovery &amp; Biotech</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p6-ch23-design.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">23</span>&nbsp; <span class="chapter-title">Design Applications</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p6-ch24-future.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">24</span>&nbsp; <span class="chapter-title">Chapter 24: Future Directions &amp; Ethical Considerations</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./references.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">References</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-7" role="navigation" aria-expanded="true">
 <span class="menu-text">Appendices</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-7" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-7" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./app-a-dl.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">A</span>&nbsp; <span class="chapter-title">Deep Learning Primer</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./app-b-deployment.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">B</span>&nbsp; <span class="chapter-title">Model Deployment</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./app-c-model-list.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">C</span>&nbsp; <span class="chapter-title">Referenced Models</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./app-d-resources.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">D</span>&nbsp; <span class="chapter-title">Additional Resources</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./app-e-glossary.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">E</span>&nbsp; <span class="chapter-title">Glossary</span></span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#ch-01" id="toc-ch-01" class="nav-link active" data-scroll-target="#ch-01"><span class="header-section-number">E.1</span> CH 01</a>
  <ul class="collapse">
  <li><a href="#sequencing-technologies-data" id="toc-sequencing-technologies-data" class="nav-link" data-scroll-target="#sequencing-technologies-data"><span class="header-section-number">E.1.1</span> Sequencing Technologies &amp; Data</a></li>
  <li><a href="#targeting-strategies" id="toc-targeting-strategies" class="nav-link" data-scroll-target="#targeting-strategies"><span class="header-section-number">E.1.2</span> Targeting Strategies</a></li>
  <li><a href="#alignment-processing" id="toc-alignment-processing" class="nav-link" data-scroll-target="#alignment-processing"><span class="header-section-number">E.1.3</span> Alignment &amp; Processing</a></li>
  <li><a href="#variant-calling" id="toc-variant-calling" class="nav-link" data-scroll-target="#variant-calling"><span class="header-section-number">E.1.4</span> Variant Calling</a></li>
  <li><a href="#phasing" id="toc-phasing" class="nav-link" data-scroll-target="#phasing"><span class="header-section-number">E.1.5</span> Phasing</a></li>
  <li><a href="#variant-types" id="toc-variant-types" class="nav-link" data-scroll-target="#variant-types"><span class="header-section-number">E.1.6</span> Variant Types</a></li>
  <li><a href="#difficult-regions" id="toc-difficult-regions" class="nav-link" data-scroll-target="#difficult-regions"><span class="header-section-number">E.1.7</span> Difficult Regions</a></li>
  <li><a href="#benchmarking" id="toc-benchmarking" class="nav-link" data-scroll-target="#benchmarking"><span class="header-section-number">E.1.8</span> Benchmarking</a></li>
  <li><a href="#key-resourcestools-may-warrant-brief-glossary-entries" id="toc-key-resourcestools-may-warrant-brief-glossary-entries" class="nav-link" data-scroll-target="#key-resourcestools-may-warrant-brief-glossary-entries"><span class="header-section-number">E.1.9</span> Key Resources/Tools (may warrant brief glossary entries)</a></li>
  </ul></li>
  <li><a href="#ch-02" id="toc-ch-02" class="nav-link" data-scroll-target="#ch-02"><span class="header-section-number">E.2</span> CH 02</a>
  <ul class="collapse">
  <li><a href="#reference-coordinate-systems" id="toc-reference-coordinate-systems" class="nav-link" data-scroll-target="#reference-coordinate-systems"><span class="header-section-number">E.2.1</span> Reference &amp; Coordinate Systems</a></li>
  <li><a href="#variant-types-properties" id="toc-variant-types-properties" class="nav-link" data-scroll-target="#variant-types-properties"><span class="header-section-number">E.2.2</span> Variant Types &amp; Properties</a></li>
  <li><a href="#population-genetics-metrics" id="toc-population-genetics-metrics" class="nav-link" data-scroll-target="#population-genetics-metrics"><span class="header-section-number">E.2.3</span> Population Genetics Metrics</a></li>
  <li><a href="#functional-genomics" id="toc-functional-genomics" class="nav-link" data-scroll-target="#functional-genomics"><span class="header-section-number">E.2.4</span> Functional Genomics</a></li>
  <li><a href="#expression-genetics" id="toc-expression-genetics" class="nav-link" data-scroll-target="#expression-genetics"><span class="header-section-number">E.2.5</span> Expression Genetics</a></li>
  <li><a href="#clinical-interpretation" id="toc-clinical-interpretation" class="nav-link" data-scroll-target="#clinical-interpretation"><span class="header-section-number">E.2.6</span> Clinical Interpretation</a></li>
  <li><a href="#study-designs-statistics" id="toc-study-designs-statistics" class="nav-link" data-scroll-target="#study-designs-statistics"><span class="header-section-number">E.2.7</span> Study Designs &amp; Statistics</a></li>
  </ul></li>
  <li><a href="#ch-03" id="toc-ch-03" class="nav-link" data-scroll-target="#ch-03"><span class="header-section-number">E.3</span> CH 03</a></li>
  <li><a href="#ch-04" id="toc-ch-04" class="nav-link" data-scroll-target="#ch-04"><span class="header-section-number">E.4</span> CH 04</a></li>
  <li><a href="#ch-05" id="toc-ch-05" class="nav-link" data-scroll-target="#ch-05"><span class="header-section-number">E.5</span> CH 05</a></li>
  <li><a href="#ch-06" id="toc-ch-06" class="nav-link" data-scroll-target="#ch-06"><span class="header-section-number">E.6</span> CH 06</a></li>
  <li><a href="#ch-07" id="toc-ch-07" class="nav-link" data-scroll-target="#ch-07"><span class="header-section-number">E.7</span> CH 07</a></li>
  <li><a href="#ch-08" id="toc-ch-08" class="nav-link" data-scroll-target="#ch-08"><span class="header-section-number">E.8</span> CH 08</a></li>
  <li><a href="#ch-09" id="toc-ch-09" class="nav-link" data-scroll-target="#ch-09"><span class="header-section-number">E.9</span> CH 09</a></li>
  <li><a href="#ch-10" id="toc-ch-10" class="nav-link" data-scroll-target="#ch-10"><span class="header-section-number">E.10</span> CH 10</a></li>
  <li><a href="#ch-11" id="toc-ch-11" class="nav-link" data-scroll-target="#ch-11"><span class="header-section-number">E.11</span> CH 11</a></li>
  <li><a href="#ch-12" id="toc-ch-12" class="nav-link" data-scroll-target="#ch-12"><span class="header-section-number">E.12</span> CH 12</a></li>
  <li><a href="#ch-13" id="toc-ch-13" class="nav-link" data-scroll-target="#ch-13"><span class="header-section-number">E.13</span> CH 13</a></li>
  <li><a href="#ch-14" id="toc-ch-14" class="nav-link" data-scroll-target="#ch-14"><span class="header-section-number">E.14</span> CH 14</a></li>
  <li><a href="#ch-15" id="toc-ch-15" class="nav-link" data-scroll-target="#ch-15"><span class="header-section-number">E.15</span> CH 15</a></li>
  <li><a href="#ch-16" id="toc-ch-16" class="nav-link" data-scroll-target="#ch-16"><span class="header-section-number">E.16</span> CH 16</a></li>
  <li><a href="#ch-17" id="toc-ch-17" class="nav-link" data-scroll-target="#ch-17"><span class="header-section-number">E.17</span> CH 17</a></li>
  <li><a href="#ch-18" id="toc-ch-18" class="nav-link" data-scroll-target="#ch-18"><span class="header-section-number">E.18</span> CH 18</a></li>
  <li><a href="#ch-19" id="toc-ch-19" class="nav-link" data-scroll-target="#ch-19"><span class="header-section-number">E.19</span> CH 19</a></li>
  <li><a href="#ch-20" id="toc-ch-20" class="nav-link" data-scroll-target="#ch-20"><span class="header-section-number">E.20</span> CH 20</a></li>
  <li><a href="#apx-a" id="toc-apx-a" class="nav-link" data-scroll-target="#apx-a"><span class="header-section-number">E.21</span> APX A</a></li>
  <li><a href="#apx-b" id="toc-apx-b" class="nav-link" data-scroll-target="#apx-b"><span class="header-section-number">E.22</span> APX B</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">


<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./app-a-dl.html">Appendices</a></li><li class="breadcrumb-item"><a href="./app-e-glossary.html"><span class="chapter-number">E</span>&nbsp; <span class="chapter-title">Glossary</span></a></li></ol></nav>
<div class="quarto-title">
<h1 class="title"><span id="sec-apx-glossary" class="quarto-section-identifier">Appendix E — Glossary</span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="ch-01" class="level2" data-number="E.1">
<h2 data-number="E.1" class="anchored" data-anchor-id="ch-01"><span class="header-section-number">E.1</span> CH 01</h2>
<section id="sequencing-technologies-data" class="level3" data-number="E.1.1">
<h3 data-number="E.1.1" class="anchored" data-anchor-id="sequencing-technologies-data"><span class="header-section-number">E.1.1</span> Sequencing Technologies &amp; Data</h3>
<section id="glo-next-generation-sequencing-ngs" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-next-generation-sequencing-ngs">Next-generation sequencing (NGS)</h4>
<p>High-throughput DNA sequencing technologies that allow rapid…stretches of DNA, producing millions of short reads in parallel….</p>
</section>
<section id="glo-illumina-sequencing" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-illumina-sequencing">Illumina sequencing</h4>
<p>A widely used NGS technology that utilizes reversible dye-terminators to sequence DNA by synthesis</p>
</section>
<section id="glo-short-reads-paired-end-reads" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-short-reads-paired-end-reads">Short reads / Paired-end reads</h4>
<p>DNA sequences generated by NGS technologies, typically rangi… a DNA fragment, providing additional information for alignment.</p>
</section>
<section id="glo-long-read-sequencing-pacbio-hifi-oxford-nanopore" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-long-read-sequencing-pacbio-hifi-oxford-nanopore">Long-read sequencing (PacBio HiFi, Oxford Nanopore)</h4>
<p>DNA sequencing technologies that produce longer reads, typic…ases, allowing for better resolution of complex genomic regions.</p>
</section>
<section id="glo-circular-consensus-sequencing" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-circular-consensus-sequencing">Circular consensus sequencing</h4>
<p>A sequencing method used by PacBio to generate highly accura… long reads by repeatedly sequencing the same DNA molecule.</p>
</section>
<section id="glo-base-calling" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-base-calling">Base calling</h4>
<p>The process of determining the nucleotide sequence from raw sequencing data.</p>
</section>
<section id="glo-fastq" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-fastq">FASTQ</h4>
<p>A file format that stores both nucleotide sequences and their corresponding quality scores.</p>
</section>
<section id="glo-read-depth-coverage-e-g-30-100" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-read-depth-coverage-e-g-30-100">Read depth / Coverage (e.g., 30×, 100×)</h4>
<p>The number of times a particular nucleotide is sequenced, indicating the reliability of the sequencing data.</p>
</section>
</section>
<section id="targeting-strategies" class="level3" data-number="E.1.2">
<h3 data-number="E.1.2" class="anchored" data-anchor-id="targeting-strategies"><span class="header-section-number">E.1.2</span> Targeting Strategies</h3>
<section id="glo-targeted-gene-panel" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-targeted-gene-panel">Targeted gene panel</h4>
<p>A sequencing approach that focuses on a specific set of gene…or cost-effective analysis of known disease-associated variants.</p>
</section>
<section id="glo-whole-exome-sequencing-wes" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-whole-exome-sequencing-wes">Whole-exome sequencing (WES)</h4>
<p>A sequencing approach that targets all protein-coding region…the genome, providing a comprehensive view of coding variants.</p>
</section>
<section id="glo-whole-genome-sequencing-wgs" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-whole-genome-sequencing-wgs">Whole-genome sequencing (WGS)</h4>
<p>A sequencing approach that captures the entire genome, inclu…ng regions, providing the most comprehensive view of genetic var</p>
</section>
<section id="glo-capture-efficiency" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-capture-efficiency">Capture efficiency</h4>
<p>The effectiveness of a targeted sequencing approach in enric…interest, impacting the overall quality and coverage of the data.</p>
</section>
</section>
<section id="alignment-processing" class="level3" data-number="E.1.3">
<h3 data-number="E.1.3" class="anchored" data-anchor-id="alignment-processing"><span class="header-section-number">E.1.3</span> Alignment &amp; Processing</h3>
<section id="glo-read-alignment-mapping" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-read-alignment-mapping">Read alignment / Mapping</h4>
<p>The process of aligning sequencing reads to a reference genome to determine their genomic origin.</p>
</section>
<section id="glo-seed-and-extend-alignment" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-seed-and-extend-alignment">Seed-and-extend alignment</h4>
<p>An algorithmic approach for read alignment that first identi…ences (seeds) and then extends the alignment around these seeds.</p>
</section>
<section id="glo-pcr-duplicates" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-pcr-duplicates">PCR duplicates</h4>
<p>Identical sequencing reads that originate from the same DNA …esulting from PCR amplification, which can bias variant calling.</p>
</section>
<section id="glo-base-quality-score-recalibration-bqsr" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-base-quality-score-recalibration-bqsr">Base quality score recalibration (BQSR)</h4>
<p>A process that adjusts the quality scores of sequencing read…or systematic errors made by the sequencer.</p>
</section>
<section id="glo-mapping-quality" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-mapping-quality">Mapping quality</h4>
<p>A measure of the confidence that a read is correctly aligned to the reference genome.</p>
</section>
<section id="glo-reference-bias" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-reference-bias">Reference bias</h4>
<p>The tendency for sequencing and alignment processes to preferentially detect alleles present in the reference genome.</p>
</section>
</section>
<section id="variant-calling" class="level3" data-number="E.1.4">
<h3 data-number="E.1.4" class="anchored" data-anchor-id="variant-calling"><span class="header-section-number">E.1.4</span> Variant Calling</h3>
<section id="glo-variant-calling" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-variant-calling">Variant calling</h4>
<p>The process of identifying variants from sequencing data by comparing it to a reference genome.</p>
</section>
<section id="glo-genotype-likelihood" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-genotype-likelihood">Genotype likelihood</h4>
<p>The probability of observing the sequencing data given a particular genotype.</p>
</section>
<section id="glo-pair-hmm-pair-hidden-markov-model" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-pair-hmm-pair-hidden-markov-model">Pair-HMM (pair hidden Markov model)</h4>
<p>A statistical model used in variant calling to calculate the likelihood of different alignments between reads and the reference genome.</p>
</section>
<section id="glo-joint-genotyping-cohort-calling" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-joint-genotyping-cohort-calling">Joint genotyping / Cohort calling</h4>
<p>The process of simultaneously calling variants across multiple samples to improve accuracy and consistency.</p>
</section>
<section id="glo-gvcf-genomic-vcf" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-gvcf-genomic-vcf">gVCF (genomic VCF)</h4>
<p>A variant call format that includes information about both va…sites, allowing for joint genotyping.</p>
</section>
<section id="glo-vcf-variant-call-format" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-vcf-variant-call-format">VCF (variant call format)</h4>
<p>A standardized file format for storing variant information, including SNPs, indels, and structural variants.</p>
</section>
<section id="glo-vqsr-variant-quality-score-recalibration" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-vqsr-variant-quality-score-recalibration">VQSR (Variant Quality Score Recalibration)</h4>
<p>A method for improving the accuracy of variant calls by mode…lationship between variant quality scores and various annotatio</p>
</section>
<section id="glo-pileup" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-pileup">Pileup</h4>
<p>A summary of the base calls at each position in a set of align…ing reads, used for variant calling and visualization.</p>
</section>
</section>
<section id="phasing" class="level3" data-number="E.1.5">
<h3 data-number="E.1.5" class="anchored" data-anchor-id="phasing"><span class="header-section-number">E.1.5</span> Phasing</h3>
<section id="glo-haplotype-phasing" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-haplotype-phasing">Haplotype phasing</h4>
<p>The process of determining which variants are inherited together on the same chromosome.</p>
</section>
<section id="glo-read-backed-phasing" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-read-backed-phasing">Read-backed phasing</h4>
<p>A method of phasing that uses sequencing reads that span multiple variants to determine their phase.</p>
</section>
<section id="glo-statistical-phasing" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-statistical-phasing">Statistical phasing</h4>
<p>A method of phasing that uses population-level genotype data and statistical models to infer haplotypes.</p>
</section>
<section id="glo-compound-heterozygosity" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-compound-heterozygosity">Compound heterozygosity</h4>
<p>The presence of two different variants at a particular gene locus, one on each chromosome of a pair.</p>
</section>
<section id="glo-cis-vs-trans-configuration" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-cis-vs-trans-configuration">Cis vs.&nbsp;trans configuration</h4>
<p>Describes the relative arrangement of two variants on the same chromosome (cis) or on different chromosomes (trans).</p>
</section>
</section>
<section id="variant-types" class="level3" data-number="E.1.6">
<h3 data-number="E.1.6" class="anchored" data-anchor-id="variant-types"><span class="header-section-number">E.1.6</span> Variant Types</h3>
<section id="glo-snv-single-nucleotide-variant" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-snv-single-nucleotide-variant">SNV (single nucleotide variant)</h4>
<p>A variation in a single nucleotide that occurs at a specific position in the genome.</p>
</section>
<section id="glo-indel" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-indel">Indel</h4>
<p>An insertion or deletion of bases in the genome of an organism.</p>
</section>
<section id="glo-structural-variant" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-structural-variant">Structural variant</h4>
<p>A large-scale alteration in the genome, such as a deletion, duplication, inversion, or translocation.</p>
</section>
<section id="glo-multi-nucleotide-variant-mnv" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-multi-nucleotide-variant-mnv">Multi-nucleotide variant (MNV)</h4>
<p>A variation that affects multiple consecutive nucleotides in the genome.</p>
</section>
<section id="glo-mosaic-variant" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-mosaic-variant">Mosaic variant</h4>
<p>A genetic variant that is present in some but not all cells of an organism, often arising during development.</p>
</section>
<section id="glo-somatic-variant" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-somatic-variant">Somatic variant</h4>
<p>A genetic variant that occurs in non-germline cells and is not inherited, often associated with cancer.</p>
</section>
<section id="glo-germline-variant" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-germline-variant">Germline variant</h4>
<p>A genetic variant that is present in the egg or sperm and can be passed on to offspring.</p>
</section>
<section id="glo-de-novo-variant" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-de-novo-variant">De novo variant</h4>
<p>A genetic variant that arises spontaneously in an individual and is not inherited from either parent.</p>
</section>
</section>
<section id="difficult-regions" class="level3" data-number="E.1.7">
<h3 data-number="E.1.7" class="anchored" data-anchor-id="difficult-regions"><span class="header-section-number">E.1.7</span> Difficult Regions</h3>
<section id="glo-segmental-duplication" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-segmental-duplication">Segmental duplication</h4>
<p>Large, highly similar sequences in the genome that can complicate read alignment and variant calling.</p>
</section>
<section id="glo-paralog-paralogous-gene" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-paralog-paralogous-gene">Paralog / Paralogous gene</h4>
<p>A gene that is related to another gene in the same organism due to a duplication event.</p>
</section>
<section id="glo-homopolymer" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-homopolymer">Homopolymer</h4>
<p>A sequence of identical nucleotides in a row, which can be prone to sequencing errors.</p>
</section>
<section id="glo-low-complexity-region" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-low-complexity-region">Low-complexity region</h4>
<p>A region of the genome with a simple sequence composition, which can be challenging for alignment and variant calling.</p>
</section>
<section id="glo-hla-region-mhc" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-hla-region-mhc">HLA region / MHC</h4>
<p>The human leukocyte antigen (HLA) region, also known as the major histocompatibility complex (MHC), is a highly polymorphic region involved in immune response.</p>
</section>
</section>
<section id="benchmarking" class="level3" data-number="E.1.8">
<h3 data-number="E.1.8" class="anchored" data-anchor-id="benchmarking"><span class="header-section-number">E.1.8</span> Benchmarking</h3>
<section id="glo-precision-positive-predictive-value" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-precision-positive-predictive-value">Precision (positive predictive value)</h4>
<p>The proportion of true positive variant calls among all positive calls.</p>
</section>
<section id="glo-recall-sensitivity" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-recall-sensitivity">Recall (sensitivity)</h4>
<p>The proportion of true positive variant calls detected among all actual variants.</p>
</section>
<section id="glo-f1-score" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-f1-score">F1 score</h4>
<p>The harmonic mean of precision and recall, providing a single metric for evaluating variant calling performance.</p>
</section>
<section id="glo-true-positive-tp-false-positive-fp-false-negative-fn" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-true-positive-tp-false-positive-fp-false-negative-fn">True positive (TP) / False positive (FP) / False negative (FN)</h4>
<p>Metrics used to evaluate the accuracy of variant calls, where TP represents correctly identified variants, FP represents incorrectly identified variants, and FN represents missed variants.</p>
</section>
<section id="glo-high-confidence-region" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-high-confidence-region">High-confidence region</h4>
<p>A region of the genome where variant calls are considered to b… reliable, often used for benchmarking and validation.</p>
</section>
</section>
<section id="key-resourcestools-may-warrant-brief-glossary-entries" class="level3" data-number="E.1.9">
<h3 data-number="E.1.9" class="anchored" data-anchor-id="key-resourcestools-may-warrant-brief-glossary-entries"><span class="header-section-number">E.1.9</span> Key Resources/Tools (may warrant brief glossary entries)</h3>
<section id="glo-giab-genome-in-a-bottle" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-giab-genome-in-a-bottle">GIAB (Genome in a Bottle)</h4>
<p>A consortium that develops reference materials and data for benchmarking genome sequencing and variant calling.</p>
</section>
<section id="glo-deepvariant" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-deepvariant">DeepVariant</h4>
<p>A deep learning-based variant caller developed by Google that identifies genetic variants from sequencing data.</p>
</section>
<section id="glo-glnexus" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-glnexus">GLnexus</h4>
<p>A tool for joint variant calling across multiple samples, designed to work with DeepVariant outputs.</p>
</section>
<section id="glo-haplotypecaller" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-haplotypecaller">HaplotypeCaller</h4>
<p>A variant caller from the Genome Analysis Toolkit (GATK) that uses local de-novo assembly of haplotypes to call variants.</p>
</section>
</section>
</section>
<section id="ch-02" class="level2" data-number="E.2">
<h2 data-number="E.2" class="anchored" data-anchor-id="ch-02"><span class="header-section-number">E.2</span> CH 02</h2>
<section id="reference-coordinate-systems" class="level3" data-number="E.2.1">
<h3 data-number="E.2.1" class="anchored" data-anchor-id="reference-coordinate-systems"><span class="header-section-number">E.2.1</span> Reference &amp; Coordinate Systems</h3>
<section id="glo-reference-genome-assembly" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-reference-genome-assembly">Reference genome/assembly</h4>
<p>A digital nucleic acid sequence database, assembled as a repre…example of a species’ set of genes. Multiple versions exist.</p>
</section>
<section id="glo-grch37" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-grch37">GRCh37</h4>
<p>The 37th version of the Genome Reference Consortium human genome assembly.</p>
</section>
<section id="glo-grch38" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-grch38">GRCh38</h4>
<p>The 38th version of the Genome Reference Consortium human genome assembly.</p>
</section>
<section id="glo-t2t-chm13" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-t2t-chm13">T2T-CHM13</h4>
<p>The Telomere-to-Telomere (T2T) CHM13 human genome assembly, r…ting a complete, gapless sequence of a human genome.</p>
</section>
<section id="glo-pangenome-reference" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-pangenome-reference">Pangenome reference</h4>
<p>A reference that represents the genetic diversity of a species, rather than a single individual.</p>
</section>
<section id="glo-gene-model" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-gene-model">Gene model</h4>
<p>A representation of the structure of a gene, including its exons, introns, and regulatory elements.</p>
</section>
<section id="glo-canonical-transcript" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-canonical-transcript">Canonical transcript</h4>
<p>The most biologically relevant transcript of a gene, often used as the reference for annotation.</p>
</section>
<section id="glo-alternative-transcript-isoform" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-alternative-transcript-isoform">Alternative transcript/isoform</h4>
<p>Different versions of a transcript produced from the same gene due to alternative splicing or other mechanisms.</p>
</section>
<section id="glo-mane-select" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-mane-select">MANE Select</h4>
<p>Matched Annotation from NCBI and EMBL-EBI (MANE) Select is a …cripts that are consistently annotated across databases.</p>
</section>
</section>
<section id="variant-types-properties" class="level3" data-number="E.2.2">
<h3 data-number="E.2.2" class="anchored" data-anchor-id="variant-types-properties"><span class="header-section-number">E.2.2</span> Variant Types &amp; Properties</h3>
<section id="glo-allele-frequency" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-allele-frequency">Allele frequency</h4>
<p>The proportion of a specific allele among all alleles of a gene in a population.</p>
</section>
<section id="glo-maf-minor-allele-frequency" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-maf-minor-allele-frequency">MAF (minor allele frequency)</h4>
<p>The frequency at which the less common allele occurs in a given population.</p>
</section>
<section id="glo-rsid" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-rsid">rsID</h4>
<p>A unique identifier assigned to a single nucleotide polymorphism (SNP) in the dbSNP database.</p>
</section>
<section id="glo-loss-of-function-lof-variant" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-loss-of-function-lof-variant">Loss-of-function (LoF) variant</h4>
<p>A genetic variant that results in reduced or abolished protein function.</p>
</section>
<section id="glo-ultra-rare-variant" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-ultra-rare-variant">Ultra-rare variant</h4>
<p>A genetic variant that is extremely uncommon in the population, often with a frequency of less than 0.01%.</p>
</section>
</section>
<section id="population-genetics-metrics" class="level3" data-number="E.2.3">
<h3 data-number="E.2.3" class="anchored" data-anchor-id="population-genetics-metrics"><span class="header-section-number">E.2.3</span> Population Genetics Metrics</h3>
<section id="glo-ld" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-ld">Linkage disequilibrium</h4>
<p>A non-random association of alleles at different loci in a given population.</p>
</section>
<section id="glo-pli-probability-of-being-loss-of-function-intolerant" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-pli-probability-of-being-loss-of-function-intolerant">pLI (probability of being loss-of-function intolerant)</h4>
<p>A metric that estimates the likelihood that a gene is intolerant to loss-of-function variants.</p>
</section>
<section id="glo-loeuf-loss-of-function-observed-expected-upper-bound-fraction" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-loeuf-loss-of-function-observed-expected-upper-bound-fraction">LOEUF (loss-of-function observed/expected upper bound fraction)</h4>
<p>A metric that quantifies the observed versus expected number of loss-of-function variants in a gene.</p>
</section>
<section id="glo-constraint-metrics" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-constraint-metrics">Constraint metrics</h4>
<p>Metrics that assess the tolerance of a gene to functional genetic variation.</p>
</section>
<section id="glo-imputation" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-imputation">Imputation</h4>
<p>The process of inferring unobserved genotypes in a study sample based on observed genotypes and a reference panel.</p>
</section>
</section>
<section id="functional-genomics" class="level3" data-number="E.2.4">
<h3 data-number="E.2.4" class="anchored" data-anchor-id="functional-genomics"><span class="header-section-number">E.2.4</span> Functional Genomics</h3>
<section id="glo-chip-seq" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-chip-seq">ChIP-seq</h4>
<p>Chromatin Immunoprecipitation followed by sequencing, a method used to analyze protein-DNA interactions.</p>
</section>
<section id="glo-dnase-seq" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-dnase-seq">DNase-seq</h4>
<p>A method to identify regions of open chromatin by sequencing DNA fragments generated by DNase I digestion.</p>
</section>
<section id="glo-atac-seq" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-atac-seq">ATAC-seq</h4>
<p>Assay for Transposase-Accessible Chromatin using sequencing, a technique to study chromatin accessibility.</p>
</section>
<section id="glo-hi-c" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-hi-c">Hi-C</h4>
<p>A method to study the three-dimensional architecture of genomes by capturing chromatin interactions.</p>
</section>
<section id="glo-chromatin-accessibility" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-chromatin-accessibility">Chromatin accessibility</h4>
<p>The degree to which DNA is exposed and available for binding by proteins, often assessed by DNase-seq or ATAC-seq.</p>
</section>
<section id="glo-histone-modification" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-histone-modification">Histone modification</h4>
<p>Chemical modifications to histone proteins that can influence chromatin structure and gene expression.</p>
</section>
<section id="glo-peak-calling" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-peak-calling">Peak calling</h4>
<p>The process of identifying regions of the genome with signific…ment of sequencing reads, often used in ChIP-seq and ATAC-seq analyses.</p>
</section>
<section id="glo-signal-track" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-signal-track">Signal track</h4>
<p>A graphical representation of sequencing data across the genom… intensity of signals such as read coverage or enrichment.</p>
</section>
</section>
<section id="expression-genetics" class="level3" data-number="E.2.5">
<h3 data-number="E.2.5" class="anchored" data-anchor-id="expression-genetics"><span class="header-section-number">E.2.5</span> Expression Genetics</h3>
<section id="glo-eqtl-expression-quantitative-trait-locus" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-eqtl-expression-quantitative-trait-locus">eQTL (expression quantitative trait locus)</h4>
<p>A genomic locus that explains variation in gene expression levels.</p>
</section>
<section id="glo-splicing-qtl" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-splicing-qtl">Splicing QTL</h4>
<p>A genomic locus that affects the splicing of pre-mRNA.</p>
</section>
<section id="glo-molecular-qtl" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-molecular-qtl">Molecular QTL</h4>
<p>A quantitative trait locus that influences molecular traits such as gene expression, protein levels, or metabolite concentrations.</p>
</section>
<section id="glo-cis-regulatory" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-cis-regulatory">Cis-regulatory</h4>
<p>Referring to regulatory elements, such as promoters or enhanc…ated on the same molecule of DNA as the gene they regulate.</p>
</section>
<section id="glo-colocalization" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-colocalization">Colocalization</h4>
<p>The occurrence of two or more genetic signals at the same genomic location, suggesting a shared causal variant.</p>
</section>
<section id="glo-dropout-single-cell-context" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-dropout-single-cell-context">Dropout (single-cell context)</h4>
<p>The failure to detect a transcript in a single-cell RNA-seq ex…often due to low mRNA capture efficiency.</p>
</section>
</section>
<section id="clinical-interpretation" class="level3" data-number="E.2.6">
<h3 data-number="E.2.6" class="anchored" data-anchor-id="clinical-interpretation"><span class="header-section-number">E.2.6</span> Clinical Interpretation</h3>
<section id="glo-acmg-amp-criteria" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-acmg-amp-criteria">ACMG/AMP criteria</h4>
<p>A set of guidelines developed by the American College of Medic…ogy (AMP) for the interpretation of sequence variants. These c…vide a framework for classifying variants into categories such path</p>
</section>
<section id="glo-pathogenicity" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-pathogenicity">Pathogenicity</h4>
<p>The ability of a genetic variant to cause disease.</p>
</section>
<section id="glo-haploinsufficiency" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-haploinsufficiency">Haploinsufficiency</h4>
<p>A condition in which a single functional copy of a gene is in…maintain normal function, leading to a disease phenotype.</p>
</section>
<section id="glo-triplosensitivity" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-triplosensitivity">Triplosensitivity</h4>
<p>A condition in which an extra copy of a gene leads to a disease phenotype.</p>
</section>
<section id="glo-gene-disease-validity" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-gene-disease-validity">Gene-disease validity</h4>
<p>The strength of evidence supporting a relationship between a gene and a disease.</p>
</section>
<section id="glo-pharmacogenomics" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-pharmacogenomics">Pharmacogenomics</h4>
<p>The study of how genetic variation affects an individual’s response to drugs.</p>
</section>
<section id="glo-diplotype" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-diplotype">Diplotype</h4>
<p>The combination of alleles at multiple loci on a single chromosome that are inherited together.</p>
</section>
</section>
<section id="study-designs-statistics" class="level3" data-number="E.2.7">
<h3 data-number="E.2.7" class="anchored" data-anchor-id="study-designs-statistics"><span class="header-section-number">E.2.7</span> Study Designs &amp; Statistics</h3>
<section id="glo-gwas-summary-statistics" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-gwas-summary-statistics">GWAS summary statistics</h4>
<p>Aggregated data from genome-wide association studies, typicall…ormation on the association between genetic variants and traits across the genome.</p>
</section>
<section id="glo-fine-mapping" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-fine-mapping">Fine-mapping</h4>
<p>The process of identifying the specific causal variants within…omic region associated with a trait.</p>
</section>
<section id="glo-effect-size" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-effect-size">Effect size</h4>
<p>A measure of the strength of the relationship between a genetic variant and a trait.</p>
</section>
<section id="glo-ascertainment-bias" class="level4 unnumbered">
<h4 class="unnumbered anchored" data-anchor-id="glo-ascertainment-bias">Ascertainment bias</h4>
<p>A systematic distortion in the estimation of genetic effects d…non-random sampling of individuals or variants.</p>
</section>
</section>
</section>
<section id="ch-03" class="level2" data-number="E.3">
<h2 data-number="E.3" class="anchored" data-anchor-id="ch-03"><span class="header-section-number">E.3</span> CH 03</h2>
</section>
<section id="ch-04" class="level2" data-number="E.4">
<h2 data-number="E.4" class="anchored" data-anchor-id="ch-04"><span class="header-section-number">E.4</span> CH 04</h2>
</section>
<section id="ch-05" class="level2" data-number="E.5">
<h2 data-number="E.5" class="anchored" data-anchor-id="ch-05"><span class="header-section-number">E.5</span> CH 05</h2>
</section>
<section id="ch-06" class="level2" data-number="E.6">
<h2 data-number="E.6" class="anchored" data-anchor-id="ch-06"><span class="header-section-number">E.6</span> CH 06</h2>
</section>
<section id="ch-07" class="level2" data-number="E.7">
<h2 data-number="E.7" class="anchored" data-anchor-id="ch-07"><span class="header-section-number">E.7</span> CH 07</h2>
</section>
<section id="ch-08" class="level2" data-number="E.8">
<h2 data-number="E.8" class="anchored" data-anchor-id="ch-08"><span class="header-section-number">E.8</span> CH 08</h2>
</section>
<section id="ch-09" class="level2" data-number="E.9">
<h2 data-number="E.9" class="anchored" data-anchor-id="ch-09"><span class="header-section-number">E.9</span> CH 09</h2>
</section>
<section id="ch-10" class="level2" data-number="E.10">
<h2 data-number="E.10" class="anchored" data-anchor-id="ch-10"><span class="header-section-number">E.10</span> CH 10</h2>
</section>
<section id="ch-11" class="level2" data-number="E.11">
<h2 data-number="E.11" class="anchored" data-anchor-id="ch-11"><span class="header-section-number">E.11</span> CH 11</h2>
</section>
<section id="ch-12" class="level2" data-number="E.12">
<h2 data-number="E.12" class="anchored" data-anchor-id="ch-12"><span class="header-section-number">E.12</span> CH 12</h2>
</section>
<section id="ch-13" class="level2" data-number="E.13">
<h2 data-number="E.13" class="anchored" data-anchor-id="ch-13"><span class="header-section-number">E.13</span> CH 13</h2>
</section>
<section id="ch-14" class="level2" data-number="E.14">
<h2 data-number="E.14" class="anchored" data-anchor-id="ch-14"><span class="header-section-number">E.14</span> CH 14</h2>
</section>
<section id="ch-15" class="level2" data-number="E.15">
<h2 data-number="E.15" class="anchored" data-anchor-id="ch-15"><span class="header-section-number">E.15</span> CH 15</h2>
</section>
<section id="ch-16" class="level2" data-number="E.16">
<h2 data-number="E.16" class="anchored" data-anchor-id="ch-16"><span class="header-section-number">E.16</span> CH 16</h2>
</section>
<section id="ch-17" class="level2" data-number="E.17">
<h2 data-number="E.17" class="anchored" data-anchor-id="ch-17"><span class="header-section-number">E.17</span> CH 17</h2>
</section>
<section id="ch-18" class="level2" data-number="E.18">
<h2 data-number="E.18" class="anchored" data-anchor-id="ch-18"><span class="header-section-number">E.18</span> CH 18</h2>
</section>
<section id="ch-19" class="level2" data-number="E.19">
<h2 data-number="E.19" class="anchored" data-anchor-id="ch-19"><span class="header-section-number">E.19</span> CH 19</h2>
</section>
<section id="ch-20" class="level2" data-number="E.20">
<h2 data-number="E.20" class="anchored" data-anchor-id="ch-20"><span class="header-section-number">E.20</span> CH 20</h2>
</section>
<section id="apx-a" class="level2" data-number="E.21">
<h2 data-number="E.21" class="anchored" data-anchor-id="apx-a"><span class="header-section-number">E.21</span> APX A</h2>
</section>
<section id="apx-b" class="level2" data-number="E.22">
<h2 data-number="E.22" class="anchored" data-anchor-id="apx-b"><span class="header-section-number">E.22</span> APX B</h2>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    // Ensure there is a toggle, if there isn't float one in the top right
    if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
      const a = window.document.createElement('a');
      a.classList.add('top-right');
      a.classList.add('quarto-color-scheme-toggle');
      a.href = "";
      a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
      const i = window.document.createElement("i");
      i.classList.add('bi');
      a.appendChild(i);
      window.document.body.appendChild(a);
    }
    setColorSchemeToggle(hasAlternateSentinel())
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
      const outerScaffold = trigger.parentElement.cloneNode(true);
      const codeEl = outerScaffold.querySelector('code');
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./app-d-resources.html" class="pagination-link" aria-label="Additional Resources">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">D</span>&nbsp; <span class="chapter-title">Additional Resources</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
  </div>
</nav>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      &nbsp;
    </div>   
    <div class="nav-footer-center">
<p>Copyright 2025, Josh Meehl</p>
</div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>




</body></html>