<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.8.26">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>References – Genomic Foundation Models</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./app-a-dl.html" rel="next">
<link href="./p6-ch27-future.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js" type="module"></script>
<script src="site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="site_libs/quarto-html/axe/axe-check.js" type="module"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-587c61ba64f3a5504c4d52d930310e48.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-dark-b758ccaa5987ceb1b75504551e579abf.css" rel="stylesheet" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-587c61ba64f3a5504c4d52d930310e48.css" rel="stylesheet" class="quarto-color-scheme-extra" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-a1553387a0f784068632030e9fbb8a3c.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="site_libs/bootstrap/bootstrap-dark-d1e9b63c6b6094879b9f94a7628e3370.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<link href="site_libs/bootstrap/bootstrap-a1553387a0f784068632030e9fbb8a3c.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme-extra" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-sidebar floating quarto-light"><script id="quarto-html-before-body" type="application/javascript">
    const toggleBodyColorMode = (bsSheetEl) => {
      const mode = bsSheetEl.getAttribute("data-mode");
      const bodyEl = window.document.querySelector("body");
      if (mode === "dark") {
        bodyEl.classList.add("quarto-dark");
        bodyEl.classList.remove("quarto-light");
      } else {
        bodyEl.classList.add("quarto-light");
        bodyEl.classList.remove("quarto-dark");
      }
    }
    const toggleBodyColorPrimary = () => {
      const bsSheetEl = window.document.querySelector("link#quarto-bootstrap:not([rel=disabled-stylesheet])");
      if (bsSheetEl) {
        toggleBodyColorMode(bsSheetEl);
      }
    }
    const setColorSchemeToggle = (alternate) => {
      const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
      for (let i=0; i < toggles.length; i++) {
        const toggle = toggles[i];
        if (toggle) {
          if (alternate) {
            toggle.classList.add("alternate");
          } else {
            toggle.classList.remove("alternate");
          }
        }
      }
    };
    const toggleColorMode = (alternate) => {
      // Switch the stylesheets
      const primaryStylesheets = window.document.querySelectorAll('link.quarto-color-scheme:not(.quarto-color-alternate)');
      const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
      manageTransitions('#quarto-margin-sidebar .nav-link', false);
      if (alternate) {
        // note: dark is layered on light, we don't disable primary!
        enableStylesheet(alternateStylesheets);
        for (const sheetNode of alternateStylesheets) {
          if (sheetNode.id === "quarto-bootstrap") {
            toggleBodyColorMode(sheetNode);
          }
        }
      } else {
        disableStylesheet(alternateStylesheets);
        enableStylesheet(primaryStylesheets)
        toggleBodyColorPrimary();
      }
      manageTransitions('#quarto-margin-sidebar .nav-link', true);
      // Switch the toggles
      setColorSchemeToggle(alternate)
      // Hack to workaround the fact that safari doesn't
      // properly recolor the scrollbar when toggling (#1455)
      if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
        manageTransitions("body", false);
        window.scrollTo(0, 1);
        setTimeout(() => {
          window.scrollTo(0, 0);
          manageTransitions("body", true);
        }, 40);
      }
    }
    const disableStylesheet = (stylesheets) => {
      for (let i=0; i < stylesheets.length; i++) {
        const stylesheet = stylesheets[i];
        stylesheet.rel = 'disabled-stylesheet';
      }
    }
    const enableStylesheet = (stylesheets) => {
      for (let i=0; i < stylesheets.length; i++) {
        const stylesheet = stylesheets[i];
        if(stylesheet.rel !== 'stylesheet') { // for Chrome, which will still FOUC without this check
          stylesheet.rel = 'stylesheet';
        }
      }
    }
    const manageTransitions = (selector, allowTransitions) => {
      const els = window.document.querySelectorAll(selector);
      for (let i=0; i < els.length; i++) {
        const el = els[i];
        if (allowTransitions) {
          el.classList.remove('notransition');
        } else {
          el.classList.add('notransition');
        }
      }
    }
    const isFileUrl = () => {
      return window.location.protocol === 'file:';
    }
    const hasAlternateSentinel = () => {
      let styleSentinel = getColorSchemeSentinel();
      if (styleSentinel !== null) {
        return styleSentinel === "alternate";
      } else {
        return false;
      }
    }
    const setStyleSentinel = (alternate) => {
      const value = alternate ? "alternate" : "default";
      if (!isFileUrl()) {
        window.localStorage.setItem("quarto-color-scheme", value);
      } else {
        localAlternateSentinel = value;
      }
    }
    const getColorSchemeSentinel = () => {
      if (!isFileUrl()) {
        const storageValue = window.localStorage.getItem("quarto-color-scheme");
        return storageValue != null ? storageValue : localAlternateSentinel;
      } else {
        return localAlternateSentinel;
      }
    }
    const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
      const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
      const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
      let newTheme = '';
      if(authorPrefersDark) {
        newTheme = isAlternate ? baseTheme : alternateTheme;
      } else {
        newTheme = isAlternate ? alternateTheme : baseTheme;
      }
      const changeGiscusTheme = () => {
        // From: https://github.com/giscus/giscus/issues/336
        const sendMessage = (message) => {
          const iframe = document.querySelector('iframe.giscus-frame');
          if (!iframe) return;
          iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
        }
        sendMessage({
          setConfig: {
            theme: newTheme
          }
        });
      }
      const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
      if (isGiscussLoaded) {
        changeGiscusTheme();
      }
    };
    const authorPrefersDark = false;
    const darkModeDefault = authorPrefersDark;
      document.querySelector('link#quarto-text-highlighting-styles.quarto-color-scheme-extra').rel = 'disabled-stylesheet';
      document.querySelector('link#quarto-bootstrap.quarto-color-scheme-extra').rel = 'disabled-stylesheet';
    let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
    // Dark / light mode switch
    window.quartoToggleColorScheme = () => {
      // Read the current dark / light value
      let toAlternate = !hasAlternateSentinel();
      toggleColorMode(toAlternate);
      setStyleSentinel(toAlternate);
      toggleGiscusIfUsed(toAlternate, darkModeDefault);
      window.dispatchEvent(new Event('resize'));
    };
    // Switch to dark mode if need be
    if (hasAlternateSentinel()) {
      toggleColorMode(true);
    } else {
      toggleColorMode(false);
    }
  </script>

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./references.html">References</a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="./">Genomic Foundation Models</a> 
        <div class="sidebar-tools-main">
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Introduction</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./preface.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Preface</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="./p1--foundations.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Part I: Foundations</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p1-ch01-ngs.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Sequencing: From Reads to Variants</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p1-ch02-data.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">The Genomic Data Landscape</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p1-ch03-pgs.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">GWAS &amp; Polygenic Scores</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p1-ch04-cadd.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Deleteriousness Scores</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="./p2--principles.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Part III: Core Principles</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p2-ch05-tokens.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Sequence Representation &amp; Tokens</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p2-ch06-transformers.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Transformer Architecture for Genomics</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p2-ch07-foundation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Genomic Foundation Models: Concepts &amp; Taxonomy</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p2-ch08-pretrain.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Pretraining Objectives &amp; Strategies</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p2-ch09-transfer.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Transfer Learning &amp; Deployment</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="./p3--architectures.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Part II: Deep Learning Architectures</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p3-ch10-cnn.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">CNN Sequence-to-Function Models</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p3-ch11-dna.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">DNA and Genomic Models</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p3-ch12-rna.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">12</span>&nbsp; <span class="chapter-title">RNA &amp; Transcript-Level Models</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p3-ch13-plm.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">13</span>&nbsp; <span class="chapter-title">Protein Language Models</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p3-ch14-hybrid.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">14</span>&nbsp; <span class="chapter-title">Long-range Hybrid Models</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" role="navigation" aria-expanded="true">
 <span class="menu-text">p4–multi-modal_multi-scale.qmd</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-4" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-4" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p4-ch15-sc-epi.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">15</span>&nbsp; <span class="chapter-title">Single-Cell &amp; Epigenomic Models</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p4-ch16-networks.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">16</span>&nbsp; <span class="chapter-title">Graphs, Networks, and Biology</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p4-ch17-systems.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">17</span>&nbsp; <span class="chapter-title">Multi-Omics &amp; Systems Biology</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="./p5--eval-interp.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Part V: Evaluation and Reliability</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-5" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-5" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p5-ch18-benchmarks.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">18</span>&nbsp; <span class="chapter-title">Benchmarks for Genomic Models</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p5-ch19-eval.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">19</span>&nbsp; <span class="chapter-title">Evaluation of Models</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p5-ch20-vep.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">20</span>&nbsp; <span class="chapter-title">Variant Effect Prediction</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p5-ch21-confound.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">21</span>&nbsp; <span class="chapter-title">Confounders in Model Training</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p5-ch22-interp.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">22</span>&nbsp; <span class="chapter-title">Interpretability &amp; Mechanisms</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="./p6--translation.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Part VI — Translation and Application</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-6" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-6" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p6-ch23-clinical.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">23</span>&nbsp; <span class="chapter-title">Clinical Risk Prediction</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p6-ch24-variants.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">24</span>&nbsp; <span class="chapter-title">Pathogenic Variant Discovery</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p6-ch25-drugs.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">25</span>&nbsp; <span class="chapter-title">Drug Discovery &amp; Biotech</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p6-ch26-design.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">26</span>&nbsp; <span class="chapter-title">Sequence Design</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./p6-ch27-future.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">27</span>&nbsp; <span class="chapter-title">Future Work &amp; Ethics</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./references.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text">References</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-7" role="navigation" aria-expanded="true">
 <span class="menu-text">Appendices</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-7" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-7" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./app-a-dl.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">A</span>&nbsp; <span class="chapter-title">Deep Learning Primer</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./app-b-deployment.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">B</span>&nbsp; <span class="chapter-title">Model Deployment</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./app-c-model-list.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">C</span>&nbsp; <span class="chapter-title">Referenced Models</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./app-d-resources.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">D</span>&nbsp; <span class="chapter-title">Additional Resources</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./app-e-glossary.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">E</span>&nbsp; <span class="chapter-title">Glossary</span></span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
        
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">


<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">References</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list">
<div id="ref-abramson_alphafold3_2024" class="csl-entry" role="listitem">
Abramson, Josh, Jonas Adler, Jack Dunger, Richard Evans, Tim Green,
Alexander Pritzel, Olaf Ronneberger, et al. 2024.
<span>“[<span>AlphaFold3</span>] <span>Accurate</span> Structure
Prediction of Biomolecular Interactions with <span>AlphaFold</span>
3.”</span> <em>Nature</em> 630 (8016): 493–500. <a href="https://doi.org/10.1038/s41586-024-07487-w">https://doi.org/10.1038/s41586-024-07487-w</a>.
</div>
<div id="ref-adzhubei_polyphen_2010" class="csl-entry" role="listitem">
Adzhubei, Ivan A., Steffen Schmidt, Leonid Peshkin, Vasily E. Ramensky,
Anna Gerasimova, Peer Bork, Alexey S. Kondrashov, and Shamil R. Sunyaev.
2010. <span>“A Method and Server for Predicting Damaging Missense
Mutations.”</span> <em>Nature Methods</em> 7 (4): 248–49. <a href="https://doi.org/10.1038/nmeth0410-248">https://doi.org/10.1038/nmeth0410-248</a>.
</div>
<div id="ref-ahdritz_openfold_2024" class="csl-entry" role="listitem">
Ahdritz, Gustaf, Nazim Bouatta, Christina Floristean, Sachin Kadyan,
Qinghui Xia, William Gerecke, Timothy J. O’Donnell, et al. 2024.
<span>“<span>OpenFold</span>: Retraining <span>AlphaFold2</span> Yields
New Insights into Its Learning Mechanisms and Capacity for
Generalization.”</span> <em>Nature Methods</em> 21 (8): 1514–24. <a href="https://doi.org/10.1038/s41592-024-02272-z">https://doi.org/10.1038/s41592-024-02272-z</a>.
</div>
<div id="ref-null_all-of-us_2019" class="csl-entry" role="listitem">
All of Us Research Program Investigators, All of Us; 2019. <span>“The
<span>‘<span>All</span> of <span>Us</span>’</span> <span>Research</span>
<span>Program</span>.”</span> <em>New England Journal of Medicine</em>
381 (7): 668–76. <a href="https://doi.org/10.1056/NEJMsr1809937">https://doi.org/10.1056/NEJMsr1809937</a>.
</div>
<div id="ref-amberger_omim_2015" class="csl-entry" role="listitem">
Amberger, Joanna S., Carol A. Bocchini, François Schiettecatte, Alan F.
Scott, and Ada Hamosh. 2015. <span>“<span>OMIM</span>.org:
<span>Online</span> <span>Mendelian</span> <span>Inheritance</span> in
<span>Man</span> (<span>OMIM</span>®), an Online Catalog of Human Genes
and Genetic Disorders.”</span> <em>Nucleic Acids Research</em> 43 (D1):
D789–98. <a href="https://doi.org/10.1093/nar/gku1205">https://doi.org/10.1093/nar/gku1205</a>.
</div>
<div id="ref-auton_1kgp_2015" class="csl-entry" role="listitem">
Auton, Adam, Gonçalo R. Abecasis, David M. Altshuler, Richard M. Durbin,
Gonçalo R. Abecasis, David R. Bentley, Aravinda Chakravarti, et al.
2015. <span>“A Global Reference for Human Genetic Variation.”</span>
<em>Nature</em> 526 (7571): 68–74. <a href="https://doi.org/10.1038/nature15393">https://doi.org/10.1038/nature15393</a>.
</div>
<div id="ref-avsec_enformer_2021" class="csl-entry" role="listitem">
Avsec, Žiga, Vikram Agarwal, D. Visentin, J. Ledsam, A.
Grabska-Barwinska, Kyle R. Taylor, Yannis Assael, J. Jumper, Pushmeet
Kohli, and David R. Kelley. 2021. <span>“[<span>Enformer</span>]
<span>Effective</span> Gene Expression Prediction from Sequence by
Integrating Long-Range Interactions.”</span> <em>Nature Methods</em> 18
(October): 1196–1203. <a href="https://doi.org/10.1038/s41592-021-01252-x">https://doi.org/10.1038/s41592-021-01252-x</a>.
</div>
<div id="ref-avsec_alphagenome_2025" class="csl-entry" role="listitem">
Avsec, Ziga, Natasha Latysheva, and Jun Cheng. 2025.
<span>“<span>AlphaGenome</span>: <span>AI</span> for Better
Understanding the Genome.”</span> <em>Google DeepMind</em>. <a href="https://deepmind.google/discover/blog/alphagenome-ai-for-better-understanding-the-genome/">https://deepmind.google/discover/blog/alphagenome-ai-for-better-understanding-the-genome/</a>.
</div>
<div id="ref-baek_rosettafold_2021" class="csl-entry" role="listitem">
Baek, Minkyung, Frank DiMaio, Ivan Anishchenko, Justas Dauparas, Sergey
Ovchinnikov, Gyu Rie Lee, Jue Wang, et al. 2021. <span>“Accurate
Prediction of Protein Structures and Interactions Using a Three-Track
Neural Network.”</span> <em>Science</em> 373 (6557): 871–76. <a href="https://doi.org/10.1126/science.abj8754">https://doi.org/10.1126/science.abj8754</a>.
</div>
<div id="ref-benegas_gpn-msa_2024" class="csl-entry" role="listitem">
Benegas, Gonzalo, Carlos Albors, Alan J. Aw, Chengzhong Ye, and Yun S.
Song. 2024. <span>“<span>GPN</span>-<span>MSA</span>: An Alignment-Based
<span>DNA</span> Language Model for Genome-Wide Variant Effect
Prediction.”</span> <em>bioRxiv</em>, April, 2023.10.10.561776. <a href="https://doi.org/10.1101/2023.10.10.561776">https://doi.org/10.1101/2023.10.10.561776</a>.
</div>
<div id="ref-benegas_gpn_2023" class="csl-entry" role="listitem">
Benegas, Gonzalo, Sanjit Singh Batra, and Yun S. Song. 2023.
<span>“[<span>GPN</span>] <span>DNA</span> Language Models Are Powerful
Predictors of Genome-Wide Variant Effects.”</span> <em>Proceedings of
the National Academy of Sciences</em> 120 (44): e2311219120. <a href="https://doi.org/10.1073/pnas.2311219120">https://doi.org/10.1073/pnas.2311219120</a>.
</div>
<div id="ref-benegas_traitgym_2025" class="csl-entry" role="listitem">
Benegas, Gonzalo, Gökcen Eraslan, and Yun S. Song. 2025.
<span>“[<span>TraitGym</span>] <span>Benchmarking</span>
<span>DNA</span> <span>Sequence</span> <span>Models</span> for
<span>Causal</span> <span>Regulatory</span> <span>Variant</span>
<span>Prediction</span> in <span>Human</span>
<span>Genetics</span>.”</span> bioRxiv. <a href="https://doi.org/10.1101/2025.02.11.637758">https://doi.org/10.1101/2025.02.11.637758</a>.
</div>
<div id="ref-benegas_genomic_2024" class="csl-entry" role="listitem">
Benegas, Gonzalo, Chengzhong Ye, Carlos Albors, Jianan Canal Li, and Yun
S. Song. 2024. <span>“Genomic <span>Language</span> <span>Models</span>:
<span>Opportunities</span> and <span>Challenges</span>.”</span> arXiv.
<a href="https://doi.org/10.48550/arXiv.2407.11435">https://doi.org/10.48550/arXiv.2407.11435</a>.
</div>
<div id="ref-deboer_deciphering_2020" class="csl-entry" role="listitem">
Boer, Carl G. de, Eeshit Dhaval Vaishnav, Ronen Sadeh, Esteban Luis
Abeyta, Nir Friedman, and Aviv Regev. 2020. <span>“Deciphering
Eukaryotic Gene-Regulatory Logic with 100 Million Random
Promoters.”</span> <em>Nature Biotechnology</em> 38 (1): 56–65. <a href="https://doi.org/10.1038/s41587-019-0315-8">https://doi.org/10.1038/s41587-019-0315-8</a>.
</div>
<div id="ref-bommasani_foundation_2022" class="csl-entry" role="listitem">
Bommasani, Rishi, Drew A. Hudson, Ehsan Adeli, Russ Altman, Simran
Arora, Sydney von Arx, Michael S. Bernstein, et al. 2022. <span>“On the
<span>Opportunities</span> and <span>Risks</span> of
<span>Foundation</span> <span>Models</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2108.07258">https://doi.org/10.48550/arXiv.2108.07258</a>.
</div>
<div id="ref-brandes_genome-wide_2023" class="csl-entry" role="listitem">
Brandes, Nadav, Grant Goldman, Charlotte H. Wang, Chun Jimmie Ye, and
Vasilis Ntranos. 2023. <span>“Genome-Wide Prediction of Disease Variant
Effects with a Deep Protein Language Model.”</span> <em>Nature
Genetics</em> 55 (9): 1512–22. <a href="https://doi.org/10.1038/s41588-023-01465-0">https://doi.org/10.1038/s41588-023-01465-0</a>.
</div>
<div id="ref-brixi_evo_2025" class="csl-entry" role="listitem">
Brixi, Garyk, Matthew G. Durrant, Jerome Ku, Michael Poli, Greg
Brockman, Daniel Chang, Gabriel A. Gonzalez, et al. 2025.
<span>“[<span>Evo</span> 2] <span>Genome</span> Modeling and Design
Across All Domains of Life with <span>Evo</span> 2.”</span> bioRxiv. <a href="https://doi.org/10.1101/2025.02.18.638918">https://doi.org/10.1101/2025.02.18.638918</a>.
</div>
<div id="ref-browning_beagle_2021" class="csl-entry" role="listitem">
Browning, Brian L., Xiaowen Tian, Ying Zhou, and Sharon R. Browning.
2021. <span>“Fast Two-Stage Phasing of Large-Scale Sequence
Data.”</span> <em>American Journal of Human Genetics</em> 108 (10):
1880–90. <a href="https://doi.org/10.1016/j.ajhg.2021.08.005">https://doi.org/10.1016/j.ajhg.2021.08.005</a>.
</div>
<div id="ref-bycroft_ukbiobank_2018" class="csl-entry" role="listitem">
Bycroft, Clare, Colin Freeman, Desislava Petkova, Gavin Band, Lloyd T.
Elliott, Kevin Sharp, Allan Motyer, et al. 2018. <span>“The
<span>UK</span> <span>Biobank</span> Resource with Deep Phenotyping and
Genomic Data.”</span> <em>Nature</em> 562 (7726): 203–9. <a href="https://doi.org/10.1038/s41586-018-0579-z">https://doi.org/10.1038/s41586-018-0579-z</a>.
</div>
<div id="ref-camillo_cpgpt_2024" class="csl-entry" role="listitem">
Camillo, Lucas Paulo de Lima, Raghav Sehgal, Jenel Armstrong, Albert T.
Higgins-Chen, Steve Horvath, and Bo Wang. 2024.
<span>“<span>CpGPT</span>: A <span>Foundation</span> <span>Model</span>
for <span>DNA</span> <span>Methylation</span>.”</span> bioRxiv. <a href="https://doi.org/10.1101/2024.10.24.619766">https://doi.org/10.1101/2024.10.24.619766</a>.
</div>
<div id="ref-cao_glue_2022" class="csl-entry" role="listitem">
Cao, Zhi-Jie, and Ge Gao. 2022. <span>“[<span>GLUE</span>]
<span>Multi</span>-Omics Single-Cell Data Integration and Regulatory
Inference with Graph-Linked Embedding.”</span> <em>Nature
Biotechnology</em> 40 (10): 1458–66. <a href="https://doi.org/10.1038/s41587-022-01284-4">https://doi.org/10.1038/s41587-022-01284-4</a>.
</div>
<div id="ref-chafai_review_2023" class="csl-entry" role="listitem">
Chafai, Narjice, Ichrak Hayah, Isidore Houaga, and Bouabid Badaoui.
2023. <span>“A Review of Machine Learning Models Applied to Genomic
Prediction in Animal Breeding.”</span> <em>Frontiers in Genetics</em> 14
(September). <a href="https://doi.org/10.3389/fgene.2023.1150596">https://doi.org/10.3389/fgene.2023.1150596</a>.
</div>
<div id="ref-chandak_primekg_2023" class="csl-entry" role="listitem">
Chandak, Payal, Kexin Huang, and Marinka Zitnik. 2023.
<span>“[<span>PrimeKG</span>] <span>Building</span> a Knowledge Graph to
Enable Precision Medicine.”</span> <em>Scientific Data</em> 10 (1): 67.
<a href="https://doi.org/10.1038/s41597-023-01960-3">https://doi.org/10.1038/s41597-023-01960-3</a>.
</div>
<div id="ref-chen_rna-fm_2022" class="csl-entry" role="listitem">
Chen, Jiayang, Zhihang Hu, Siqi Sun, Qingxiong Tan, Yixuan Wang, Qinze
Yu, Licheng Zong, et al. 2022. <span>“[<span>RNA</span>-<span>FM</span>]
<span>Interpretable</span> <span>RNA</span> <span>Foundation</span>
<span>Model</span> from <span>Unannotated</span> <span>Data</span> for
<span>Highly</span> <span>Accurate</span> <span>RNA</span>
<span>Structure</span> and <span>Function</span>
<span>Predictions</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2204.00300">https://doi.org/10.48550/arXiv.2204.00300</a>.
</div>
<div id="ref-chen_deepsea_2022" class="csl-entry" role="listitem">
Chen, Kathleen M., Aaron K. Wong, Olga G. Troyanskaya, and Jian Zhou.
2022. <span>“[<span>DeepSEA</span> <span>Sei</span>] <span>A</span>
Sequence-Based Global Map of Regulatory Activity for Deciphering Human
Genetics.”</span> <em>Nature Genetics</em> 54 (7): 940–49. <a href="https://doi.org/10.1038/s41588-022-01102-2">https://doi.org/10.1038/s41588-022-01102-2</a>.
</div>
<div id="ref-chen_causal_2025" class="csl-entry" role="listitem">
Chen, Liyin. 2025. <span>“Causal <span>Genomics</span> in the
<span>Deep</span> <span>Learning</span> <span>Era</span>.”</span> MDPI
AG. <a href="https://doi.org/10.20944/preprints202503.2081.v1">https://doi.org/10.20944/preprints202503.2081.v1</a>.
</div>
<div id="ref-cheng_alphamissense_2023" class="csl-entry" role="listitem">
Cheng, Jun, Guido Novati, Joshua Pan, Clare Bycroft, Akvilė Žemgulytė,
Taylor Applebaum, Alexander Pritzel, et al. 2023.
<span>“[<span>AlphaMissense</span>] <span>Accurate</span> Proteome-Wide
Missense Variant Effect Prediction with
<span>AlphaMissense</span>.”</span> <em>Science</em> 381 (6664):
eadg7492. <a href="https://doi.org/10.1126/science.adg7492">https://doi.org/10.1126/science.adg7492</a>.
</div>
<div id="ref-cheng_dnalongbench_2024" class="csl-entry" role="listitem">
Cheng, Wenduo, Zhenqiao Song, Yang Zhang, Shike Wang, Danqing Wang, Muyu
Yang, Lei Li, and Jian Ma. 2024. <span>“<span>DNALONGBENCH</span>:
<span>A</span> <span>Benchmark</span> <span>Suite</span>
<span>For</span> <span>Long</span>-<span>Range</span> <span>DNA</span>
<span>Prediction</span> <span>Tasks</span>,”</span> October. <a href="https://openreview.net/forum?id=opv67PpqLS">https://openreview.net/forum?id=opv67PpqLS</a>.
</div>
<div id="ref-choi_prs_2020" class="csl-entry" role="listitem">
Choi, Shing Wan, Timothy Shin-Heng Mak, and Paul F. O’Reilly. 2020.
<span>“[<span>PRS</span>] <span>Tutorial</span>: A Guide to Performing
Polygenic Risk Score Analyses.”</span> <em>Nature Protocols</em> 15 (9):
2759–72. <a href="https://doi.org/10.1038/s41596-020-0353-1">https://doi.org/10.1038/s41596-020-0353-1</a>.
</div>
<div id="ref-chung_carbamazepine_2004" class="csl-entry" role="listitem">
Chung, Wen-Hung, Shuen-Iu Hung, Hong-Shang Hong, Mo-Song Hsih, Li-Cheng
Yang, Hsin-Chun Ho, Jer-Yuarn Wu, and Yuan-Tsong Chen. 2004. <span>“A
Marker for <span>Stevens</span>–<span>Johnson</span> Syndrome.”</span>
<em>Nature</em> 428 (6982): 486–86. <a href="https://doi.org/10.1038/428486a">https://doi.org/10.1038/428486a</a>.
</div>
<div id="ref-clarke_deeprvat_2024" class="csl-entry" role="listitem">
Clarke, Brian, Eva Holtkamp, Hakime Öztürk, Marcel Mück, Magnus
Wahlberg, Kayla Meyer, Felix Munzlinger, et al. 2024.
<span>“[<span>DeepRVAT</span>] <span>Integration</span> of Variant
Annotations Using Deep Set Networks Boosts Rare Variant Association
Testing.”</span> <em>Nature Genetics</em> 56 (10): 2271–80. <a href="https://doi.org/10.1038/s41588-024-01919-z">https://doi.org/10.1038/s41588-024-01919-z</a>.
</div>
<div id="ref-consens_transformers_2023" class="csl-entry" role="listitem">
Consens, Micaela E., Cameron Dufault, Michael Wainberg, Duncan Forster,
Mehran Karimzadeh, Hani Goodarzi, Fabian J. Theis, Alan Moses, and Bo
Wang. 2023. <span>“To <span>Transformers</span> and <span>Beyond</span>:
<span>Large</span> <span>Language</span> <span>Models</span> for the
<span>Genome</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2311.07621">https://doi.org/10.48550/arXiv.2311.07621</a>.
</div>
<div id="ref-corso_diffdock_2022" class="csl-entry" role="listitem">
Corso, Gabriele, Hannes Stärk, Bowen Jing, Regina Barzilay, and Tommi
Jaakkola. 2022. <span>“<span>DiffDock</span>: <span>Diffusion</span>
<span>Steps</span>, <span>Twists</span>, and <span>Turns</span> for
<span>Molecular</span> <span>Docking</span>.”</span> <em>arXiv.org</em>.
<a href="https://arxiv.org/abs/2210.01776v2">https://arxiv.org/abs/2210.01776v2</a>.
</div>
<div id="ref-cui_scgpt_2024" class="csl-entry" role="listitem">
Cui, Haotian, Chloe Wang, Hassaan Maan, Kuan Pang, Fengning Luo, Nan
Duan, and Bo Wang. 2024. <span>“<span class="nocase">scGPT</span>:
Toward Building a Foundation Model for Single-Cell Multi-Omics Using
Generative <span>AI</span>.”</span> <em>Nature Methods</em> 21 (8):
1470–80. <a href="https://doi.org/10.1038/s41592-024-02201-0">https://doi.org/10.1038/s41592-024-02201-0</a>.
</div>
<div id="ref-dabernig_ont_2024" class="csl-entry" role="listitem">
Dabernig-Heinz, Johanna, Mara Lohde, Martin Hölzer, Adriana Cabal, Rick
Conzemius, Christian Brandt, Matthias Kohl, et al. 2024. <span>“A
Multicenter Study on Accuracy and Reproducibility of Nanopore
Sequencing-Based Genotyping of Bacterial Pathogens.”</span> <em>Journal
of Clinical Microbiology</em> 62 (9): e00628–24. <a href="https://doi.org/10.1128/jcm.00628-24">https://doi.org/10.1128/jcm.00628-24</a>.
</div>
<div id="ref-dalla-torre_nucleotide_2023" class="csl-entry" role="listitem">
Dalla-Torre, Hugo, Liam Gonzalez, Javier Mendoza-Revilla, Nicolas Lopez
Carranza, Adam Henryk Grzywaczewski, Francesco Oteri, Christian Dallago,
et al. 2023. <span>“Nucleotide <span>Transformer</span>: Building and
Evaluating Robust Foundation Models for Human Genomics.”</span>
<em>Nature Methods</em> 22 (2): 287–97. <a href="https://doi.org/10.1038/s41592-024-02523-z">https://doi.org/10.1038/s41592-024-02523-z</a>.
</div>
<div id="ref-dauparas_proteinmpnn_2022" class="csl-entry" role="listitem">
Dauparas, J., I. Anishchenko, N. Bennett, H. Bai, R. J. Ragotte, L. F.
Milles, B. I. M. Wicky, et al. 2022. <span>“Robust Deep Learning–Based
Protein Sequence Design Using <span>ProteinMPNN</span>.”</span>
<em>Science</em> 378 (6615): 49–56. <a href="https://doi.org/10.1126/science.add2187">https://doi.org/10.1126/science.add2187</a>.
</div>
<div id="ref-davydov_gerp_2010" class="csl-entry" role="listitem">
Davydov, Eugene V., David L. Goode, Marina Sirota, Gregory M. Cooper,
Arend Sidow, and Serafim Batzoglou. 2010. <span>“Identifying a
<span>High</span> <span>Fraction</span> of the <span>Human</span>
<span>Genome</span> to Be Under <span>Selective</span>
<span>Constraint</span> <span>Using</span> <span>GERP</span>++.”</span>
<em>PLOS Computational Biology</em> 6 (12): e1001025. <a href="https://doi.org/10.1371/journal.pcbi.1001025">https://doi.org/10.1371/journal.pcbi.1001025</a>.
</div>
<div id="ref-depristo_gatk_2011" class="csl-entry" role="listitem">
DePristo, Mark A., Eric Banks, Ryan Poplin, Kiran V. Garimella, Jared R.
Maguire, Christopher Hartl, Anthony A. Philippakis, et al. 2011.
<span>“A Framework for Variation Discovery and Genotyping Using
Next-Generation <span>DNA</span> Sequencing Data.”</span> <em>Nature
Genetics</em> 43 (5): 491–98. <a href="https://doi.org/10.1038/ng.806">https://doi.org/10.1038/ng.806</a>.
</div>
<div id="ref-devlin_bert_2019" class="csl-entry" role="listitem">
Devlin, Jacob, Ming-Wei Chang, Kenton Lee, and Kristina Toutanova. 2019.
<span>“<span>BERT</span>: <span>Pre</span>-Training of <span>Deep</span>
<span>Bidirectional</span> <span>Transformers</span> for
<span>Language</span> <span>Understanding</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.1810.04805">https://doi.org/10.48550/arXiv.1810.04805</a>.
</div>
<div id="ref-du_gene2vec_2019" class="csl-entry" role="listitem">
Du, Jingcheng, Peilin Jia, Yulin Dai, Cui Tao, Zhongming Zhao, and Degui
Zhi. 2019. <span>“Gene2vec: Distributed Representation of Genes Based on
Co-Expression.”</span> <em>BMC Genomics</em> 20 (1): 82. <a href="https://doi.org/10.1186/s12864-018-5370-x">https://doi.org/10.1186/s12864-018-5370-x</a>.
</div>
<div id="ref-edgar_geo_2002" class="csl-entry" role="listitem">
Edgar, Ron, Michael Domrachev, and Alex E. Lash. 2002. <span>“Gene
<span>Expression</span> <span>Omnibus</span>: <span>NCBI</span> Gene
Expression and Hybridization Array Data Repository.”</span> <em>Nucleic
Acids Research</em> 30 (1): 207–10. <a href="https://doi.org/10.1093/nar/30.1.207">https://doi.org/10.1093/nar/30.1.207</a>.
</div>
<div id="ref-elnaggar_prottrans_2021" class="csl-entry" role="listitem">
Elnaggar, Ahmed, Michael Heinzinger, Christian Dallago, Ghalia Rihawi,
Yu Wang, Llion Jones, Tom Gibbs, et al. 2021.
<span>“<span>ProtTrans</span>: <span>Towards</span>
<span>Cracking</span> the <span>Language</span> of <span>Life</span>’s
<span>Code</span> <span>Through</span>
<span>Self</span>-<span>Supervised</span> <span>Deep</span>
<span>Learning</span> and <span>High</span> <span>Performance</span>
<span>Computing</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2007.06225">https://doi.org/10.48550/arXiv.2007.06225</a>.
</div>
<div id="ref-ferruz_protgpt2_2022" class="csl-entry" role="listitem">
Ferruz, Noelia, Steffen Schmidt, and Birte Höcker. 2022.
<span>“<span>ProtGPT2</span> Is a Deep Unsupervised Language Model for
Protein Design.”</span> <em>Nature Communications</em> 13 (1): 4348. <a href="https://doi.org/10.1038/s41467-022-32007-7">https://doi.org/10.1038/s41467-022-32007-7</a>.
</div>
<div id="ref-finn_hmmer_2011" class="csl-entry" role="listitem">
Finn, Robert D., Jody Clements, and Sean R. Eddy. 2011.
<span>“<span>HMMER</span> Web Server: Interactive Sequence Similarity
Searching.”</span> <em>Nucleic Acids Research</em> 39 (suppl_2): W29–37.
<a href="https://doi.org/10.1093/nar/gkr367">https://doi.org/10.1093/nar/gkr367</a>.
</div>
<div id="ref-fishman_gena-lm_2025" class="csl-entry" role="listitem">
Fishman, Veniamin, Yuri Kuratov, Aleksei Shmelev, Maxim Petrov, Dmitry
Penzar, Denis Shepelin, Nikolay Chekanov, Olga Kardymon, and Mikhail
Burtsev. 2025. <span>“<span>GENA</span>-<span>LM</span>: A Family of
Open-Source Foundational <span>DNA</span> Language Models for Long
Sequences.”</span> <em>Nucleic Acids Research</em> 53 (2): gkae1310. <a href="https://doi.org/10.1093/nar/gkae1310">https://doi.org/10.1093/nar/gkae1310</a>.
</div>
<div id="ref-frankish_gencode_2019" class="csl-entry" role="listitem">
Frankish, Adam, Mark Diekhans, Anne-Maud Ferreira, Rory Johnson, Irwin
Jungreis, Jane Loveland, Jonathan M Mudge, et al. 2019.
<span>“<span>GENCODE</span> Reference Annotation for the Human and Mouse
Genomes.”</span> <em>Nucleic Acids Research</em> 47 (D1): D766–73. <a href="https://doi.org/10.1093/nar/gky955">https://doi.org/10.1093/nar/gky955</a>.
</div>
<div id="ref-frazer_eve_2021" class="csl-entry" role="listitem">
Frazer, Jonathan, Pascal Notin, Mafalda Dias, Aidan Gomez, Joseph K.
Min, Kelly Brock, Yarin Gal, and Debora S. Marks. 2021.
<span>“[<span>EVE</span>] <span>Disease</span> Variant Prediction with
Deep Generative Models of Evolutionary Data.”</span> <em>Nature</em> 599
(7883): 91–95. <a href="https://doi.org/10.1038/s41586-021-04043-8">https://doi.org/10.1038/s41586-021-04043-8</a>.
</div>
<div id="ref-fudenberg_akita_2020" class="csl-entry" role="listitem">
Fudenberg, Geoff, David R. Kelley, and Katherine S. Pollard. 2020.
<span>“[<span>Akita</span>] <span>Predicting</span> <span>3D</span>
Genome Folding from <span>DNA</span> Sequence with
<span>Akita</span>.”</span> <em>Nature Methods</em> 17 (11): 1111–17. <a href="https://doi.org/10.1038/s41592-020-0958-x">https://doi.org/10.1038/s41592-020-0958-x</a>.
</div>
<div id="ref-gamazon_predixcan_2015" class="csl-entry" role="listitem">
Gamazon, Eric R., Heather E. Wheeler, Kaanan P. Shah, Sahar V.
Mozaffari, Keston Aquino-Michaels, Robert J. Carroll, Anne E. Eyler, et
al. 2015. <span>“A Gene-Based Association Method for Mapping Traits
Using Reference Transcriptome Data.”</span> <em>Nature Genetics</em> 47
(9): 1091–98. <a href="https://doi.org/10.1038/ng.3367">https://doi.org/10.1038/ng.3367</a>.
</div>
<div id="ref-gao_high-ppi_2023" class="csl-entry" role="listitem">
Gao, Ziqi, Chenran Jiang, Jiawen Zhang, Xiaosen Jiang, Lanqing Li,
Peilin Zhao, Huanming Yang, Yong Huang, and Jia Li. 2023.
<span>“[<span>HIGH</span>-<span>PPI</span>] <span>Hierarchical</span>
Graph Learning for Protein–Protein Interaction.”</span> <em>Nature
Communications</em> 14 (1): 1093. <a href="https://doi.org/10.1038/s41467-023-36736-1">https://doi.org/10.1038/s41467-023-36736-1</a>.
</div>
<div id="ref-garrison_vgtool_2018" class="csl-entry" role="listitem">
Garrison, Erik, Jouni Sirén, Adam M. Novak, Glenn Hickey, Jordan M.
Eizenga, Eric T. Dawson, William Jones, et al. 2018. <span>“Variation
Graph Toolkit Improves Read Mapping by Representing Genetic Variation in
the Reference.”</span> <em>Nature Biotechnology</em> 36 (9): 875–79. <a href="https://doi.org/10.1038/nbt.4227">https://doi.org/10.1038/nbt.4227</a>.
</div>
<div id="ref-georgantas_delphi_2024" class="csl-entry" role="listitem">
Georgantas, Costa, Zoltán Kutalik, and Jonas Richiardi. 2024.
<span>“Delphi: <span>A</span> <span>Deep</span>-Learning
<span>Method</span> for <span>Polygenic</span> <span>Risk</span>
<span>Prediction</span>.”</span> medRxiv. <a href="https://doi.org/10.1101/2024.04.19.24306079">https://doi.org/10.1101/2024.04.19.24306079</a>.
</div>
<div id="ref-goodwin_10year_2016" class="csl-entry" role="listitem">
Goodwin, Sara, John D. McPherson, and W. Richard McCombie. 2016.
<span>“Coming of Age: Ten Years of Next-Generation Sequencing
Technologies.”</span> <em>Nature Reviews Genetics</em> 17 (6): 333–51.
<a href="https://doi.org/10.1038/nrg.2016.49">https://doi.org/10.1038/nrg.2016.49</a>.
</div>
<div id="ref-gresova_genomic_2023" class="csl-entry" role="listitem">
Grešová, Katarína, Vlastimil Martinek, David Čechák, Petr Šimeček, and
Panagiotis Alexiou. 2023. <span>“Genomic Benchmarks: A Collection of
Datasets for Genomic Sequence Classification.”</span> <em>BMC Genomic
Data</em> 24 (1): 25. <a href="https://doi.org/10.1186/s12863-023-01123-8">https://doi.org/10.1186/s12863-023-01123-8</a>.
</div>
<div id="ref-guo_foundation_2025" class="csl-entry" role="listitem">
Guo, Fei, Renchu Guan, Yaohang Li, Qi Liu, Xiaowo Wang, Can Yang, and
Jianxin Wang. 2025. <span>“Foundation Models in Bioinformatics.”</span>
<em>National Science Review</em> 12 (4): nwaf028. <a href="https://doi.org/10.1093/nsr/nwaf028">https://doi.org/10.1093/nsr/nwaf028</a>.
</div>
<div id="ref-gusev_twas_2016" class="csl-entry" role="listitem">
Gusev, Alexander, Arthur Ko, Huwenbo Shi, Gaurav Bhatia, Wonil Chung,
Brenda W. J. H. Penninx, Rick Jansen, et al. 2016. <span>“Integrative
Approaches for Large-Scale Transcriptome-Wide Association
Studies.”</span> <em>Nature Genetics</em> 48 (3): 245–52. <a href="https://doi.org/10.1038/ng.3506">https://doi.org/10.1038/ng.3506</a>.
</div>
<div id="ref-hamilton_graphsage_2017" class="csl-entry" role="listitem">
Hamilton, William L., Rex Ying, and Jure Leskovec. 2017.
<span>“[<span>GraphSAGE</span>] <span>Inductive</span>
<span>Representation</span> <span>Learning</span> on <span>Large</span>
<span>Graphs</span>.”</span> <em>arXiv.org</em>. <a href="https://arxiv.org/abs/1706.02216v4">https://arxiv.org/abs/1706.02216v4</a>.
</div>
<div id="ref-he_nucleic_2023" class="csl-entry" role="listitem">
He, Shujun, Baizhen Gao, Rushant Sabnis, and Qing Sun. 2023.
<span>“Nucleic <span>Transformer</span>: <span>Classifying</span>
<span>DNA</span> <span>Sequences</span> with
<span>Self</span>-<span>Attention</span> and
<span>Convolutions</span>.”</span> <em>ACS Synthetic Biology</em> 12
(11): 3205–14. <a href="https://doi.org/10.1021/acssynbio.3c00154">https://doi.org/10.1021/acssynbio.3c00154</a>.
</div>
<div id="ref-hoffmann_needl_2024" class="csl-entry" role="listitem">
Hoffmann, Markus, Julian M Poschenrieder, Massimiliano Incudini, Sylvie
Baier, Amelie Fritz, Andreas Maier, Michael Hartung, et al. 2024.
<span>“[<span>NeEDL</span>] <span>Network</span> Medicine-Based
Epistasis Detection in Complex Diseases: Ready for Quantum
Computing.”</span> <em>Nucleic Acids Research</em> 52 (17): 10144–60. <a href="https://doi.org/10.1093/nar/gkae697">https://doi.org/10.1093/nar/gkae697</a>.
</div>
<div id="ref-hudaiberdiev_trednet_2023" class="csl-entry" role="listitem">
Hudaiberdiev, Sanjarbek, D. Leland Taylor, Wei Song, Narisu Narisu,
Redwan M. Bhuiyan, Henry J. Taylor, Xuming Tang, et al. 2023.
<span>“[<span>TREDNet</span>] <span>Modeling</span> Islet Enhancers
Using Deep Learning Identifies Candidate Causal Variants at Loci
Associated with <span>T2D</span> and Glycemic Traits.”</span>
<em>Proceedings of the National Academy of Sciences</em> 120 (35):
e2206612120. <a href="https://doi.org/10.1073/pnas.2206612120">https://doi.org/10.1073/pnas.2206612120</a>.
</div>
<div id="ref-ioannidis_revel_2016" class="csl-entry" role="listitem">
Ioannidis, Nilah M., Joseph H. Rothstein, Vikas Pejaver, Sumit Middha,
Shannon K. McDonnell, Saurabh Baheti, Anthony Musolf, et al. 2016.
<span>“<span>REVEL</span>: <span>An</span> <span>Ensemble</span>
<span>Method</span> for <span>Predicting</span> the
<span>Pathogenicity</span> of <span>Rare</span> <span>Missense</span>
<span>Variants</span>.”</span> <em>The American Journal of Human
Genetics</em> 99 (4): 877–85. <a href="https://doi.org/10.1016/j.ajhg.2016.08.016">https://doi.org/10.1016/j.ajhg.2016.08.016</a>.
</div>
<div id="ref-ishigaki_multi-ancestry_2022" class="csl-entry" role="listitem">
Ishigaki, Kazuyoshi, Saori Sakaue, Chikashi Terao, Yang Luo, Kyuto
Sonehara, Kensuke Yamaguchi, Tiffany Amariuta, et al. 2022.
<span>“Multi-Ancestry Genome-Wide Association Analyses Identify Novel
Genetic Mechanisms in Rheumatoid Arthritis.”</span> <em>Nature
Genetics</em> 54 (11): 1640–51. <a href="https://doi.org/10.1038/s41588-022-01213-w">https://doi.org/10.1038/s41588-022-01213-w</a>.
</div>
<div id="ref-jaganathan_spliceai_2019" class="csl-entry" role="listitem">
Jaganathan, Kishore, Sofia Kyriazopoulou Panagiotopoulou, Jeremy F.
McRae, Siavash Fazel Darbandi, David Knowles, Yang I. Li, Jack A.
Kosmicki, et al. 2019. <span>“[<span>SpliceAI</span>]
<span>Predicting</span> <span>Splicing</span> from <span>Primary</span>
<span>Sequence</span> with <span>Deep</span>
<span>Learning</span>.”</span> <em>Cell</em> 176 (3): 535–548.e24. <a href="https://doi.org/10.1016/j.cell.2018.12.015">https://doi.org/10.1016/j.cell.2018.12.015</a>.
</div>
<div id="ref-jagota_cross-protein_2023" class="csl-entry" role="listitem">
Jagota, Milind, Chengzhong Ye, Carlos Albors, Ruchir Rastogi, Antoine
Koehl, Nilah Ioannidis, and Yun S. Song. 2023. <span>“Cross-Protein
Transfer Learning Substantially Improves Disease Variant
Prediction.”</span> <em>Genome Biology</em> 24 (1): 182. <a href="https://doi.org/10.1186/s13059-023-03024-6">https://doi.org/10.1186/s13059-023-03024-6</a>.
</div>
<div id="ref-ji_dnabert_2021" class="csl-entry" role="listitem">
Ji, Yanrong, Zhihan Zhou, Han Liu, and Ramana V Davuluri. 2021.
<span>“<span>DNABERT</span>: Pre-Trained <span>Bidirectional</span>
<span>Encoder</span> <span>Representations</span> from
<span>Transformers</span> Model for <span>DNA</span>-Language in
Genome.”</span> <em>Bioinformatics</em> 37 (15): 2112–20. <a href="https://doi.org/10.1093/bioinformatics/btab083">https://doi.org/10.1093/bioinformatics/btab083</a>.
</div>
<div id="ref-jiang_cutesv_2020" class="csl-entry" role="listitem">
Jiang, Tao, Yongzhuang Liu, Yue Jiang, Junyi Li, Yan Gao, Zhe Cui,
Yadong Liu, Bo Liu, and Yadong Wang. 2020. <span>“Long-Read-Based Human
Genomic Structural Variation Detection with <span class="nocase">cuteSV</span>.”</span> <em>Genome Biology</em> 21 (1):
189. <a href="https://doi.org/10.1186/s13059-020-02107-y">https://doi.org/10.1186/s13059-020-02107-y</a>.
</div>
<div id="ref-jumper_alphafold2_2021" class="csl-entry" role="listitem">
Jumper, John, Richard Evans, Alexander Pritzel, Tim Green, Michael
Figurnov, Olaf Ronneberger, Kathryn Tunyasuvunakool, et al. 2021.
<span>“[<span>AlphaFold2</span>] <span>Highly</span> Accurate Protein
Structure Prediction with <span>AlphaFold</span>.”</span>
<em>Nature</em> 596 (7873): 583–89. <a href="https://doi.org/10.1038/s41586-021-03819-2">https://doi.org/10.1038/s41586-021-03819-2</a>.
</div>
<div id="ref-jurenaite_setquence_2024" class="csl-entry" role="listitem">
Jurenaite, Neringa, Daniel León-Periñán, Veronika Donath, Sunna Torge,
and René Jäkel. 2024. <span>“<span>SetQuence</span> &amp;
<span>SetOmic</span>: <span>Deep</span> Set Transformers for Whole
Genome and Exome Tumour Analysis.”</span> <em>BioSystems</em> 235
(January): 105095. <a href="https://doi.org/10.1016/j.biosystems.2023.105095">https://doi.org/10.1016/j.biosystems.2023.105095</a>.
</div>
<div id="ref-kagda_encode_2025" class="csl-entry" role="listitem">
Kagda, Meenakshi S., Bonita Lam, Casey Litton, Corinn Small, Cricket A.
Sloan, Emma Spragins, Forrest Tanaka, et al. 2025. <span>“Data
Navigation on the <span>ENCODE</span> Portal.”</span> <em>Nature
Communications</em> 16 (1): 9592. <a href="https://doi.org/10.1038/s41467-025-64343-9">https://doi.org/10.1038/s41467-025-64343-9</a>.
</div>
<div id="ref-karczewski_gnomad_2020" class="csl-entry" role="listitem">
Karczewski, Konrad J., Laurent C. Francioli, Grace Tiao, Beryl B.
Cummings, Jessica Alföldi, Qingbo Wang, Ryan L. Collins, et al. 2020.
<span>“The Mutational Constraint Spectrum Quantified from Variation in
141,456 Humans.”</span> <em>Nature</em> 581 (7809): 434–43. <a href="https://doi.org/10.1038/s41586-020-2308-7">https://doi.org/10.1038/s41586-020-2308-7</a>.
</div>
<div id="ref-kelley_basenji2_2020" class="csl-entry" role="listitem">
Kelley, David R. 2020. <span>“[<span>Basenji2</span>]
<span>Cross</span>-Species Regulatory Sequence Activity
Prediction.”</span> <em>PLOS Computational Biology</em> 16 (7):
e1008050. <a href="https://doi.org/10.1371/journal.pcbi.1008050">https://doi.org/10.1371/journal.pcbi.1008050</a>.
</div>
<div id="ref-kelley_basenji_2018" class="csl-entry" role="listitem">
Kelley, David R., Yakir A. Reshef, Maxwell Bileschi, David Belanger,
Cory Y. McLean, and Jasper Snoek. 2018. <span>“[<span>Basenji2</span>]
<span>Sequential</span> Regulatory Activity Prediction Across
Chromosomes with Convolutional Neural Networks.”</span> <em>Genome
Research</em> 28 (5): 739–50. <a href="https://doi.org/10.1101/gr.227819.117">https://doi.org/10.1101/gr.227819.117</a>.
</div>
<div id="ref-kipf_gcn_2017" class="csl-entry" role="listitem">
Kipf, Thomas N., and Max Welling. 2017.
<span>“Semi-<span>Supervised</span> <span>Classification</span> with
<span>Graph</span> <span>Convolutional</span>
<span>Networks</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.1609.02907">https://doi.org/10.48550/arXiv.1609.02907</a>.
</div>
<div id="ref-kircher_general_2014" class="csl-entry" role="listitem">
Kircher, Martin, Daniela M. Witten, Preti Jain, Brian J. O’Roak, Gregory
M. Cooper, and Jay Shendure. 2014. <span>“A General Framework for
Estimating the Relative Pathogenicity of Human Genetic Variants.”</span>
<em>Nature Genetics</em> 46 (3): 310–15. <a href="https://doi.org/10.1038/ng.2892">https://doi.org/10.1038/ng.2892</a>.
</div>
<div id="ref-krusche_happy_2019" class="csl-entry" role="listitem">
Krusche, Peter, Len Trigg, Paul C. Boutros, Christopher E. Mason,
Francisco M. De La Vega, Benjamin L. Moore, Mar Gonzalez-Porta, et al.
2019. <span>“Best <span>Practices</span> for <span>Benchmarking</span>
<span>Germline</span> <span>Small</span> <span>Variant</span>
<span>Calls</span> in <span>Human</span> <span>Genomes</span>.”</span>
<em>Nature Biotechnology</em> 37 (5): 555–60. <a href="https://doi.org/10.1038/s41587-019-0054-x">https://doi.org/10.1038/s41587-019-0054-x</a>.
</div>
<div id="ref-kundaje_roadmap_2015" class="csl-entry" role="listitem">
Kundaje, Anshul, Wouter Meuleman, Jason Ernst, Misha Bilenky, Angela
Yen, Alireza Heravi-Moussavi, Pouya Kheradpour, et al. 2015.
<span>“Integrative Analysis of 111 Reference Human Epigenomes.”</span>
<em>Nature</em> 518 (7539): 317–30. <a href="https://doi.org/10.1038/nature14248">https://doi.org/10.1038/nature14248</a>.
</div>
<div id="ref-kurki_finngen_2023" class="csl-entry" role="listitem">
Kurki, Mitja I., Juha Karjalainen, Priit Palta, Timo P. Sipilä, Kati
Kristiansson, Kati M. Donner, Mary P. Reeve, et al. 2023.
<span>“<span>FinnGen</span> Provides Genetic Insights from a
Well-Phenotyped Isolated Population.”</span> <em>Nature</em> 613 (7944):
508–18. <a href="https://doi.org/10.1038/s41586-022-05473-8">https://doi.org/10.1038/s41586-022-05473-8</a>.
</div>
<div id="ref-lambert_pgs-catalog_2021" class="csl-entry" role="listitem">
Lambert, Samuel A., Laurent Gil, Simon Jupp, Scott C. Ritchie, Yu Xu,
Annalisa Buniello, Aoife McMahon, et al. 2021. <span>“The
<span>Polygenic</span> <span>Score</span> <span>Catalog</span> as an
Open Database for Reproducibility and Systematic Evaluation.”</span>
<em>Nature Genetics</em> 53 (4): 420–25. <a href="https://doi.org/10.1038/s41588-021-00783-5">https://doi.org/10.1038/s41588-021-00783-5</a>.
</div>
<div id="ref-landrum_clinvar_2018" class="csl-entry" role="listitem">
Landrum, Melissa J, Jennifer M Lee, Mark Benson, Garth R Brown, Chen
Chao, Shanmuga Chitipiralla, Baoshan Gu, et al. 2018.
<span>“<span>ClinVar</span>: Improving Access to Variant Interpretations
and Supporting Evidence.”</span> <em>Nucleic Acids Research</em> 46
(D1): D1062–67. <a href="https://doi.org/10.1093/nar/gkx1153">https://doi.org/10.1093/nar/gkx1153</a>.
</div>
<div id="ref-lee_g2pt_2025" class="csl-entry" role="listitem">
Lee, Ingoo, Zachary S. Wallace, Yuqi Wang, Sungjoon Park, Hojung Nam,
Amit R. Majithia, and Trey Ideker. 2025. <span>“[<span>G2PT</span>]
<span>A</span> Genotype-Phenotype Transformer to Assess and Explain
Polygenic Risk.”</span> bioRxiv. <a href="https://doi.org/10.1101/2024.10.23.619940">https://doi.org/10.1101/2024.10.23.619940</a>.
</div>
<div id="ref-li_cgmega_2024" class="csl-entry" role="listitem">
Li, Hao, Zebei Han, Yu Sun, Fu Wang, Pengzhen Hu, Yuang Gao, Xuemei Bai,
et al. 2024. <span>“<span>CGMega</span>: Explainable Graph Neural
Network Framework with Attention Mechanisms for Cancer Gene Module
Dissection.”</span> <em>Nature Communications</em> 15 (1): 5997. <a href="https://doi.org/10.1038/s41467-024-50426-6">https://doi.org/10.1038/s41467-024-50426-6</a>.
</div>
<div id="ref-li_bwa-mem_2013" class="csl-entry" role="listitem">
Li, Heng. 2013. <span>“Aligning Sequence Reads, Clone Sequences and
Assembly Contigs with <span>BWA</span>-<span>MEM</span>.”</span> arXiv.
<a href="https://doi.org/10.48550/arXiv.1303.3997">https://doi.org/10.48550/arXiv.1303.3997</a>.
</div>
<div id="ref-li_mapping_2014" class="csl-entry" role="listitem">
———. 2014. <span>“Towards <span>Better</span> <span>Understanding</span>
of <span>Artifacts</span> in <span>Variant</span> <span>Calling</span>
from <span>High</span>-<span>Coverage</span>
<span>Samples</span>.”</span> <em>Bioinformatics</em> 30 (20): 2843–51.
<a href="https://doi.org/10.1093/bioinformatics/btu356">https://doi.org/10.1093/bioinformatics/btu356</a>.
</div>
<div id="ref-li_minimap2_2018" class="csl-entry" role="listitem">
———. 2018. <span>“Minimap2: Pairwise Alignment for Nucleotide
Sequences.”</span> <em>Bioinformatics</em> 34 (18): 3094–3100. <a href="https://doi.org/10.1093/bioinformatics/bty191">https://doi.org/10.1093/bioinformatics/bty191</a>.
</div>
<div id="ref-li_mogcn_2022" class="csl-entry" role="listitem">
Li, Xiao, Jie Ma, Ling Leng, Mingfei Han, Mansheng Li, Fuchu He, and
Yunping Zhu. 2022. <span>“<span>MoGCN</span>: <span>A</span>
<span>Multi</span>-<span>Omics</span> <span>Integration</span>
<span>Method</span> <span>Based</span> on <span>Graph</span>
<span>Convolutional</span> <span>Network</span> for <span>Cancer</span>
<span>Subtype</span> <span>Analysis</span>.”</span> <em>Frontiers in
Genetics</em> 13 (February). <a href="https://doi.org/10.3389/fgene.2022.806842">https://doi.org/10.3389/fgene.2022.806842</a>.
</div>
<div id="ref-li_genomic_2023" class="csl-entry" role="listitem">
Li, Zehui, Akashaditya Das, William A. V. Beardall, Yiren Zhao, and
Guy-Bart Stan. 2023. <span>“Genomic <span>Interpreter</span>:
<span>A</span> <span>Hierarchical</span> <span>Genomic</span>
<span>Deep</span> <span>Neural</span> <span>Network</span> with
<span>1D</span> <span>Shifted</span> <span>Window</span>
<span>Transformer</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2306.05143">https://doi.org/10.48550/arXiv.2306.05143</a>.
</div>
<div id="ref-li_omnidna_2025" class="csl-entry" role="listitem">
Li, Zehui, Vallijah Subasri, Yifei Shen, Dongsheng Li, Yiren Zhao,
Guy-Bart Stan, and Caihua Shan. 2025. <span>“Omni-<span>DNA</span>:
<span>A</span> <span>Unified</span> <span>Genomic</span>
<span>Foundation</span> <span>Model</span> for
<span>Cross</span>-<span>Modal</span> and
<span>Multi</span>-<span>Task</span> <span>Learning</span>.”</span>
arXiv. <a href="https://doi.org/10.48550/arXiv.2502.03499">https://doi.org/10.48550/arXiv.2502.03499</a>.
</div>
<div id="ref-liao_pangenome_2023" class="csl-entry" role="listitem">
Liao, Wen-Wei, Mobin Asri, Jana Ebler, Daniel Doerr, Marina Haukness,
Glenn Hickey, Shuangjia Lu, et al. 2023. <span>“A Draft Human Pangenome
Reference.”</span> <em>Nature</em> 617 (7960): 312–24. <a href="https://doi.org/10.1038/s41586-023-05896-x">https://doi.org/10.1038/s41586-023-05896-x</a>.
</div>
<div id="ref-lin_esm-2_2022" class="csl-entry" role="listitem">
Lin, Zeming, Halil Akin, Roshan Rao, Brian Hie, Zhongkai Zhu, Wenting
Lu, Allan dos Santos Costa, et al. 2022. <span>“[<span>ESM</span>-2]
<span>Language</span> Models of Protein Sequences at the Scale of
Evolution Enable Accurate Structure Prediction.”</span> bioRxiv. <a href="https://doi.org/10.1101/2022.07.20.500902">https://doi.org/10.1101/2022.07.20.500902</a>.
</div>
<div id="ref-linder_borzoi_2025" class="csl-entry" role="listitem">
Linder, Johannes, Divyanshi Srivastava, Han Yuan, Vikram Agarwal, and
David R. Kelley. 2025. <span>“[<span>Borzoi</span>]
<span>Predicting</span> <span>RNA</span>-Seq Coverage from
<span>DNA</span> Sequence as a Unifying Model of Gene
Regulation.”</span> <em>Nature Genetics</em> 57 (4): 949–61. <a href="https://doi.org/10.1038/s41588-024-02053-6">https://doi.org/10.1038/s41588-024-02053-6</a>.
</div>
<div id="ref-liu_life-code_2025" class="csl-entry" role="listitem">
Liu, Zicheng, Siyuan Li, Zhiyuan Chen, Fang Wu, Chang Yu, Qirong Yang,
Yucheng Guo, Yujie Yang, Xiaoming Zhang, and Stan Z. Li. 2025.
<span>“Life-<span>Code</span>: <span>Central</span> <span>Dogma</span>
<span>Modeling</span> with <span>Multi</span>-<span>Omics</span>
<span>Sequence</span> <span>Unification</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2502.07299">https://doi.org/10.48550/arXiv.2502.07299</a>.
</div>
<div id="ref-loh_eagle_2016" class="csl-entry" role="listitem">
Loh, Po-Ru, Petr Danecek, Pier Francesco Palamara, Christian
Fuchsberger, Yakir A Reshef, Hilary K Finucane, Sebastian Schoenherr, et
al. 2016. <span>“Reference-Based Phasing Using the
<span>Haplotype</span> <span>Reference</span> <span>Consortium</span>
Panel.”</span> <em>Nature Genetics</em> 48 (11): 1443–48. <a href="https://doi.org/10.1038/ng.3679">https://doi.org/10.1038/ng.3679</a>.
</div>
<div id="ref-madani_progen_2023" class="csl-entry" role="listitem">
Madani, Ali, Ben Krause, Eric R. Greene, Subu Subramanian, Benjamin P.
Mohr, James M. Holton, Jose Luis Olmos, et al. 2023. <span>“Large
Language Models Generate Functional Protein Sequences Across Diverse
Families.”</span> <em>Nature Biotechnology</em> 41 (8): 1099–1106. <a href="https://doi.org/10.1038/s41587-022-01618-2">https://doi.org/10.1038/s41587-022-01618-2</a>.
</div>
<div id="ref-mallal_abacavir_2008" class="csl-entry" role="listitem">
Mallal, Simon, Elizabeth Phillips, Giampiero Carosi, Jean-Michel Molina,
Cassy Workman, Janez Tomažič, Eva Jägel-Guedes, et al. 2008.
<span>“<span>HLA</span>-<span>B</span>*5701 <span>Screening</span> for
<span>Hypersensitivity</span> to <span>Abacavir</span>.”</span> <em>New
England Journal of Medicine</em> 358 (6): 568–79. <a href="https://doi.org/10.1056/NEJMoa0706135">https://doi.org/10.1056/NEJMoa0706135</a>.
</div>
<div id="ref-manolio_finding_2009" class="csl-entry" role="listitem">
Manolio, Teri A., Francis S. Collins, Nancy J. Cox, David B. Goldstein,
Lucia A. Hindorff, David J. Hunter, Mark I. McCarthy, et al. 2009.
<span>“Finding the Missing Heritability of Complex Diseases.”</span>
<em>Nature</em> 461 (7265): 747–53. <a href="https://doi.org/10.1038/nature08494">https://doi.org/10.1038/nature08494</a>.
</div>
<div id="ref-manzo_comparative_2025" class="csl-entry" role="listitem">
Manzo, Gaetano, Kathryn Borkowski, and Ivan Ovcharenko. 2025.
<span>“Comparative <span>Analysis</span> of <span>Deep</span>
<span>Learning</span> <span>Models</span> for <span>Predicting</span>
<span>Causative</span> <span>Regulatory</span>
<span>Variants</span>.”</span> <em>bioRxiv: The Preprint Server for
Biology</em>, June, 2025.05.19.654920. <a href="https://doi.org/10.1101/2025.05.19.654920">https://doi.org/10.1101/2025.05.19.654920</a>.
</div>
<div id="ref-marees_gwas_2018" class="csl-entry" role="listitem">
Marees, Andries T., Hilde de Kluiver, Sven Stringer, Florence Vorspan,
Emmanuel Curis, Cynthia Marie-Claire, and Eske M. Derks. 2018.
<span>“[<span>GWAS</span>] <span>A</span> Tutorial on Conducting
Genome-Wide Association Studies: <span>Quality</span> Control and
Statistical Analysis.”</span> <em>International Journal of Methods in
Psychiatric Research</em> 27 (2): e1608. <a href="https://doi.org/10.1002/mpr.1608">https://doi.org/10.1002/mpr.1608</a>.
</div>
<div id="ref-marquet_vespag_2024" class="csl-entry" role="listitem">
Marquet, Céline, Julius Schlensok, Marina Abakarova, Burkhard Rost, and
Elodie Laine. 2024. <span>“[<span>VespaG</span>]
<span>Expert</span>-Guided Protein Language Models Enable Accurate and
Blazingly Fast Fitness Prediction.”</span> <em>Bioinformatics</em> 40
(11): btae621. <a href="https://doi.org/10.1093/bioinformatics/btae621">https://doi.org/10.1093/bioinformatics/btae621</a>.
</div>
<div id="ref-maurano_noncode_2012" class="csl-entry" role="listitem">
Maurano, Matthew T., Richard Humbert, Eric Rynes, Robert E. Thurman,
Eric Haugen, Hao Wang, Alex P. Reynolds, et al. 2012. <span>“Systematic
<span>Localization</span> of <span>Common</span>
<span>Disease</span>-<span>Associated</span> <span>Variation</span> in
<span>Regulatory</span> <span>DNA</span>.”</span> <em>Science</em> 337
(6099): 1190–95. <a href="https://doi.org/10.1126/science.1222794">https://doi.org/10.1126/science.1222794</a>.
</div>
<div id="ref-medvedev_biotoken_2025" class="csl-entry" role="listitem">
Medvedev, Aleksandr, Karthik Viswanathan, Praveenkumar Kanithi, Kirill
Vishniakov, Prateek Munjal, Clément Christophe, Marco AF Pimentel,
Ronnie Rajan, and Shadab Khan. 2025. <span>“<span>BioToken</span> and
<span>BioFM</span> – <span>Biologically</span>-<span>Informed</span>
<span>Tokenization</span> <span>Enables</span> <span>Accurate</span> and
<span>Efficient</span> <span>Genomic</span> <span>Foundation</span>
<span>Models</span>.”</span> bioRxiv. <a href="https://doi.org/10.1101/2025.03.27.645711">https://doi.org/10.1101/2025.03.27.645711</a>.
</div>
<div id="ref-meier_esm-1v_2021" class="csl-entry" role="listitem">
Meier, Joshua, Roshan Rao, Robert Verkuil, Jason Liu, Tom Sercu, and
Alexander Rives. 2021. <span>“[<span>ESM</span>-1v]
<span>Language</span> Models Enable Zero-Shot Prediction of the Effects
of Mutations on Protein Function.”</span> bioRxiv. <a href="https://doi.org/10.1101/2021.07.09.450648">https://doi.org/10.1101/2021.07.09.450648</a>.
</div>
<div id="ref-mirdita_colabfold_2022" class="csl-entry" role="listitem">
Mirdita, Milot, Konstantin Schütze, Yoshitaka Moriwaki, Lim Heo, Sergey
Ovchinnikov, and Martin Steinegger. 2022. <span>“<span>ColabFold</span>:
Making Protein Folding Accessible to All.”</span> <em>Nature
Methods</em> 19 (6): 679–82. <a href="https://doi.org/10.1038/s41592-022-01488-1">https://doi.org/10.1038/s41592-022-01488-1</a>.
</div>
<div id="ref-morales_mane_2022" class="csl-entry" role="listitem">
Morales, Joannella, Shashikant Pujar, Jane E. Loveland, Alex Astashyn,
Ruth Bennett, Andrew Berry, Eric Cox, et al. 2022. <span>“A Joint
<span>NCBI</span> and <span>EMBL</span>-<span>EBI</span> Transcript Set
for Clinical Genomics and Research.”</span> <em>Nature</em> 604 (7905):
310–15. <a href="https://doi.org/10.1038/s41586-022-04558-8">https://doi.org/10.1038/s41586-022-04558-8</a>.
</div>
<div id="ref-mountjoy_open_2021" class="csl-entry" role="listitem">
Mountjoy, Edward, Ellen M. Schmidt, Miguel Carmona, Jeremy
Schwartzentruber, Gareth Peat, Alfredo Miranda, Luca Fumis, et al. 2021.
<span>“An Open Approach to Systematically Prioritize Causal Variants and
Genes at All Published Human <span>GWAS</span> Trait-Associated
Loci.”</span> <em>Nature Genetics</em> 53 (11): 1527–33. <a href="https://doi.org/10.1038/s41588-021-00945-5">https://doi.org/10.1038/s41588-021-00945-5</a>.
</div>
<div id="ref-mukherjee_embedgem_2024" class="csl-entry" role="listitem">
Mukherjee, Sumit, Zachary R. McCaw, Jingwen Pei, Anna Merkoulovitch, Tom
Soare, Raghav Tandon, David Amar, et al. 2024.
<span>“<span>EmbedGEM</span>: A Framework to Evaluate the Utility of
Embeddings for Genetic Discovery.”</span> <em>Bioinformatics
Advances</em> 4 (1). <a href="https://doi.org/10.1093/bioadv/vbae135">https://doi.org/10.1093/bioadv/vbae135</a>.
</div>
<div id="ref-naghipourfar_cdsfm_2024" class="csl-entry" role="listitem">
Naghipourfar, Mohsen, Siyu Chen, Mathew K. Howard, Christian B.
Macdonald, Ali Saberi, Timo Hagen, Mohammad R. K. Mofrad, Willow
Coyote-Maestas, and Hani Goodarzi. 2024. <span>“[<span class="nocase">cdsFM</span> - <span>EnCodon</span>/<span>DeCodon</span>]
<span>A</span> <span>Suite</span> of <span>Foundation</span>
<span>Models</span> <span>Captures</span> the <span>Contextual</span>
<span>Interplay</span> <span>Between</span> <span>Codons</span>.”</span>
bioRxiv. <a href="https://doi.org/10.1101/2024.10.10.617568">https://doi.org/10.1101/2024.10.10.617568</a>.
</div>
<div id="ref-ng_sift_2003" class="csl-entry" role="listitem">
Ng, Pauline C., and Steven Henikoff. 2003. <span>“<span>SIFT</span>:
<span>Predicting</span> Amino Acid Changes That Affect Protein
Function.”</span> <em>Nucleic Acids Research</em> 31 (13): 3812–14. <a href="https://doi.org/10.1093/nar/gkg509">https://doi.org/10.1093/nar/gkg509</a>.
</div>
<div id="ref-nguyen_hyenadna_2023" class="csl-entry" role="listitem">
Nguyen, Eric, Michael Poli, Marjan Faizi, Armin Thomas, Callum
Birch-Sykes, Michael Wornow, Aman Patel, et al. 2023.
<span>“<span>HyenaDNA</span>: <span>Long</span>-<span>Range</span>
<span>Genomic</span> <span>Sequence</span> <span>Modeling</span> at
<span>Single</span> <span>Nucleotide</span>
<span>Resolution</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2306.15794">https://doi.org/10.48550/arXiv.2306.15794</a>.
</div>
<div id="ref-nielsen_error_2011" class="csl-entry" role="listitem">
Nielsen, Rasmus, Joshua S. Paul, Anders Albrechtsen, and Yun S. Song.
2011. <span>“Genotype and <span>SNP</span> Calling from Next-Generation
Sequencing Data.”</span> <em>Nature Reviews. Genetics</em> 12 (6):
443–51. <a href="https://doi.org/10.1038/nrg2986">https://doi.org/10.1038/nrg2986</a>.
</div>
<div id="ref-notin_proteingym_2023" class="csl-entry" role="listitem">
Notin, Pascal, Aaron Kollasch, Daniel Ritter, Lood van Niekerk,
Steffanie Paul, Han Spinner, Nathan Rollins, et al. 2023.
<span>“<span>ProteinGym</span>: <span>Large</span>-<span>Scale</span>
<span>Benchmarks</span> for <span>Protein</span> <span>Fitness</span>
<span>Prediction</span> and <span>Design</span>.”</span> <em>Advances in
Neural Information Processing Systems</em> 36 (December): 64331–79. <a href="https://papers.nips.cc/paper_files/paper/2023/hash/cac723e5ff29f65e3fcbb0739ae91bee-Abstract-Datasets_and_Benchmarks.html">https://papers.nips.cc/paper_files/paper/2023/hash/cac723e5ff29f65e3fcbb0739ae91bee-Abstract-Datasets_and_Benchmarks.html</a>.
</div>
<div id="ref-nurk_complete_2022" class="csl-entry" role="listitem">
Nurk, Sergey, Sergey Koren, Arang Rhie, Mikko Rautiainen, Andrey V.
Bzikadze, Alla Mikheenko, Mitchell R. Vollger, et al. 2022. <span>“The
Complete Sequence of a Human Genome.”</span> <em>Science</em> 376
(6588): 44–53. <a href="https://doi.org/10.1126/science.abj6987">https://doi.org/10.1126/science.abj6987</a>.
</div>
<div id="ref-oconnell_shapeit2_2014" class="csl-entry" role="listitem">
O’Connell, Jared, Deepti Gurdasani, Olivier Delaneau, Nicola Pirastu,
Sheila Ulivi, Massimiliano Cocca, Michela Traglia, et al. 2014. <span>“A
<span>General</span> <span>Approach</span> for <span>Haplotype</span>
<span>Phasing</span> Across the <span>Full</span> <span>Spectrum</span>
of <span>Relatedness</span>.”</span> <em>PLOS Genetics</em> 10 (4):
e1004234. <a href="https://doi.org/10.1371/journal.pgen.1004234">https://doi.org/10.1371/journal.pgen.1004234</a>.
</div>
<div id="ref-oleary_refseq_2016" class="csl-entry" role="listitem">
O’Leary, Nuala A., Mathew W. Wright, J. Rodney Brister, Stacy Ciufo,
Diana Haddad, Rich McVeigh, Bhanu Rajput, et al. 2016. <span>“Reference
Sequence (<span>RefSeq</span>) Database at <span>NCBI</span>: Current
Status, Taxonomic Expansion, and Functional Annotation.”</span>
<em>Nucleic Acids Research</em> 44 (D1): D733–45. <a href="https://doi.org/10.1093/nar/gkv1189">https://doi.org/10.1093/nar/gkv1189</a>.
</div>
<div id="ref-orenbuch_popeve_2025" class="csl-entry" role="listitem">
Orenbuch, Rose, Courtney A. Shearer, Aaron W. Kollasch, Aviv D. Spinner,
Thomas Hopf, Lood van Niekerk, Dinko Franceschi, Mafalda Dias, Jonathan
Frazer, and Debora S. Marks. 2025. <span>“[<span class="nocase">popEVE</span>] <span>Proteome</span>-Wide Model for Human
Disease Genetics.”</span> <em>Nature Genetics</em>, November, 1–10. <a href="https://doi.org/10.1038/s41588-025-02400-1">https://doi.org/10.1038/s41588-025-02400-1</a>.
</div>
<div id="ref-noauthor_pbsv_2025" class="csl-entry" role="listitem">
<span>“<span>PacificBiosciences</span>/Pbsv.”</span> 2025. PacBio. <a href="https://github.com/PacificBiosciences/pbsv">https://github.com/PacificBiosciences/pbsv</a>.
</div>
<div id="ref-pasaniuc_dissecting_2016" class="csl-entry" role="listitem">
Pasaniuc, Bogdan, and Alkes L. Price. 2016. <span>“Dissecting the
Genetics of Complex Traits Using Summary Association Statistics.”</span>
<em>Nature Reviews Genetics</em> 18 (2): 117–27. <a href="https://doi.org/10.1038/nrg.2016.142">https://doi.org/10.1038/nrg.2016.142</a>.
</div>
<div id="ref-patterson_population_2006" class="csl-entry" role="listitem">
Patterson, Nick, Alkes L. Price, and David Reich. 2006.
<span>“Population <span>Structure</span> and
<span>Eigenanalysis</span>.”</span> <em>PLOS Genetics</em> 2 (12): e190.
<a href="https://doi.org/10.1371/journal.pgen.0020190">https://doi.org/10.1371/journal.pgen.0020190</a>.
</div>
<div id="ref-peer_estimation_2008" class="csl-entry" role="listitem">
Pe’er, Itsik, Roman Yelensky, David Altshuler, and Mark J. Daly. 2008.
<span>“Estimation of the Multiple Testing Burden for Genomewide
Association Studies of Nearly All Common Variants.”</span> <em>Genetic
Epidemiology</em> 32 (4): 381–85. <a href="https://doi.org/10.1002/gepi.20303">https://doi.org/10.1002/gepi.20303</a>.
</div>
<div id="ref-pearce_transcriptformer_2025" class="csl-entry" role="listitem">
Pearce, James D., Sara E. Simmonds, Gita Mahmoudabadi, Lakshmi Krishnan,
Giovanni Palla, Ana-Maria Istrate, Alexander Tarashansky, et al. 2025.
<span>“[<span>TranscriptFormer</span>]
<span>Cross</span>-<span>Species</span> <span>Generative</span>
<span>Cell</span> <span>Atlas</span> <span>Across</span> 1.5
<span>Billion</span> <span>Years</span> of <span>Evolution</span>:
<span>The</span> <span>TranscriptFormer</span> <span>Single</span>-Cell
<span>Model</span>.”</span> bioRxiv. <a href="https://doi.org/10.1101/2025.04.25.650731">https://doi.org/10.1101/2025.04.25.650731</a>.
</div>
<div id="ref-pejaver_calibration_2022" class="csl-entry" role="listitem">
Pejaver, Vikas, Alicia B. Byrne, Bing-Jian Feng, Kymberleigh A. Pagel,
Sean D. Mooney, Rachel Karchin, Anne O’Donnell-Luria, et al. 2022.
<span>“Calibration of Computational Tools for Missense Variant
Pathogenicity Classification and <span>ClinGen</span> Recommendations
for <span>PP3</span>/<span>BP4</span> Criteria.”</span> <em>American
Journal of Human Genetics</em> 109 (12): 2163–77. <a href="https://doi.org/10.1016/j.ajhg.2022.10.013">https://doi.org/10.1016/j.ajhg.2022.10.013</a>.
</div>
<div id="ref-poplin_deepvariant_2018" class="csl-entry" role="listitem">
Poplin, Ryan, Pi-Chuan Chang, David Alexander, Scott Schwartz, Thomas
Colthurst, Alexander Ku, Dan Newburger, et al. 2018.
<span>“[<span>DeepVariant</span>] <span>A</span> Universal
<span>SNP</span> and Small-Indel Variant Caller Using Deep Neural
Networks.”</span> <em>Nature Biotechnology</em> 36 (10): 983–87. <a href="https://doi.org/10.1038/nbt.4235">https://doi.org/10.1038/nbt.4235</a>.
</div>
<div id="ref-price_pca_2006" class="csl-entry" role="listitem">
Price, Alkes L., Nick J. Patterson, Robert M. Plenge, Michael E.
Weinblatt, Nancy A. Shadick, and David Reich. 2006. <span>“Principal
Components Analysis Corrects for Stratification in Genome-Wide
Association Studies.”</span> <em>Nature Genetics</em> 38 (8): 904–9. <a href="https://doi.org/10.1038/ng1847">https://doi.org/10.1038/ng1847</a>.
</div>
<div id="ref-raffel_t5_2019" class="csl-entry" role="listitem">
Raffel, Colin, Noam Shazeer, Adam Roberts, Katherine Lee, Sharan Narang,
Michael Matena, Yanqi Zhou, Wei Li, and Peter J. Liu. 2019.
<span>“Exploring the <span>Limits</span> of <span>Transfer</span>
<span>Learning</span> with a <span>Unified</span>
<span>Text</span>-to-<span>Text</span> <span>Transformer</span>.”</span>
arXiv. <a href="https://doi.org/10.48550/arXiv.1910.10683">https://doi.org/10.48550/arXiv.1910.10683</a>.
</div>
<div id="ref-rakowski_mifm_2025" class="csl-entry" role="listitem">
Rakowski, Alexander, and Christoph Lippert. 2025.
<span>“[<span>MIFM</span>] <span>Multiple</span> Instance Fine-Mapping:
Predicting Causal Regulatory Variants with a Deep Sequence
Model.”</span> medRxiv. <a href="https://doi.org/10.1101/2025.06.13.25329551">https://doi.org/10.1101/2025.06.13.25329551</a>.
</div>
<div id="ref-rao_tape_2019" class="csl-entry" role="listitem">
Rao, Roshan, Nicholas Bhattacharya, Neil Thomas, Yan Duan, Xi Chen, John
Canny, Pieter Abbeel, and Yun S. Song. 2019. <span>“Evaluating
<span>Protein</span> <span>Transfer</span> <span>Learning</span> with
<span>TAPE</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.1906.08230">https://doi.org/10.48550/arXiv.1906.08230</a>.
</div>
<div id="ref-noauthor_rtg-core_2025" class="csl-entry" role="listitem">
<span>“<span>RealTimeGenomics</span>/Rtg-Core.”</span> 2025. Real Time
Genomics. <a href="https://github.com/RealTimeGenomics/rtg-core">https://github.com/RealTimeGenomics/rtg-core</a>.
</div>
<div id="ref-regev_cell-atlas_2017" class="csl-entry" role="listitem">
Regev, Aviv, Sarah A Teichmann, Eric S Lander, Ido Amit, Christophe
Benoist, Ewan Birney, Bernd Bodenmiller, et al. 2017. <span>“The
<span>Human</span> <span>Cell</span> <span>Atlas</span>.”</span> Edited
by Thomas R Gingeras. <em>eLife</em> 6 (December): e27041. <a href="https://doi.org/10.7554/eLife.27041">https://doi.org/10.7554/eLife.27041</a>.
</div>
<div id="ref-rehm_clingen_2015" class="csl-entry" role="listitem">
Rehm, Heidi L., Jonathan S. Berg, Lisa D. Brooks, Carlos D. Bustamante,
James P. Evans, Melissa J. Landrum, David H. Ledbetter, et al. 2015.
<span>“<span>ClinGen</span> — <span>The</span> <span>Clinical</span>
<span>Genome</span> <span>Resource</span>.”</span> <em>New England
Journal of Medicine</em> 372 (23): 2235–42. <a href="https://doi.org/10.1056/NEJMsr1406261">https://doi.org/10.1056/NEJMsr1406261</a>.
</div>
<div id="ref-remmert_hhblits_2012" class="csl-entry" role="listitem">
Remmert, Michael, Andreas Biegert, Andreas Hauser, and Johannes Söding.
2012. <span>“<span>HHblits</span>: Lightning-Fast Iterative Protein
Sequence Searching by <span>HMM</span>-<span>HMM</span>
Alignment.”</span> <em>Nature Methods</em> 9 (2): 173–75. <a href="https://doi.org/10.1038/nmeth.1818">https://doi.org/10.1038/nmeth.1818</a>.
</div>
<div id="ref-rentzsch_cadd_2019" class="csl-entry" role="listitem">
Rentzsch, Philipp, Daniela Witten, Gregory M Cooper, Jay Shendure, and
Martin Kircher. 2019. <span>“<span>CADD</span>: Predicting the
Deleteriousness of Variants Throughout the Human Genome.”</span>
<em>Nucleic Acids Research</em> 47 (D1): D886–94. <a href="https://doi.org/10.1093/nar/gky1016">https://doi.org/10.1093/nar/gky1016</a>.
</div>
<div id="ref-riesselman_deepsequence_2018" class="csl-entry" role="listitem">
Riesselman, Adam J., John B. Ingraham, and Debora S. Marks. 2018.
<span>“Deep Generative Models of Genetic Variation Capture the Effects
of Mutations.”</span> <em>Nature Methods</em> 15 (10): 816–22. <a href="https://doi.org/10.1038/s41592-018-0138-4">https://doi.org/10.1038/s41592-018-0138-4</a>.
</div>
<div id="ref-rives_esm_2021" class="csl-entry" role="listitem">
Rives, Alexander, Joshua Meier, Tom Sercu, Siddharth Goyal, Zeming Lin,
Jason Liu, Demi Guo, et al. 2021. <span>“[<span>ESM</span>-1b]
<span>Biological</span> Structure and Function Emerge from Scaling
Unsupervised Learning to 250 Million Protein Sequences.”</span>
<em>Proceedings of the National Academy of Sciences of the United States
of America</em> 118 (15): e2016239118. <a href="https://doi.org/10.1073/pnas.2016239118">https://doi.org/10.1073/pnas.2016239118</a>.
</div>
<div id="ref-robinson_hla-db_2020" class="csl-entry" role="listitem">
Robinson, James, Dominic J Barker, Xenia Georgiou, Michael A Cooper,
Paul Flicek, and Steven G E Marsh. 2020.
<span>“<span>IPD</span>-<span>IMGT</span>/<span>HLA</span>
<span>Database</span>.”</span> <em>Nucleic Acids Research</em> 48 (D1):
D948–55. <a href="https://doi.org/10.1093/nar/gkz950">https://doi.org/10.1093/nar/gkz950</a>.
</div>
<div id="ref-sakaue_hla_2023" class="csl-entry" role="listitem">
Sakaue, Saori, Saisriram Gurajala, Michelle Curtis, Yang Luo, Wanson
Choi, Kazuyoshi Ishigaki, Joyce B. Kang, et al. 2023. <span>“Tutorial: A
Statistical Genetics Guide to Identifying <span>HLA</span> Alleles
Driving Complex Disease.”</span> <em>Nature Protocols</em> 18 (9):
2625–41. <a href="https://doi.org/10.1038/s41596-023-00853-4">https://doi.org/10.1038/s41596-023-00853-4</a>.
</div>
<div id="ref-sanabria_grover_2024" class="csl-entry" role="listitem">
Sanabria, Melissa, Jonas Hirsch, Pierre M. Joubert, and Anna R. Poetsch.
2024. <span>“[<span>GROVER</span>] <span>DNA</span> Language Model
<span>GROVER</span> Learns Sequence Context in the Human Genome.”</span>
<em>Nature Machine Intelligence</em> 6 (8): 911–23. <a href="https://doi.org/10.1038/s42256-024-00872-0">https://doi.org/10.1038/s42256-024-00872-0</a>.
</div>
<div id="ref-schiff_caduceus_2024" class="csl-entry" role="listitem">
Schiff, Yair, Chia-Hsiang Kao, Aaron Gokaslan, Tri Dao, Albert Gu, and
Volodymyr Kuleshov. 2024. <span>“Caduceus:
<span>Bi</span>-<span>Directional</span> <span>Equivariant</span>
<span>Long</span>-<span>Range</span> <span>DNA</span>
<span>Sequence</span> <span>Modeling</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2403.03234">https://doi.org/10.48550/arXiv.2403.03234</a>.
</div>
<div id="ref-schubach_cadd_2024" class="csl-entry" role="listitem">
Schubach, Max, Thorben Maass, Lusiné Nazaretyan, Sebastian Röner, and
Martin Kircher. 2024. <span>“<span>CADD</span> V1.7: Using Protein
Language Models, Regulatory <span>CNNs</span> and Other Nucleotide-Level
Scores to Improve Genome-Wide Variant Predictions.”</span> <em>Nucleic
Acids Research</em> 52 (D1): D1143–54. <a href="https://doi.org/10.1093/nar/gkad989">https://doi.org/10.1093/nar/gkad989</a>.
</div>
<div id="ref-shafin_pepper_2021" class="csl-entry" role="listitem">
Shafin, Kishwar, Trevor Pesout, Pi-Chuan Chang, Maria Nattestad, Alexey
Kolesnikov, Sidharth Goel, Gunjan Baid, et al. 2021.
<span>“Haplotype-Aware Variant Calling with
<span>PEPPER</span>-<span>Margin</span>-<span>DeepVariant</span> Enables
High Accuracy in Nanopore Long-Reads.”</span> <em>Nature Methods</em> 18
(11): 1322–32. <a href="https://doi.org/10.1038/s41592-021-01299-w">https://doi.org/10.1038/s41592-021-01299-w</a>.
</div>
<div id="ref-sherry_dbsnp_2001" class="csl-entry" role="listitem">
Sherry, S. T., M.-H. Ward, M. Kholodov, J. Baker, L. Phan, E. M.
Smigielski, and K. Sirotkin. 2001. <span>“<span class="nocase">dbSNP</span>: The <span>NCBI</span> Database of Genetic
Variation.”</span> <em>Nucleic Acids Research</em> 29 (1): 308–11. <a href="https://doi.org/10.1093/nar/29.1.308">https://doi.org/10.1093/nar/29.1.308</a>.
</div>
<div id="ref-siepel_phastcons_2005" class="csl-entry" role="listitem">
Siepel, Adam, Gill Bejerano, Jakob S. Pedersen, Angie S. Hinrichs,
Minmei Hou, Kate Rosenbloom, Hiram Clawson, et al. 2005.
<span>“[<span>PhastCons</span>] <span>Evolutionarily</span> Conserved
Elements in Vertebrate, Insect, Worm, and Yeast Genomes.”</span>
<em>Genome Research</em> 15 (8): 1034–50. <a href="https://doi.org/10.1101/gr.3715005">https://doi.org/10.1101/gr.3715005</a>.
</div>
<div id="ref-sirugo_diversity_2019" class="csl-entry" role="listitem">
Sirugo, Giorgio, Scott M. Williams, and Sarah A. Tishkoff. 2019.
<span>“The <span>Missing</span> <span>Diversity</span> in
<span>Human</span> <span>Genetic</span> <span>Studies</span>.”</span>
<em>Cell</em> 177 (1): 26–31. <a href="https://doi.org/10.1016/j.cell.2019.02.048">https://doi.org/10.1016/j.cell.2019.02.048</a>.
</div>
<div id="ref-smolka_sniffles2_2024" class="csl-entry" role="listitem">
Smolka, Moritz, Luis F. Paulin, Christopher M. Grochowski, Dominic W.
Horner, Medhat Mahmoud, Sairam Behera, Ester Kalef-Ezra, et al. 2024.
<span>“Detection of Mosaic and Population-Level Structural Variants with
<span>Sniffles2</span>.”</span> <em>Nature Biotechnology</em> 42 (10):
1571–80. <a href="https://doi.org/10.1038/s41587-023-02024-y">https://doi.org/10.1038/s41587-023-02024-y</a>.
</div>
<div id="ref-sollis_gwas-catalog_2023" class="csl-entry" role="listitem">
Sollis, Elliot, Abayomi Mosaku, Ala Abid, Annalisa Buniello, Maria
Cerezo, Laurent Gil, Tudor Groza, et al. 2023. <span>“The
<span>NHGRI</span>-<span>EBI</span> <span>GWAS</span>
<span>Catalog</span>: Knowledgebase and Deposition Resource.”</span>
<em>Nucleic Acids Research</em> 51 (D1): D977–85. <a href="https://doi.org/10.1093/nar/gkac1010">https://doi.org/10.1093/nar/gkac1010</a>.
</div>
<div id="ref-song_t1k_2022" class="csl-entry" role="listitem">
Song, Li, Gali Bai, X. Shirley Liu, Bo Li, and Heng Li. 2022.
<span>“<span>T1K</span>: Efficient and Accurate <span>KIR</span> and
<span>HLA</span> Genotyping with Next-Generation Sequencing
Data.”</span> bioRxiv. <a href="https://doi.org/10.1101/2022.10.26.513955">https://doi.org/10.1101/2022.10.26.513955</a>.
</div>
<div id="ref-steinegger_mmseqs2_2017" class="csl-entry" role="listitem">
Steinegger, Martin, and Johannes Söding. 2017.
<span>“<span>MMseqs2</span> Enables Sensitive Protein Sequence Searching
for the Analysis of Massive Data Sets.”</span> <em>Nature
Biotechnology</em> 35 (11): 1026–28. <a href="https://doi.org/10.1038/nbt.3988">https://doi.org/10.1038/nbt.3988</a>.
</div>
<div id="ref-sullivan_leveraging_2023" class="csl-entry" role="listitem">
Sullivan, Patrick F., Jennifer R. S. Meadows, Steven Gazal, BaDoi N.
Phan, Xue Li, Diane P. Genereux, Michael X. Dong, et al. 2023.
<span>“Leveraging Base-Pair Mammalian Constraint to Understand Genetic
Variation and Human Disease.”</span> <em>Science</em> 380 (6643):
eabn2937. <a href="https://doi.org/10.1126/science.abn2937">https://doi.org/10.1126/science.abn2937</a>.
</div>
<div id="ref-suzek_uniref_2007" class="csl-entry" role="listitem">
Suzek, Baris E., Hongzhan Huang, Peter McGarvey, Raja Mazumder, and
Cathy H. Wu. 2007. <span>“<span>UniRef</span>: Comprehensive and
Non-Redundant <span>UniProt</span> Reference Clusters.”</span>
<em>Bioinformatics</em> 23 (10): 1282–88. <a href="https://doi.org/10.1093/bioinformatics/btm098">https://doi.org/10.1093/bioinformatics/btm098</a>.
</div>
<div id="ref-gnomAD" class="csl-entry" role="listitem">
<span>“The <span>Genome</span> <span>Aggregation</span>
<span>Database</span> (<span class="nocase">gnomAD</span>).”</span> n.d.
Accessed July 3, 2025. <a href="https://www.nature.com/immersive/d42859-020-00002-x/index.html">https://www.nature.com/immersive/d42859-020-00002-x/index.html</a>.
</div>
<div id="ref-gtex_2020" class="csl-entry" role="listitem">
The GTEx Consortium. 2020. <span>“The <span>GTEx</span>
<span>Consortium</span> Atlas of Genetic Regulatory Effects Across Human
Tissues.”</span> <em>Science</em> 369 (6509): 1318–30. <a href="https://doi.org/10.1126/science.aaz1776">https://doi.org/10.1126/science.aaz1776</a>.
</div>
<div id="ref-tabula_sapiens_2022" class="csl-entry" role="listitem">
The Tabula Sapiens Consortium. 2022. <span>“The <span>Tabula</span>
<span>Sapiens</span>: <span>A</span> Multiple-Organ, Single-Cell
Transcriptomic Atlas of Humans.”</span> <em>Science</em> 376 (6594):
eabl4896. <a href="https://doi.org/10.1126/science.abl4896">https://doi.org/10.1126/science.abl4896</a>.
</div>
<div id="ref-theodoris_geneformer_2023" class="csl-entry" role="listitem">
Theodoris, Christina V., Ling Xiao, Anant Chopra, Mark D. Chaffin, Zeina
R. Al Sayed, Matthew C. Hill, Helene Mantineo, et al. 2023.
<span>“[<span>Geneformer</span>] <span>Transfer</span> Learning Enables
Predictions in Network Biology.”</span> <em>Nature</em> 618 (7965):
616–24. <a href="https://doi.org/10.1038/s41586-023-06139-9">https://doi.org/10.1038/s41586-023-06139-9</a>.
</div>
<div id="ref-trop_genomics_2024" class="csl-entry" role="listitem">
Trop, Evan, Yair Schiff, Edgar Mariano Marroquin, Chia Hsiang Kao, Aaron
Gokaslan, McKinley Polen, Mingyi Shao, et al. 2024. <span>“The
<span>Genomics</span> <span>Long</span>-<span>Range</span>
<span>Benchmark</span>: <span>Advancing</span> <span>DNA</span>
<span>Language</span> <span>Models</span>,”</span> October. <a href="https://openreview.net/forum?id=8O9HLDrmtq">https://openreview.net/forum?id=8O9HLDrmtq</a>.
</div>
<div id="ref-van_der_auwera_gatk_best_2018" class="csl-entry" role="listitem">
Van der Auwera, Geraldine A., Mauricio O. Carneiro, Christopher Hartl,
Ryan Poplin, Guillermo del Angel, Ami Levy-Moonshine, Tadeusz Jordan, et
al. 2018. <span>“From <span>FastQ</span> <span>Data</span> to
<span>High</span>-<span>Confidence</span> <span>Variant</span>
<span>Calls</span>: <span>The</span> <span>Genome</span>
<span>Analysis</span> <span>Toolkit</span> <span>Best</span>
<span>Practices</span> <span>Pipeline</span>.”</span> <em>Current
Protocols in Bioinformatics</em> 43 (1): 11.10.1–33. <a href="https://doi.org/10.1002/0471250953.bi1110s43">https://doi.org/10.1002/0471250953.bi1110s43</a>.
</div>
<div id="ref-vaswani_attention_2017" class="csl-entry" role="listitem">
Vaswani, Ashish, Noam Shazeer, Niki Parmar, Jakob Uszkoreit, Llion
Jones, Aidan N. Gomez, Lukasz Kaiser, and Illia Polosukhin. 2017.
<span>“Attention <span>Is</span> <span>All</span> <span>You</span>
<span>Need</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.1706.03762">https://doi.org/10.48550/arXiv.1706.03762</a>.
</div>
<div id="ref-velickovic_gat_2018" class="csl-entry" role="listitem">
Veličković, Petar, Guillem Cucurull, Arantxa Casanova, Adriana Romero,
Pietro Liò, and Yoshua Bengio. 2018. <span>“Graph <span>Attention</span>
<span>Networks</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.1710.10903">https://doi.org/10.48550/arXiv.1710.10903</a>.
</div>
<div id="ref-verma_diversity_2024" class="csl-entry" role="listitem">
Verma, Anurag, Jennifer E. Huffman, Alex Rodriguez, Mitchell Conery,
Molei Liu, Yuk-Lam Ho, Youngdae Kim, et al. 2024. <span>“Diversity and
Scale: <span>Genetic</span> Architecture of 2068 Traits in the
<span>VA</span> <span>Million</span> <span>Veteran</span>
<span>Program</span>.”</span> <em>Science</em> 385 (6706): eadj1182. <a href="https://doi.org/10.1126/science.adj1182">https://doi.org/10.1126/science.adj1182</a>.
</div>
<div id="ref-vilhjalmsson_modeling_2015" class="csl-entry" role="listitem">
Vilhjálmsson, Bjarni J., Jian Yang, Hilary K. Finucane, Alexander Gusev,
Sara Lindström, Stephan Ripke, Giulio Genovese, et al. 2015.
<span>“Modeling <span>Linkage</span> <span>Disequilibrium</span>
<span>Increases</span> <span>Accuracy</span> of <span>Polygenic</span>
<span>Risk</span> <span>Scores</span>.”</span> <em>American Journal of
Human Genetics</em> 97 (4): 576–92. <a href="https://doi.org/10.1016/j.ajhg.2015.09.001">https://doi.org/10.1016/j.ajhg.2015.09.001</a>.
</div>
<div id="ref-vosa_eqtl-gen_2021" class="csl-entry" role="listitem">
Võsa, Urmo, Annique Claringbould, Harm-Jan Westra, Marc Jan Bonder,
Patrick Deelen, Biao Zeng, Holger Kirsten, et al. 2021.
<span>“Large-Scale Cis- and Trans-<span class="nocase">eQTL</span>
Analyses Identify Thousands of Genetic Loci and Polygenic Scores That
Regulate Blood Gene Expression.”</span> <em>Nature Genetics</em> 53 (9):
1300–1310. <a href="https://doi.org/10.1038/s41588-021-00913-z">https://doi.org/10.1038/s41588-021-00913-z</a>.
</div>
<div id="ref-watson_rfdiffusion_2023" class="csl-entry" role="listitem">
Watson, Joseph L., David Juergens, Nathaniel R. Bennett, Brian L.
Trippe, Jason Yim, Helen E. Eisenach, Woody Ahern, et al. 2023.
<span>“De Novo Design of Protein Structure and Function with
<span>RFdiffusion</span>.”</span> <em>Nature</em> 620 (7976): 1089–1100.
<a href="https://doi.org/10.1038/s41586-023-06415-8">https://doi.org/10.1038/s41586-023-06415-8</a>.
</div>
<div id="ref-wenger_pacbiohifi_2019" class="csl-entry" role="listitem">
Wenger, Aaron M., Paul Peluso, William J. Rowell, Pi-Chuan Chang,
Richard J. Hall, Gregory T. Concepcion, Jana Ebler, et al. 2019.
<span>“Accurate Circular Consensus Long-Read Sequencing Improves Variant
Detection and Assembly of a Human Genome.”</span> <em>Nature
Biotechnology</em> 37 (10): 1155–62. <a href="https://doi.org/10.1038/s41587-019-0217-9">https://doi.org/10.1038/s41587-019-0217-9</a>.
</div>
<div id="ref-whirl-PharmGKB_2012" class="csl-entry" role="listitem">
Whirl-Carrillo, M, E M McDonagh, J M Hebert, L Gong, K Sangkuhl, C F
Thorn, R B Altman, and T E Klein. 2012. <span>“Pharmacogenomics
<span>Knowledge</span> for <span>Personalized</span>
<span>Medicine</span>.”</span> <em>Clinical Pharmacology &amp;
Therapeutics</em> 92 (4): 414–17. <a href="https://doi.org/10.1038/clpt.2012.96">https://doi.org/10.1038/clpt.2012.96</a>.
</div>
<div id="ref-wohlwend_boltz1_2025" class="csl-entry" role="listitem">
Wohlwend, Jeremy, Gabriele Corso, Saro Passaro, Noah Getz, Mateo Reveiz,
Ken Leidal, Wojtek Swiderski, et al. 2025. <span>“Boltz-1
<span>Democratizing</span> <span>Biomolecular</span>
<span>Interaction</span> <span>Modeling</span>.”</span> bioRxiv. <a href="https://doi.org/10.1101/2024.11.19.624167">https://doi.org/10.1101/2024.11.19.624167</a>.
</div>
<div id="ref-wu_genome-wide_2024" class="csl-entry" role="listitem">
Wu, Yang, Zhili Zheng, Loic Thibaut2, Michael E. Goddard, Naomi R. Wray,
Peter M. Visscher, and Jian Zeng. 2024. <span>“Genome-Wide Fine-Mapping
Improves Identification of Causal Variants.”</span> <em>Research
Square</em>, August, rs.3.rs–4759390. <a href="https://doi.org/10.21203/rs.3.rs-4759390/v1">https://doi.org/10.21203/rs.3.rs-4759390/v1</a>.
</div>
<div id="ref-yan_recent_2025" class="csl-entry" role="listitem">
Yan, Binghao, Yunbi Nam, Lingyao Li, Rebecca A. Deek, Hongzhe Li, and
Siyuan Ma. 2025. <span>“Recent Advances in Deep Learning and Language
Models for Studying the Microbiome.”</span> <em>Frontiers in
Genetics</em> 15 (January). <a href="https://doi.org/10.3389/fgene.2024.1494474">https://doi.org/10.3389/fgene.2024.1494474</a>.
</div>
<div id="ref-yang_common_2010" class="csl-entry" role="listitem">
Yang, Jian, Beben Benyamin, Brian P. McEvoy, Scott Gordon, Anjali K.
Henders, Dale R. Nyholt, Pamela A. Madden, et al. 2010. <span>“Common
<span>SNPs</span> Explain a Large Proportion of the Heritability for
Human Height.”</span> <em>Nature Genetics</em> 42 (7): 565–69. <a href="https://doi.org/10.1038/ng.608">https://doi.org/10.1038/ng.608</a>.
</div>
<div id="ref-yang_xlnet_2020" class="csl-entry" role="listitem">
Yang, Zhilin, Zihang Dai, Yiming Yang, Jaime Carbonell, Ruslan
Salakhutdinov, and Quoc V. Le. 2020. <span>“<span>XLNet</span>:
<span>Generalized</span> <span>Autoregressive</span>
<span>Pretraining</span> for <span>Language</span>
<span>Understanding</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.1906.08237">https://doi.org/10.48550/arXiv.1906.08237</a>.
</div>
<div id="ref-yeo_maxentscan_2004" class="csl-entry" role="listitem">
Yeo, Gene, and Christopher B. Burge. 2004. <span>“Maximum
<span>Entropy</span> <span>Modeling</span> of <span>Short</span>
<span>Sequence</span> <span>Motifs</span> with <span>Applications</span>
to <span>RNA</span> <span>Splicing</span> <span>Signals</span>.”</span>
<em>Journal of Computational Biology</em> 11 (2-3): 377–94. <a href="https://doi.org/10.1089/1066527041410418">https://doi.org/10.1089/1066527041410418</a>.
</div>
<div id="ref-yuan_linger_2025" class="csl-entry" role="listitem">
Yuan, Qiuyue, and Zhana Duren. 2025. <span>“[<span>LINGER</span>]
<span>Inferring</span> Gene Regulatory Networks from Single-Cell
Multiome Data Using Atlas-Scale External Data.”</span> <em>Nature
Biotechnology</em> 43 (2): 247–57. <a href="https://doi.org/10.1038/s41587-024-02182-7">https://doi.org/10.1038/s41587-024-02182-7</a>.
</div>
<div id="ref-yun_accurate_2021" class="csl-entry" role="listitem">
Yun, Taedong, Helen Li, Pi-Chuan Chang, Michael F Lin, Andrew Carroll,
and Cory Y McLean. 2021. <span>“Accurate, Scalable Cohort Variant Calls
Using <span>DeepVariant</span> and <span>GLnexus</span>.”</span>
<em>Bioinformatics</em> 36 (24): 5582–89. <a href="https://doi.org/10.1093/bioinformatics/btaa1081">https://doi.org/10.1093/bioinformatics/btaa1081</a>.
</div>
<div id="ref-zheng_cistrome_2019" class="csl-entry" role="listitem">
Zheng, Rongbin, Changxin Wan, Shenglin Mei, Qian Qin, Qiu Wu, Hanfei
Sun, Chen-Hao Chen, et al. 2019. <span>“Cistrome <span>Data</span>
<span>Browser</span>: Expanded Datasets and New Tools for Gene
Regulatory Analysis.”</span> <em>Nucleic Acids Research</em> 47 (D1):
D729–35. <a href="https://doi.org/10.1093/nar/gky1094">https://doi.org/10.1093/nar/gky1094</a>.
</div>
<div id="ref-zheng_clair3_2022" class="csl-entry" role="listitem">
Zheng, Zhenxian, Shumin Li, Junhao Su, Amy Wing-Sze Leung, Tak-Wah Lam,
and Ruibang Luo. 2022. <span>“Symphonizing Pileup and Full-Alignment for
Deep Learning-Based Long-Read Variant Calling.”</span> <em>Nature
Computational Science</em> 2 (12): 797–803. <a href="https://doi.org/10.1038/s43588-022-00387-x">https://doi.org/10.1038/s43588-022-00387-x</a>.
</div>
<div id="ref-zhou_expecto_2018" class="csl-entry" role="listitem">
Zhou, Jian, Chandra L. Theesfeld, Kevin Yao, Kathleen M. Chen, Aaron K.
Wong, and Olga G. Troyanskaya. 2018. <span>“[<span>Expecto</span>]
<span>Deep</span> Learning Sequence-Based Ab Initio Prediction of
Variant Effects on Expression and Disease Risk.”</span> <em>Nature
Genetics</em> 50 (8): 1171–79. <a href="https://doi.org/10.1038/s41588-018-0160-6">https://doi.org/10.1038/s41588-018-0160-6</a>.
</div>
<div id="ref-zhou_deepsea_2015" class="csl-entry" role="listitem">
Zhou, Jian, and Olga G. Troyanskaya. 2015. <span>“[<span>DeepSEA</span>]
<span>Predicting</span> Effects of Noncoding Variants with Deep
Learning–Based Sequence Model.”</span> <em>Nature Methods</em> 12 (10):
931–34. <a href="https://doi.org/10.1038/nmeth.3547">https://doi.org/10.1038/nmeth.3547</a>.
</div>
<div id="ref-zhou_dnabert-2_2024" class="csl-entry" role="listitem">
Zhou, Zhihan, Yanrong Ji, Weijian Li, Pratik Dutta, Ramana Davuluri, and
Han Liu. 2024. <span>“<span>DNABERT</span>-2: <span>Efficient</span>
<span>Foundation</span> <span>Model</span> and <span>Benchmark</span>
<span>For</span> <span>Multi</span>-<span>Species</span>
<span>Genome</span>.”</span> arXiv. <a href="https://doi.org/10.48550/arXiv.2306.15006">https://doi.org/10.48550/arXiv.2306.15006</a>.
</div>
<div id="ref-zook_giab_2019" class="csl-entry" role="listitem">
Zook, Justin M., Jennifer McDaniel, Nathan D. Olson, Justin Wagner,
Hemang Parikh, Haynes Heaton, Sean A. Irvine, et al. 2019. <span>“An
Open Resource for Accurately Benchmarking Small Variant and Reference
Calls.”</span> <em>Nature Biotechnology</em> 37 (5): 561–66. <a href="https://doi.org/10.1038/s41587-019-0074-6">https://doi.org/10.1038/s41587-019-0074-6</a>.
</div>
<div id="ref-zvyagin_genslms_2022" class="csl-entry" role="listitem">
Zvyagin, Maxim, Alexander Brace, Kyle Hippe, Yuntian Deng, Bin Zhang,
Cindy Orozco Bohorquez, Austin Clyde, et al. 2022.
<span>“<span>GenSLMs</span>: <span>Genome</span>-Scale Language Models
Reveal <span>SARS</span>-<span>CoV</span>-2 Evolutionary
Dynamics.”</span> bioRxiv. <a href="https://doi.org/10.1101/2022.10.10.511571">https://doi.org/10.1101/2022.10.10.511571</a>.
</div>
</div>



</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    // Ensure there is a toggle, if there isn't float one in the top right
    if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
      const a = window.document.createElement('a');
      a.classList.add('top-right');
      a.classList.add('quarto-color-scheme-toggle');
      a.href = "";
      a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
      const i = window.document.createElement("i");
      i.classList.add('bi');
      a.appendChild(i);
      window.document.body.appendChild(a);
    }
    setColorSchemeToggle(hasAlternateSentinel())
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
      const outerScaffold = trigger.parentElement.cloneNode(true);
      const codeEl = outerScaffold.querySelector('code');
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./p6-ch27-future.html" class="pagination-link" aria-label="Future Work &amp; Ethics">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">27</span>&nbsp; <span class="chapter-title">Future Work &amp; Ethics</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./app-a-dl.html" class="pagination-link" aria-label="Deep Learning Primer">
        <span class="nav-page-text"><span class="chapter-number">A</span>&nbsp; <span class="chapter-title">Deep Learning Primer</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      &nbsp;
    </div>   
    <div class="nav-footer-center">
<p>Copyright 2025, Josh Meehl</p>
</div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>




</body></html>